{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append(\"/Users/xutingfeng/Nutstore Files/wanglab工作汇报/project/statistic/proture\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd \n",
    "from proture.utils import find_comment, strip_comment_head\n",
    "from typing import Union\n",
    "pd.set_option('display.max_columns', 100)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 121,
   "metadata": {},
   "outputs": [],
   "source": [
    "gnomAD_path = \"../data/test.vcf.gz\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def load_gnomAD(gnomAD_path:str):\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<vcfpy.reader.Reader at 0x119659eb0>"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import vcfpy\n",
    "reader"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "writer = vcfpy.Writer.from_path('./test.vcf', reader.header)\n",
    "pd.DataFrame.apply()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 111,
   "metadata": {},
   "outputs": [],
   "source": [
    "import re \n",
    "class VEP_PARSER:\n",
    "    def __init__(self, vep_info_header:vcfpy.header.InfoHeaderLine) -> None:\n",
    "        self.vep_col = self.parse_vep_header_parse(vep_info_header)\n",
    "        \n",
    "    @staticmethod\n",
    "    def parse_vep_header_parse(vep_info_header):\n",
    "        if match:=re.search(r\"(?<=Format:).*\", vep_info_header.description):\n",
    "            vep_col = match.group().split(\"|\")\n",
    "            return vep_col \n",
    "\n",
    "        else:\n",
    "            raise ValueError(f\"can't find any col of vep at {vep_info_header}\")\n",
    "    def apply(self, vep_line_str:str):\n",
    "        vep_line_list = vep_line_str.split(\"|\")\n",
    "        if len(vep_line_list) != len(self.vep_col):\n",
    "            # raise ValueError(f\"{vep_line_str} is not match with {self.vep_col}\")\n",
    "            return {}\n",
    "        else:\n",
    "            return dict(zip(self.vep_col, vep_line_list))\n",
    "\n",
    "    def filter(self, func, vep_info:list,\n",
    "     how:str = \"all\"\n",
    "     ):\n",
    "        \"\"\"\n",
    "        apply vep_info will be a dict with vep_col in the vep_parser. the func accept input is a dict, which key are vep_col and values are vep_info, and output True of False, which decide this indices of vep_info drop or keep.\n",
    "\n",
    "        Args:\n",
    "            func (_type_): _description_\n",
    "            vep_info (list): _description_\n",
    "        Example:\n",
    "            import vcfpy \n",
    "\n",
    "            reader = vcfpy.Reader.from_path(gnomAD_path)\n",
    "            vep_parser = VEP_PARSER(reader.header.get_info_field_info(\"vep\"))\n",
    "\n",
    "            vep_parser.filter(lambda x: print(x) if x[\"CANONICAL\"] != \"\" else None, a.INFO[\"vep\"])\n",
    "        Returns:\n",
    "            List:\n",
    "\n",
    "        \"\"\"\n",
    "        out = []\n",
    "        for vep_line in vep_info:\n",
    "            vep_parse_result = self.apply(vep_line)\n",
    "            if not vep_parse_result:\n",
    "                raise ValueError(f\"{vep_line} is not match with {self.vep_col}\")\n",
    "\n",
    "            if func(vep_parse_result):\n",
    "                out.append(vep_line)\n",
    "\n",
    "                if how ==\"any\":\n",
    "                    break \n",
    "                elif how == \"all\":\n",
    "                    continue\n",
    "                else:\n",
    "                    raise ValueError(f\"{how} is not any or all\")\n",
    "                    \n",
    "        return out\n",
    "            # func\n",
    "    def __repr__(self) -> str:\n",
    "        return '\\t'.join(self.vep_col)\n",
    "\n",
    "\n",
    "def is_missense_mutant_at_HGVSp(HGVSp:str):\n",
    "    missense_mutant_HGVSp_pattern = r\"p.[A-Za-z]+[0-9]+[A-Za-z]+\"\n",
    "    return bool(re.search(missense_mutant_HGVSp_pattern, HGVSp))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 123,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{' Allele': 'C', 'Consequence': 'downstream_gene_variant', 'IMPACT': 'MODIFIER', 'SYMBOL': 'WASH7P', 'Gene': 'ENSG00000227232', 'Feature_type': 'Transcript', 'Feature': 'ENST00000438504', 'BIOTYPE': 'unprocessed_pseudogene', 'EXON': '', 'INTRON': '', 'HGVSc': '', 'HGVSp': '', 'cDNA_position': '', 'CDS_position': '', 'Protein_position': '', 'Amino_acids': '', 'Codons': '', 'Existing_variation': '', 'ALLELE_NUM': '1', 'DISTANCE': '1790', 'STRAND': '-1', 'FLAGS': '', 'VARIANT_CLASS': 'SNV', 'MINIMISED': '1', 'SYMBOL_SOURCE': 'HGNC', 'HGNC_ID': '38034', 'CANONICAL': 'YES', 'TSL': '', 'APPRIS': '', 'CCDS': '', 'ENSP': '', 'SWISSPROT': '', 'TREMBL': '', 'UNIPARC': '', 'GENE_PHENO': '', 'SIFT': '', 'PolyPhen': '', 'DOMAINS': '', 'HGVS_OFFSET': '', 'GMAF': '', 'AFR_MAF': '', 'AMR_MAF': '', 'EAS_MAF': '', 'EUR_MAF': '', 'SAS_MAF': '', 'AA_MAF': '', 'EA_MAF': '', 'ExAC_MAF': '', 'ExAC_Adj_MAF': '', 'ExAC_AFR_MAF': '', 'ExAC_AMR_MAF': '', 'ExAC_EAS_MAF': '', 'ExAC_FIN_MAF': '', 'ExAC_NFE_MAF': '', 'ExAC_OTH_MAF': '', 'ExAC_SAS_MAF': '', 'CLIN_SIG': '', 'SOMATIC': '', 'PHENO': '', 'PUBMED': '', 'MOTIF_NAME': '', 'MOTIF_POS': '', 'HIGH_INF_POS': '', 'MOTIF_SCORE_CHANGE': '', 'LoF': '', 'LoF_filter': '', 'LoF_flags': '', 'LoF_info': ''}\n",
      "{' Allele': 'C', 'Consequence': 'intron_variant&non_coding_transcript_variant', 'IMPACT': 'MODIFIER', 'SYMBOL': 'DDX11L1', 'Gene': 'ENSG00000223972', 'Feature_type': 'Transcript', 'Feature': 'ENST00000456328', 'BIOTYPE': 'processed_transcript', 'EXON': '', 'INTRON': '1/2', 'HGVSc': 'ENST00000456328.2:n.360-40T>C', 'HGVSp': '', 'cDNA_position': '', 'CDS_position': '', 'Protein_position': '', 'Amino_acids': '', 'Codons': '', 'Existing_variation': '', 'ALLELE_NUM': '1', 'DISTANCE': '', 'STRAND': '1', 'FLAGS': '', 'VARIANT_CLASS': 'SNV', 'MINIMISED': '1', 'SYMBOL_SOURCE': 'HGNC', 'HGNC_ID': '37102', 'CANONICAL': 'YES', 'TSL': '', 'APPRIS': '', 'CCDS': '', 'ENSP': '', 'SWISSPROT': '', 'TREMBL': '', 'UNIPARC': '', 'GENE_PHENO': '', 'SIFT': '', 'PolyPhen': '', 'DOMAINS': '', 'HGVS_OFFSET': '', 'GMAF': '', 'AFR_MAF': '', 'AMR_MAF': '', 'EAS_MAF': '', 'EUR_MAF': '', 'SAS_MAF': '', 'AA_MAF': '', 'EA_MAF': '', 'ExAC_MAF': '', 'ExAC_Adj_MAF': '', 'ExAC_AFR_MAF': '', 'ExAC_AMR_MAF': '', 'ExAC_EAS_MAF': '', 'ExAC_FIN_MAF': '', 'ExAC_NFE_MAF': '', 'ExAC_OTH_MAF': '', 'ExAC_SAS_MAF': '', 'CLIN_SIG': '', 'SOMATIC': '', 'PHENO': '', 'PUBMED': '', 'MOTIF_NAME': '', 'MOTIF_POS': '', 'HIGH_INF_POS': '', 'MOTIF_SCORE_CHANGE': '', 'LoF': '', 'LoF_filter': '', 'LoF_flags': '', 'LoF_info': ''}\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 123,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import vcfpy \n",
    "\n",
    "reader = vcfpy.Reader.from_path(gnomAD_path)\n",
    "a = next(reader)\n",
    "vep_parser = VEP_PARSER(reader.header.get_info_field_info(\"vep\"))\n",
    "                        \n",
    "vep_parser.filter(lambda x: print(x) if x[\"CANONICAL\"] != \"\" else None, a.INFO[\"vep\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['ENSP00000334393.3:p.Glu15Gly']\n",
      "['ENSP00000334393.3:p.Leu20Gln']\n",
      "['ENSP00000334393.3:p.Tyr28Phe']\n",
      "['ENSP00000334393.3:p.Val32Met']\n",
      "['ENSP00000334393.3:p.Asp45Val']\n",
      "['ENSP00000334393.3:p.Arg82Cys']\n",
      "['ENSP00000334393.3:p.Arg82His']\n",
      "['ENSP00000334393.3:p.Lys83Gln']\n",
      "['ENSP00000334393.3:p.Lys88Glu']\n",
      "['ENSP00000334393.3:p.Cys90Tyr']\n",
      "['ENSP00000334393.3:p.Gly102Glu']\n",
      "['ENSP00000334393.3:p.Ile107Val']\n",
      "['ENSP00000334393.3:p.Ile109Thr']\n",
      "['ENSP00000334393.3:p.Phe113Cys']\n",
      "['ENSP00000334393.3:p.Asp114Gly']\n",
      "['ENSP00000334393.3:p.Ile117Arg']\n",
      "['ENSP00000334393.3:p.Ile117Met']\n",
      "['ENSP00000334393.3:p.Cys120Ser']\n",
      "['ENSP00000334393.3:p.Leu123Pro']\n",
      "['ENSP00000334393.3:p.Tyr125Cys']\n",
      "['ENSP00000334393.3:p.Ile128Met']\n",
      "['ENSP00000334393.3:p.Met129Thr']\n",
      "['ENSP00000334393.3:p.Cys130Arg']\n",
      "['ENSP00000334393.3:p.Gly131Arg']\n",
      "['ENSP00000334393.3:p.Asn132Lys']\n",
      "['ENSP00000334393.3:p.Ala133Thr']\n",
      "['ENSP00000334393.3:p.Val135Ala']\n",
      "['ENSP00000334393.3:p.Gly136Ser']\n",
      "['ENSP00000334393.3:p.Met138Val']\n",
      "['ENSP00000334393.3:p.Thr141Ala']\n",
      "['ENSP00000334393.3:p.Trp142Ter']\n",
      "['ENSP00000334393.3:p.Gly143Arg']\n",
      "['ENSP00000334393.3:p.Ile144Asn']\n",
      "['ENSP00000334393.3:p.Ile144Thr']\n",
      "['ENSP00000334393.3:p.Gly145Cys']\n",
      "['ENSP00000334393.3:p.Gly145Val']\n",
      "['ENSP00000334393.3:p.His148Asn']\n",
      "['ENSP00000334393.3:p.Ser149Leu']\n",
      "['ENSP00000334393.3:p.Ser151Gly']\n",
      "['ENSP00000334393.3:p.Ala154Gly']\n",
      "['ENSP00000334393.3:p.Ala154Val']\n",
      "['ENSP00000334393.3:p.Phe155Leu']\n",
      "['ENSP00000334393.3:p.Val157Met']\n",
      "['ENSP00000334393.3:p.Leu159Val']\n",
      "['ENSP00000334393.3:p.Leu160Pro']\n",
      "['ENSP00000334393.3:p.Phe161Leu']\n",
      "['ENSP00000334393.3:p.Asn165Ser']\n",
      "['ENSP00000334393.3:p.Val167Asp']\n",
      "['ENSP00000334393.3:p.Asp168Asn']\n",
      "['ENSP00000334393.3:p.Asp168Tyr']\n",
      "['ENSP00000334393.3:p.Cys172Arg']\n",
      "['ENSP00000334393.3:p.Leu174Phe']\n",
      "['ENSP00000334393.3:p.Ala181Val']\n",
      "['ENSP00000334393.3:p.Cys182Ser']\n",
      "['ENSP00000334393.3:p.Asp184His']\n",
      "['ENSP00000334393.3:p.Tyr186His']\n",
      "['ENSP00000334393.3:p.Arg187Trp']\n",
      "['ENSP00000334393.3:p.Leu188Pro']\n",
      "['ENSP00000334393.3:p.Asp189His']\n",
      "['ENSP00000334393.3:p.Val192Phe']\n",
      "['ENSP00000334393.3:p.Val192Asp']\n",
      "['ENSP00000334393.3:p.Ala194Val']\n",
      "['ENSP00000334393.3:p.Ser196Asn']\n",
      "['ENSP00000334393.3:p.Gly197Asp']\n",
      "['ENSP00000334393.3:p.Val198Met']\n",
      "['ENSP00000334393.3:p.Ile208Thr']\n",
      "['ENSP00000334393.3:p.Ile209Val']\n",
      "['ENSP00000334393.3:p.Thr212Pro']\n",
      "['ENSP00000334393.3:p.Ile214Thr']\n",
      "['ENSP00000334393.3:p.Gln219Ter']\n",
      "['ENSP00000334393.3:p.His220Asn']\n",
      "['ENSP00000334393.3:p.Arg221Cys']\n",
      "['ENSP00000334393.3:p.Arg221His']\n",
      "['ENSP00000334393.3:p.Asp224Val']\n",
      "['ENSP00000334393.3:p.Val240Leu']\n",
      "['ENSP00000334393.3:p.Leu242Phe']\n",
      "['ENSP00000334393.3:p.Phe243Leu']\n",
      "['ENSP00000334393.3:p.Cys247Arg']\n",
      "['ENSP00000334393.3:p.Phe255Leu']\n",
      "['ENSP00000334393.3:p.Leu260Ter']\n",
      "['ENSP00000334393.3:p.Tyr268His']\n",
      "['ENSP00000334393.3:p.Val270Glu']\n",
      "['ENSP00000334393.3:p.Val270Ala']\n",
      "['ENSP00000334393.3:p.Ile278Ser']\n",
      "['ENSP00000334393.3:p.Asp286Asn']\n",
      "['ENSP00000334393.3:p.Asp286Gly']\n",
      "['ENSP00000334393.3:p.Thr289Met']\n",
      "['ENSP00000334393.3:p.Gln293Arg']\n",
      "['ENSP00000334393.3:p.Ser302Thr']\n",
      "['ENSP00000473460.1:p.Ter260ArgextTer2']\n",
      "['ENSP00000473460.1:p.Asn258Ile']\n",
      "['ENSP00000473460.1:p.Asn258Ser']\n",
      "['ENSP00000473460.1:p.Arg256Thr']\n",
      "['ENSP00000473460.1:p.Arg256Lys']\n",
      "['ENSP00000473460.1:p.Phe255Cys']\n",
      "['ENSP00000473460.1:p.Pro254Leu']\n",
      "['ENSP00000473460.1:p.Pro250Leu']\n",
      "['ENSP00000473460.1:p.Pro250Arg']\n",
      "['ENSP00000473460.1:p.Pro250Thr']\n",
      "['ENSP00000473460.1:p.Leu249Phe']\n",
      "['ENSP00000473460.1:p.Leu248Pro']\n",
      "['ENSP00000473460.1:p.Gln247Ter']\n",
      "['ENSP00000473460.1:p.Gln247Glu']\n",
      "['ENSP00000473460.1:p.Ala239Thr']\n",
      "['ENSP00000473460.1:p.Gly219Glu']\n",
      "['ENSP00000473460.1:p.Pro213Leu']\n",
      "['ENSP00000473460.1:p.Pro213Ser']\n",
      "['ENSP00000473460.1:p.Thr208Ile']\n",
      "['ENSP00000473460.1:p.Ser203Arg']\n",
      "['ENSP00000473460.1:p.Ser203Asn']\n",
      "['ENSP00000473460.1:p.Pro202Ser']\n",
      "['ENSP00000473460.1:p.Asn200Lys']\n",
      "['ENSP00000473460.1:p.Gly198Ala']\n",
      "['ENSP00000473460.1:p.Gly197Ser']\n",
      "['ENSP00000473460.1:p.Leu195Phe']\n",
      "['ENSP00000473460.1:p.Leu193Phe']\n",
      "['ENSP00000473460.1:p.His192Tyr']\n",
      "['ENSP00000473460.1:p.Thr188Ser']\n",
      "['ENSP00000473460.1:p.Thr188Ser']\n",
      "['ENSP00000473460.1:p.Gln187His']\n",
      "['ENSP00000473460.1:p.Thr183Ile']\n",
      "['ENSP00000473460.1:p.Lys182Thr']\n",
      "['ENSP00000473460.1:p.Pro181Ala']\n",
      "['ENSP00000473460.1:p.Arg180Ser']\n",
      "['ENSP00000473460.1:p.Gln178Glu']\n",
      "['ENSP00000473460.1:p.Gly177Ser']\n",
      "['ENSP00000473460.1:p.Ala174Thr']\n",
      "['ENSP00000473460.1:p.Pro173Arg']\n",
      "['ENSP00000473460.1:p.Ser171Phe']\n",
      "['ENSP00000473460.1:p.Ser171Cys']\n",
      "['ENSP00000473460.1:p.Ser170Asn']\n",
      "['ENSP00000473460.1:p.Ser170Arg']\n",
      "['ENSP00000473460.1:p.Pro169Arg']\n",
      "['ENSP00000473460.1:p.Pro169Ser']\n",
      "['ENSP00000473460.1:p.Pro169Thr']\n",
      "['ENSP00000473460.1:p.Ser165Gly']\n",
      "['ENSP00000473460.1:p.Asn164Asp']\n",
      "['ENSP00000473460.1:p.His163Pro']\n",
      "['ENSP00000473460.1:p.His163Tyr']\n",
      "['ENSP00000473460.1:p.Leu161Met']\n",
      "['ENSP00000473460.1:p.Gly155Asp']\n",
      "['ENSP00000473460.1:p.Ser151Phe']\n",
      "['ENSP00000473460.1:p.Ser146Pro']\n",
      "['ENSP00000473460.1:p.Gly144Ala']\n",
      "['ENSP00000473460.1:p.Gly144Asp']\n",
      "['ENSP00000473460.1:p.Gly143Glu']\n",
      "['ENSP00000473460.1:p.His142Tyr']\n",
      "['ENSP00000473460.1:p.His142Asn']\n",
      "['ENSP00000473460.1:p.Leu140Val']\n",
      "['ENSP00000473460.1:p.Pro139Leu']\n",
      "['ENSP00000473460.1:p.Pro139Arg']\n",
      "['ENSP00000473460.1:p.Pro139His']\n",
      "['ENSP00000473460.1:p.Pro139Ser']\n",
      "['ENSP00000473460.1:p.Ala137Val']\n",
      "['ENSP00000473460.1:p.Gln136Arg']\n",
      "['ENSP00000473460.1:p.Arg134Leu']\n",
      "['ENSP00000473460.1:p.Arg134His']\n",
      "['ENSP00000473460.1:p.Arg134Cys']\n",
      "['ENSP00000473460.1:p.Gln133Arg']\n",
      "['ENSP00000473460.1:p.Gln133Ter']\n",
      "['ENSP00000473460.1:p.Gln133Glu']\n",
      "['ENSP00000473460.1:p.Ser131Phe']\n",
      "['ENSP00000473460.1:p.Pro129Ser']\n",
      "['ENSP00000473460.1:p.Pro127Ala']\n",
      "['ENSP00000473460.1:p.Thr123Ile']\n",
      "['ENSP00000473460.1:p.Lys121Asn']\n",
      "['ENSP00000473460.1:p.Leu119Val']\n",
      "['ENSP00000473460.1:p.Gln117Pro']\n",
      "['ENSP00000473460.1:p.Met114Thr']\n",
      "['ENSP00000473460.1:p.Pro113Leu']\n",
      "['ENSP00000473460.1:p.Trp112Arg']\n",
      "['ENSP00000473460.1:p.Trp111Cys']\n",
      "['ENSP00000473460.1:p.Trp111Ter']\n",
      "['ENSP00000473460.1:p.Thr107Ile']\n",
      "['ENSP00000473460.1:p.Pro106Arg']\n",
      "['ENSP00000473460.1:p.Leu104Val']\n",
      "['ENSP00000473460.1:p.Ser101Ala']\n",
      "['ENSP00000473460.1:p.Leu99Arg']\n",
      "['ENSP00000473460.1:p.Leu99Phe']\n",
      "['ENSP00000473460.1:p.Leu99Ile']\n",
      "['ENSP00000473460.1:p.Leu98Phe']\n",
      "['ENSP00000473460.1:p.Asn97Ser']\n",
      "['ENSP00000473460.1:p.Ala95Asp']\n",
      "['ENSP00000473460.1:p.Arg94Gln']\n",
      "['ENSP00000473460.1:p.Arg94Trp']\n",
      "['ENSP00000473460.1:p.Leu93Phe']\n",
      "['ENSP00000473460.1:p.His92Tyr']\n",
      "['ENSP00000473460.1:p.Arg84Gln']\n",
      "['ENSP00000473460.1:p.Arg84Trp']\n",
      "['ENSP00000473460.1:p.Ala81Val']\n",
      "['ENSP00000473460.1:p.Ala81Ser']\n",
      "['ENSP00000473460.1:p.Val80Met']\n",
      "['ENSP00000473460.1:p.Thr79Met']\n",
      "['ENSP00000473460.1:p.Thr69Ala']\n",
      "['ENSP00000473460.1:p.Thr69Pro']\n",
      "['ENSP00000473460.1:p.Pro67Leu']\n",
      "['ENSP00000473460.1:p.Gln64Ter']\n",
      "['ENSP00000473460.1:p.Gln64Glu']\n",
      "['ENSP00000473460.1:p.Ala63Val']\n",
      "['ENSP00000473460.1:p.Gln62Arg']\n",
      "['ENSP00000473460.1:p.Arg61Ile']\n",
      "['ENSP00000473460.1:p.Pro60Ser']\n",
      "['ENSP00000473460.1:p.Gln58Arg']\n",
      "['ENSP00000473460.1:p.Gln58Glu']\n",
      "['ENSP00000473460.1:p.Gln58Lys']\n",
      "['ENSP00000473460.1:p.Phe43Ser']\n",
      "['ENSP00000473460.1:p.Pro41Thr']\n",
      "['ENSP00000473460.1:p.Pro39Ser']\n",
      "['ENSP00000473460.1:p.Phe38Cys']\n",
      "['ENSP00000473460.1:p.Arg36Gly']\n",
      "['ENSP00000473460.1:p.Thr35Lys']\n",
      "['ENSP00000473460.1:p.Thr35Pro']\n",
      "['ENSP00000473460.1:p.Cys33Arg']\n",
      "['ENSP00000473460.1:p.Leu27Phe']\n",
      "['ENSP00000473460.1:p.Gly26Val']\n",
      "['ENSP00000473460.1:p.Ser25Asn']\n",
      "['ENSP00000473460.1:p.Ser25Gly']\n",
      "['ENSP00000473460.1:p.Leu21Phe']\n",
      "['ENSP00000473460.1:p.Thr19Ser']\n",
      "['ENSP00000473460.1:p.Gln14Arg']\n",
      "['ENSP00000473460.1:p.Phe13Val']\n",
      "['ENSP00000473460.1:p.Pro11Arg']\n",
      "['ENSP00000473460.1:p.Ala8Pro']\n",
      "['ENSP00000473460.1:p.Ala8Thr']\n",
      "['ENSP00000473460.1:p.Gln7Ter']\n",
      "['ENSP00000473460.1:p.Pro6Ser']\n",
      "['ENSP00000473460.1:p.Thr5Ile']\n",
      "['ENSP00000473460.1:p.Leu3Pro']\n"
     ]
    }
   ],
   "source": [
    "import vcfpy \n",
    "\n",
    "reader = vcfpy.Reader.from_path(gnomAD_path)\n",
    "\n",
    "writer = vcfpy.Writer.from_path(\"./test.vcf.gz\", reader.header)\n",
    "\n",
    "vep_parser = VEP_PARSER(reader.header.get_info_field_info(\"vep\"))\n",
    "\n",
    "for record in reader:\n",
    "    if vep_filter:=vep_parser.filter(lambda x: is_missense_mutant_at_HGVSp(x[\"HGVSp\"]) and x[\"CANONICAL\"] != \"\", record.INFO[\"vep\"], how=\"all\"):\n",
    "        print([vep_parser.apply(i)[\"HGVSp\"] for i in vep_filter])\n",
    "        record.INFO[\"vep\"] = vep_filter\n",
    "        writer.write_record(record)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<generator object test at 0x11a427900>"
      ]
     },
     "execution_count": 120,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "range(100000)\n",
    "def test():\n",
    "    for i in range(100000):\n",
    "        yield i \n",
    "\n",
    "test()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!/usr/bin/env python\n",
    "# -*- coding: utf-8 -*-\n",
    "import vcfpy\n",
    "\n",
    "# Open input, add FILTER header, and open output file\n",
    "reader = vcfpy.Reader.from_path('input.vcf')\n",
    "reader.header.add_filter_line(vcfpy.OrderedDict([\n",
    "    ('ID', 'DP10'), ('Description', 'total DP < 10')]))\n",
    "writer = vcfpy.Writer.from_path('/dev/stdout', reader.header)\n",
    "\n",
    "# Add \"DP10\" filter to records having less than 10 reads\n",
    "for record in reader:\n",
    "    ad = sum(c.data.get('DP', 0) for c in record.calls)\n",
    "    if ad < 10:\n",
    "        record.add_filter('DP10')\n",
    "    writer.write_record(record)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Record('1', 12573, ['rs1273605438'], 'T', [Substitution(type_='SNV', value='C')], 366.59, ['PASS'], {'AC': [2], 'AN': 4196, 'AF': [0.000476644], 'rf_tp_probability': 0.322028, 'FS': 0.0, 'InbreedingCoeff': -0.0756, 'MQ': 22.0, 'MQRankSum': -0.72, 'QD': 4.03, 'ReadPosRankSum': 0.406, 'SOR': 0.591, 'BaseQRankSum': 0.156, 'ClippingRankSum': -0.387, 'DP': 159660, 'VQSLOD': 3.25, 'VQSR_culprit': 'QD', 'segdup': True, 'rf_negative_label': True, 'rf_label': 'FP', 'rf_train': True, 'variant_type': 'snv', 'allele_type': ['snv'], 'n_alt_alleles': [1], 'pab_max': [1.0], 'gq_hist_alt_bin_freq': ['0|0|0|2|0|0|1|1|1|0|0|1|0|0|0|0|0|0|0|2'], 'gq_hist_all_bin_freq': ['5328|7127|2608|3532|2074|679|849|436|132|204|108|31|43|1|14|0|7|1|1|2'], 'dp_hist_alt_bin_freq': ['2|4|1|1|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0'], 'dp_hist_alt_n_larger': [0], 'dp_hist_all_bin_freq': ['116953|6572|1691|421|85|22|4|0|0|0|0|0|0|0|0|0|0|0|0|0'], 'dp_hist_all_n_larger': [0], 'ab_hist_alt_bin_freq': ['0|0|0|0|0|1|1|0|0|1|1|0|2|1|0|1|0|0|0|0'], 'AC_nfe_seu': [0], 'AN_nfe_seu': 0, 'nhomalt_nfe_seu': [0], 'controls_AC_afr_male': [0], 'controls_AN_afr_male': 120, 'controls_AF_afr_male': [0.0], 'controls_nhomalt_afr_male': [0], 'non_neuro_AC_eas_kor': [0], 'non_neuro_AN_eas_kor': 318, 'non_neuro_AF_eas_kor': [0.0], 'non_neuro_nhomalt_eas_kor': [0], 'non_topmed_AC_amr': [2], 'non_topmed_AN_amr': 712, 'non_topmed_AF_amr': [0.00280899], 'non_topmed_nhomalt_amr': [0], 'non_cancer_AC_asj_female': [0], 'non_cancer_AN_asj_female': 24, 'non_cancer_AF_asj_female': [0.0], 'non_cancer_nhomalt_asj_female': [0], 'AC_raw': [8], 'AN_raw': 46354, 'AF_raw': [0.000172585], 'nhomalt_raw': [0], 'AC_fin_female': [0], 'AN_fin_female': 14, 'AF_fin_female': [0.0], 'nhomalt_fin_female': [0], 'non_cancer_AC_oth_female': [0], 'non_cancer_AN_oth_female': 64, 'non_cancer_AF_oth_female': [0.0], 'non_cancer_nhomalt_oth_female': [0], 'AC_nfe_bgr': [0], 'AN_nfe_bgr': 0, 'nhomalt_nfe_bgr': [0], 'non_neuro_AC_asj_female': [0], 'non_neuro_AN_asj_female': 4, 'non_neuro_AF_asj_female': [0.0], 'non_neuro_nhomalt_asj_female': [0], 'AC_sas_male': [0], 'AN_sas_male': 894, 'AF_sas_male': [0.0], 'nhomalt_sas_male': [0], 'non_neuro_AC_afr_male': [0], 'non_neuro_AN_afr_male': 260, 'non_neuro_AF_afr_male': [0.0], 'non_neuro_nhomalt_afr_male': [0], 'AC_afr_male': [0], 'AN_afr_male': 262, 'AF_afr_male': [0.0], 'nhomalt_afr_male': [0], 'AC_afr': [0], 'AN_afr': 624, 'AF_afr': [0.0], 'nhomalt_afr': [0], 'controls_AC_nfe_swe': [0], 'controls_AN_nfe_swe': 0, 'controls_nhomalt_nfe_swe': [0], 'non_neuro_AC_afr_female': [0], 'non_neuro_AN_afr_female': 360, 'non_neuro_AF_afr_female': [0.0], 'non_neuro_nhomalt_afr_female': [0], 'non_topmed_AC_amr_female': [2], 'non_topmed_AN_amr_female': 348, 'non_topmed_AF_amr_female': [0.00574713], 'non_topmed_nhomalt_amr_female': [0], 'non_cancer_AC_female': [2], 'non_cancer_AN_female': 1850, 'non_cancer_AF_female': [0.00108108], 'non_cancer_nhomalt_female': [0], 'non_cancer_AC_nfe_onf': [0], 'non_cancer_AN_nfe_onf': 166, 'non_cancer_AF_nfe_onf': [0.0], 'non_cancer_nhomalt_nfe_onf': [0], 'non_cancer_AC_male': [0], 'non_cancer_AN_male': 2346, 'non_cancer_AF_male': [0.0], 'non_cancer_nhomalt_male': [0], 'non_topmed_AC_oth_female': [0], 'non_topmed_AN_oth_female': 62, 'non_topmed_AF_oth_female': [0.0], 'non_topmed_nhomalt_oth_female': [0], 'AC_eas_female': [0], 'AN_eas_female': 278, 'AF_eas_female': [0.0], 'nhomalt_eas_female': [0], 'non_cancer_AC_sas_female': [0], 'non_cancer_AN_sas_female': 344, 'non_cancer_AF_sas_female': [0.0], 'non_cancer_nhomalt_sas_female': [0], 'AC_afr_female': [0], 'AN_afr_female': 362, 'AF_afr_female': [0.0], 'nhomalt_afr_female': [0], 'AC_sas': [0], 'AN_sas': 1238, 'AF_sas': [0.0], 'nhomalt_sas': [0], 'non_neuro_AC_female': [2], 'non_neuro_AN_female': 1768, 'non_neuro_AF_female': [0.00113122], 'non_neuro_nhomalt_female': [0], 'controls_AC_afr': [0], 'controls_AN_afr': 304, 'controls_AF_afr': [0.0], 'controls_nhomalt_afr': [0], 'non_neuro_AC_eas_jpn': [0], 'non_neuro_AN_eas_jpn': 0, 'non_neuro_nhomalt_eas_jpn': [0], 'AC_nfe_onf': [0], 'AN_nfe_onf': 166, 'AF_nfe_onf': [0.0], 'nhomalt_nfe_onf': [0], 'non_cancer_AC_amr_male': [0], 'non_cancer_AN_amr_male': 368, 'non_cancer_AF_amr_male': [0.0], 'non_cancer_nhomalt_amr_male': [0], 'controls_AC_fin_male': [0], 'controls_AN_fin_male': 0, 'controls_nhomalt_fin_male': [0], 'non_neuro_AC_nfe_nwe': [0], 'non_neuro_AN_nfe_nwe': 540, 'non_neuro_AF_nfe_nwe': [0.0], 'non_neuro_nhomalt_nfe_nwe': [0], 'AC_fin_male': [0], 'AN_fin_male': 22, 'AF_fin_male': [0.0], 'nhomalt_fin_male': [0], 'AC_nfe_female': [0], 'AN_nfe_female': 416, 'AF_nfe_female': [0.0], 'nhomalt_nfe_female': [0], 'AC_amr': [2], 'AN_amr': 716, 'AF_amr': [0.0027933], 'nhomalt_amr': [0], 'non_topmed_AC_nfe_male': [0], 'non_topmed_AN_nfe_male': 372, 'non_topmed_AF_nfe_male': [0.0], 'non_topmed_nhomalt_nfe_male': [0], 'non_neuro_AC_sas': [0], 'non_neuro_AN_sas': 1238, 'non_neuro_AF_sas': [0.0], 'non_neuro_nhomalt_sas': [0], 'non_cancer_AC_fin_male': [0], 'non_cancer_AN_fin_male': 22, 'non_cancer_AF_fin_male': [0.0], 'non_cancer_nhomalt_fin_male': [0], 'non_cancer_AC_nfe_seu': [0], 'non_cancer_AN_nfe_seu': 0, 'non_cancer_nhomalt_nfe_seu': [0], 'AC_eas': [0], 'AN_eas': 556, 'AF_eas': [0.0], 'nhomalt_eas': [0], 'nhomalt': [0], 'non_neuro_AC_nfe_female': [0], 'non_neuro_AN_nfe_female': 364, 'non_neuro_AF_nfe_female': [0.0], 'non_neuro_nhomalt_nfe_female': [0], 'non_neuro_AC_afr': [0], 'non_neuro_AN_afr': 620, 'non_neuro_AF_afr': [0.0], 'non_neuro_nhomalt_afr': [0], 'controls_AC_raw': [4], 'controls_AN_raw': 19152, 'controls_AF_raw': [0.000208855], 'controls_nhomalt_raw': [0], 'non_cancer_AC_eas': [0], 'non_cancer_AN_eas': 556, 'non_cancer_AF_eas': [0.0], 'non_cancer_nhomalt_eas': [0], 'non_cancer_AC_amr_female': [2], 'non_cancer_AN_amr_female': 348, 'non_cancer_AF_amr_female': [0.00574713], 'non_cancer_nhomalt_amr_female': [0], 'non_neuro_AC_nfe_swe': [0], 'non_neuro_AN_nfe_swe': 0, 'non_neuro_nhomalt_nfe_swe': [0], 'controls_AC_male': [0], 'controls_AN_male': 1074, 'controls_AF_male': [0.0], 'controls_nhomalt_male': [0], 'non_topmed_AC_male': [0], 'non_topmed_AN_male': 2276, 'non_topmed_AF_male': [0.0], 'non_topmed_nhomalt_male': [0], 'controls_AC_eas_jpn': [0], 'controls_AN_eas_jpn': 0, 'controls_nhomalt_eas_jpn': [0], 'controls_AC_nfe_female': [0], 'controls_AN_nfe_female': 198, 'controls_AF_nfe_female': [0.0], 'controls_nhomalt_nfe_female': [0], 'non_neuro_AC_amr': [2], 'non_neuro_AN_amr': 708, 'non_neuro_AF_amr': [0.00282486], 'non_neuro_nhomalt_amr': [0], 'non_neuro_AC_eas_female': [0], 'non_neuro_AN_eas_female': 278, 'non_neuro_AF_eas_female': [0.0], 'non_neuro_nhomalt_eas_female': [0], 'AC_asj_male': [0], 'AN_asj_male': 18, 'AF_asj_male': [0.0], 'nhomalt_asj_male': [0], 'controls_AC_nfe_male': [0], 'controls_AN_nfe_male': 182, 'controls_AF_nfe_male': [0.0], 'controls_nhomalt_nfe_male': [0], 'non_neuro_AC_fin': [0], 'non_neuro_AN_fin': 36, 'non_neuro_AF_fin': [0.0], 'non_neuro_nhomalt_fin': [0], 'non_topmed_AC_sas': [0], 'non_topmed_AN_sas': 1238, 'non_topmed_AF_sas': [0.0], 'non_topmed_nhomalt_sas': [0], 'non_cancer_AC_nfe_female': [0], 'non_cancer_AN_nfe_female': 416, 'non_cancer_AF_nfe_female': [0.0], 'non_cancer_nhomalt_nfe_female': [0], 'AC_oth_female': [0], 'AN_oth_female': 64, 'AF_oth_female': [0.0], 'nhomalt_oth_female': [0], 'non_cancer_AC_asj': [0], 'non_cancer_AN_asj': 42, 'non_cancer_AF_asj': [0.0], 'non_cancer_nhomalt_asj': [0], 'AC_nfe_swe': [0], 'AN_nfe_swe': 0, 'nhomalt_nfe_swe': [0], 'controls_AC_nfe': [0], 'controls_AN_nfe': 380, 'controls_AF_nfe': [0.0], 'controls_nhomalt_nfe': [0], 'controls_AC_oth_female': [0], 'controls_AN_oth_female': 34, 'controls_AF_oth_female': [0.0], 'controls_nhomalt_oth_female': [0], 'controls_AC_asj': [0], 'controls_AN_asj': 14, 'controls_AF_asj': [0.0], 'controls_nhomalt_asj': [0], 'non_neuro_AC_amr_male': [0], 'non_neuro_AN_amr_male': 364, 'non_neuro_AF_amr_male': [0.0], 'non_neuro_nhomalt_amr_male': [0], 'controls_AC_nfe_nwe': [0], 'controls_AN_nfe_nwe': 294, 'controls_AF_nfe_nwe': [0.0], 'controls_nhomalt_nfe_nwe': [0], 'AC_nfe_nwe': [0], 'AN_nfe_nwe': 672, 'AF_nfe_nwe': [0.0], 'nhomalt_nfe_nwe': [0], 'controls_AC_nfe_seu': [0], 'controls_AN_nfe_seu': 0, 'controls_nhomalt_nfe_seu': [0], 'controls_AC_sas_female': [0], 'controls_AN_sas_female': 188, 'controls_AF_sas_female': [0.0], 'controls_nhomalt_sas_female': [0], 'non_neuro_AC_amr_female': [2], 'non_neuro_AN_amr_female': 344, 'non_neuro_AF_amr_female': [0.00581395], 'non_neuro_nhomalt_amr_female': [0], 'non_cancer_AC_eas_jpn': [0], 'non_cancer_AN_eas_jpn': 0, 'non_cancer_nhomalt_eas_jpn': [0], 'non_neuro_AC_nfe_onf': [0], 'non_neuro_AN_nfe_onf': 146, 'non_neuro_AF_nfe_onf': [0.0], 'non_neuro_nhomalt_nfe_onf': [0], 'non_topmed_AC_eas_male': [0], 'non_topmed_AN_eas_male': 278, 'non_topmed_AF_eas_male': [0.0], 'non_topmed_nhomalt_eas_male': [0], 'AC_eas_jpn': [0], 'AN_eas_jpn': 0, 'nhomalt_eas_jpn': [0], 'non_cancer_AC_afr_male': [0], 'non_cancer_AN_afr_male': 262, 'non_cancer_AF_afr_male': [0.0], 'non_cancer_nhomalt_afr_male': [0], 'non_cancer_AC_afr': [0], 'non_cancer_AN_afr': 624, 'non_cancer_AF_afr': [0.0], 'non_cancer_nhomalt_afr': [0], 'controls_AC_amr_female': [0], 'controls_AN_amr_female': 154, 'controls_AF_amr_female': [0.0], 'controls_nhomalt_amr_female': [0], 'non_neuro_AC_fin_male': [0], 'non_neuro_AN_fin_male': 22, 'non_neuro_AF_fin_male': [0.0], 'non_neuro_nhomalt_fin_male': [0], 'AC_female': [2], 'AN_female': 1850, 'AF_female': [0.00108108], 'nhomalt_female': [0], 'non_neuro_AC_nfe_bgr': [0], 'non_neuro_AN_nfe_bgr': 0, 'non_neuro_nhomalt_nfe_bgr': [0], 'non_neuro_AC_oth_male': [0], 'non_neuro_AN_oth_male': 68, 'non_neuro_AF_oth_male': [0.0], 'non_neuro_nhomalt_oth_male': [0], 'non_topmed_AC_nfe_est': [0], 'non_topmed_AN_nfe_est': 4, 'non_topmed_AF_nfe_est': [0.0], 'non_topmed_nhomalt_nfe_est': [0], 'non_topmed_AC_nfe_nwe': [0], 'non_topmed_AN_nfe_nwe': 592, 'non_topmed_AF_nfe_nwe': [0.0], 'non_topmed_nhomalt_nfe_nwe': [0], 'non_topmed_AC_amr_male': [0], 'non_topmed_AN_amr_male': 364, 'non_topmed_AF_amr_male': [0.0], 'non_topmed_nhomalt_amr_male': [0], 'non_cancer_AC_amr': [2], 'non_cancer_AN_amr': 716, 'non_cancer_AF_amr': [0.0027933], 'non_cancer_nhomalt_amr': [0], 'non_topmed_AC_nfe_swe': [0], 'non_topmed_AN_nfe_swe': 0, 'non_topmed_nhomalt_nfe_swe': [0], 'non_topmed_AC_nfe_onf': [0], 'non_topmed_AN_nfe_onf': 134, 'non_topmed_AF_nfe_onf': [0.0], 'non_topmed_nhomalt_nfe_onf': [0], 'controls_AC_eas_kor': [0], 'controls_AN_eas_kor': 158, 'controls_AF_eas_kor': [0.0], 'controls_nhomalt_eas_kor': [0], 'non_topmed_AC_eas_oea': [0], 'non_topmed_AN_eas_oea': 238, 'non_topmed_AF_eas_oea': [0.0], 'non_topmed_nhomalt_eas_oea': [0], 'controls_AC_eas_male': [0], 'controls_AN_eas_male': 130, 'controls_AF_eas_male': [0.0], 'controls_nhomalt_eas_male': [0], 'controls_AC_oth_male': [0], 'controls_AN_oth_male': 32, 'controls_AF_oth_male': [0.0], 'controls_nhomalt_oth_male': [0], 'non_topmed_AC': [2], 'non_topmed_AN': 4050, 'non_topmed_AF': [0.000493827], 'non_topmed_nhomalt': [0], 'controls_AC_fin': [0], 'controls_AN_fin': 0, 'controls_nhomalt_fin': [0], 'AC_eas_kor': [0], 'AN_eas_kor': 318, 'AF_eas_kor': [0.0], 'nhomalt_eas_kor': [0], 'non_neuro_AC_nfe': [0], 'non_neuro_AN_nfe': 692, 'non_neuro_AF_nfe': [0.0], 'non_neuro_nhomalt_nfe': [0], 'non_neuro_AC_fin_female': [0], 'non_neuro_AN_fin_female': 14, 'non_neuro_AF_fin_female': [0.0], 'non_neuro_nhomalt_fin_female': [0], 'non_cancer_AC_nfe_male': [0], 'non_cancer_AN_nfe_male': 428, 'non_cancer_AF_nfe_male': [0.0], 'non_cancer_nhomalt_nfe_male': [0], 'controls_AC_eas_oea': [0], 'controls_AN_eas_oea': 88, 'controls_AF_eas_oea': [0.0], 'controls_nhomalt_eas_oea': [0], 'non_topmed_AC_nfe_seu': [0], 'non_topmed_AN_nfe_seu': 0, 'non_topmed_nhomalt_nfe_seu': [0], 'controls_AC_eas_female': [0], 'controls_AN_eas_female': 116, 'controls_AF_eas_female': [0.0], 'controls_nhomalt_eas_female': [0], 'non_topmed_AC_asj': [0], 'non_topmed_AN_asj': 40, 'non_topmed_AF_asj': [0.0], 'non_topmed_nhomalt_asj': [0], 'controls_AC_nfe_onf': [0], 'controls_AN_nfe_onf': 84, 'controls_AF_nfe_onf': [0.0], 'controls_nhomalt_nfe_onf': [0], 'non_neuro_AC': [2], 'non_neuro_AN': 3998, 'non_neuro_AF': [0.00050025], 'non_neuro_nhomalt': [0], 'AC_eas_oea': [0], 'AN_eas_oea': 238, 'AF_eas_oea': [0.0], 'nhomalt_eas_oea': [0], 'non_topmed_AC_nfe': [0], 'non_topmed_AN_nfe': 730, 'non_topmed_AF_nfe': [0.0], 'non_topmed_nhomalt_nfe': [0], 'non_cancer_AC_oth': [0], 'non_cancer_AN_oth': 140, 'non_cancer_AF_oth': [0.0], 'non_cancer_nhomalt_oth': [0], 'non_topmed_AC_raw': [8], 'non_topmed_AN_raw': 45148, 'non_topmed_AF_raw': [0.000177195], 'non_topmed_nhomalt_raw': [0], 'non_neuro_AC_nfe_est': [0], 'non_neuro_AN_nfe_est': 6, 'non_neuro_AF_nfe_est': [0.0], 'non_neuro_nhomalt_nfe_est': [0], 'non_topmed_AC_oth_male': [0], 'non_topmed_AN_oth_male': 72, 'non_topmed_AF_oth_male': [0.0], 'non_topmed_nhomalt_oth_male': [0], 'non_cancer_AC_oth_male': [0], 'non_cancer_AN_oth_male': 76, 'non_cancer_AF_oth_male': [0.0], 'non_cancer_nhomalt_oth_male': [0], 'AC_nfe_est': [0], 'AN_nfe_est': 6, 'AF_nfe_est': [0.0], 'nhomalt_nfe_est': [0], 'non_cancer_AC_afr_female': [0], 'non_cancer_AN_afr_female': 362, 'non_cancer_AF_afr_female': [0.0], 'non_cancer_nhomalt_afr_female': [0], 'non_topmed_AC_afr_male': [0], 'non_topmed_AN_afr_male': 256, 'non_topmed_AF_afr_male': [0.0], 'non_topmed_nhomalt_afr_male': [0], 'AC_eas_male': [0], 'AN_eas_male': 278, 'AF_eas_male': [0.0], 'nhomalt_eas_male': [0], 'controls_AC_eas': [0], 'controls_AN_eas': 246, 'controls_AF_eas': [0.0], 'controls_nhomalt_eas': [0], 'non_neuro_AC_eas_male': [0], 'non_neuro_AN_eas_male': 278, 'non_neuro_AF_eas_male': [0.0], 'non_neuro_nhomalt_eas_male': [0], 'non_cancer_AC_nfe_nwe': [0], 'non_cancer_AN_nfe_nwe': 672, 'non_cancer_AF_nfe_nwe': [0.0], 'non_cancer_nhomalt_nfe_nwe': [0], 'controls_AC_sas': [0], 'controls_AN_sas': 614, 'controls_AF_sas': [0.0], 'controls_nhomalt_sas': [0], 'non_neuro_AC_sas_male': [0], 'non_neuro_AN_sas_male': 894, 'non_neuro_AF_sas_male': [0.0], 'non_neuro_nhomalt_sas_male': [0], 'non_neuro_AC_asj_male': [0], 'non_neuro_AN_asj_male': 16, 'non_neuro_AF_asj_male': [0.0], 'non_neuro_nhomalt_asj_male': [0], 'non_cancer_AC_nfe_bgr': [0], 'non_cancer_AN_nfe_bgr': 0, 'non_cancer_nhomalt_nfe_bgr': [0], 'controls_AC_oth': [0], 'controls_AN_oth': 66, 'controls_AF_oth': [0.0], 'controls_nhomalt_oth': [0], 'non_cancer_AC_eas_female': [0], 'non_cancer_AN_eas_female': 278, 'non_cancer_AF_eas_female': [0.0], 'non_cancer_nhomalt_eas_female': [0], 'AC_nfe': [0], 'AN_nfe': 844, 'AF_nfe': [0.0], 'nhomalt_nfe': [0], 'non_topmed_AC_female': [2], 'non_topmed_AN_female': 1774, 'non_topmed_AF_female': [0.0011274], 'non_topmed_nhomalt_female': [0], 'non_neuro_AC_asj': [0], 'non_neuro_AN_asj': 20, 'non_neuro_AF_asj': [0.0], 'non_neuro_nhomalt_asj': [0], 'non_topmed_AC_eas_female': [0], 'non_topmed_AN_eas_female': 278, 'non_topmed_AF_eas_female': [0.0], 'non_topmed_nhomalt_eas_female': [0], 'non_neuro_AC_raw': [7], 'non_neuro_AN_raw': 39484, 'non_neuro_AF_raw': [0.000177287], 'non_neuro_nhomalt_raw': [0], 'non_topmed_AC_eas': [0], 'non_topmed_AN_eas': 556, 'non_topmed_AF_eas': [0.0], 'non_topmed_nhomalt_eas': [0], 'non_topmed_AC_fin_male': [0], 'non_topmed_AN_fin_male': 22, 'non_topmed_AF_fin_male': [0.0], 'non_topmed_nhomalt_fin_male': [0], 'non_cancer_AC_asj_male': [0], 'non_cancer_AN_asj_male': 18, 'non_cancer_AF_asj_male': [0.0], 'non_cancer_nhomalt_asj_male': [0], 'AC_fin': [0], 'AN_fin': 36, 'AF_fin': [0.0], 'nhomalt_fin': [0], 'AC_nfe_male': [0], 'AN_nfe_male': 428, 'AF_nfe_male': [0.0], 'nhomalt_nfe_male': [0], 'non_topmed_AC_eas_kor': [0], 'non_topmed_AN_eas_kor': 318, 'non_topmed_AF_eas_kor': [0.0], 'non_topmed_nhomalt_eas_kor': [0], 'controls_AC_amr_male': [0], 'controls_AN_amr_male': 172, 'controls_AF_amr_male': [0.0], 'controls_nhomalt_amr_male': [0], 'non_neuro_AC_eas_oea': [0], 'non_neuro_AN_eas_oea': 238, 'non_neuro_AF_eas_oea': [0.0], 'non_neuro_nhomalt_eas_oea': [0], 'AC_sas_female': [0], 'AN_sas_female': 344, 'AF_sas_female': [0.0], 'nhomalt_sas_female': [0], 'controls_AC_afr_female': [0], 'controls_AN_afr_female': 184, 'controls_AF_afr_female': [0.0], 'controls_nhomalt_afr_female': [0], 'controls_AC_amr': [0], 'controls_AN_amr': 326, 'controls_AF_amr': [0.0], 'controls_nhomalt_amr': [0], 'non_topmed_AC_eas_jpn': [0], 'non_topmed_AN_eas_jpn': 0, 'non_topmed_nhomalt_eas_jpn': [0], 'AC_asj_female': [0], 'AN_asj_female': 24, 'AF_asj_female': [0.0], 'nhomalt_asj_female': [0], 'non_topmed_AC_nfe_bgr': [0], 'non_topmed_AN_nfe_bgr': 0, 'non_topmed_nhomalt_nfe_bgr': [0], 'non_cancer_AC_nfe_est': [0], 'non_cancer_AN_nfe_est': 6, 'non_cancer_AF_nfe_est': [0.0], 'non_cancer_nhomalt_nfe_est': [0], 'non_neuro_AC_eas': [0], 'non_neuro_AN_eas': 556, 'non_neuro_AF_eas': [0.0], 'non_neuro_nhomalt_eas': [0], 'non_cancer_AC_nfe': [0], 'non_cancer_AN_nfe': 844, 'non_cancer_AF_nfe': [0.0], 'non_cancer_nhomalt_nfe': [0], 'non_neuro_AC_male': [0], 'non_neuro_AN_male': 2230, 'non_neuro_AF_male': [0.0], 'non_neuro_nhomalt_male': [0], 'non_neuro_AC_sas_female': [0], 'non_neuro_AN_sas_female': 344, 'non_neuro_AF_sas_female': [0.0], 'non_neuro_nhomalt_sas_female': [0], 'AC_asj': [0], 'AN_asj': 42, 'AF_asj': [0.0], 'nhomalt_asj': [0], 'controls_AC_nfe_est': [0], 'controls_AN_nfe_est': 2, 'controls_AF_nfe_est': [0.0], 'controls_nhomalt_nfe_est': [0], 'non_topmed_AC_asj_female': [0], 'non_topmed_AN_asj_female': 22, 'non_topmed_AF_asj_female': [0.0], 'non_topmed_nhomalt_asj_female': [0], 'non_cancer_AC_nfe_swe': [0], 'non_cancer_AN_nfe_swe': 0, 'non_cancer_nhomalt_nfe_swe': [0], 'non_cancer_AC': [2], 'non_cancer_AN': 4196, 'non_cancer_AF': [0.000476644], 'non_cancer_nhomalt': [0], 'non_topmed_AC_oth': [0], 'non_topmed_AN_oth': 134, 'non_topmed_AF_oth': [0.0], 'non_topmed_nhomalt_oth': [0], 'non_topmed_AC_fin_female': [0], 'non_topmed_AN_fin_female': 14, 'non_topmed_AF_fin_female': [0.0], 'non_topmed_nhomalt_fin_female': [0], 'non_cancer_AC_fin_female': [0], 'non_cancer_AN_fin_female': 14, 'non_cancer_AF_fin_female': [0.0], 'non_cancer_nhomalt_fin_female': [0], 'AC_oth': [0], 'AN_oth': 140, 'AF_oth': [0.0], 'nhomalt_oth': [0], 'non_neuro_AC_nfe_male': [0], 'non_neuro_AN_nfe_male': 328, 'non_neuro_AF_nfe_male': [0.0], 'non_neuro_nhomalt_nfe_male': [0], 'controls_AC_female': [0], 'controls_AN_female': 876, 'controls_AF_female': [0.0], 'controls_nhomalt_female': [0], 'non_cancer_AC_fin': [0], 'non_cancer_AN_fin': 36, 'non_cancer_AF_fin': [0.0], 'non_cancer_nhomalt_fin': [0], 'non_topmed_AC_fin': [0], 'non_topmed_AN_fin': 36, 'non_topmed_AF_fin': [0.0], 'non_topmed_nhomalt_fin': [0], 'non_cancer_AC_eas_oea': [0], 'non_cancer_AN_eas_oea': 238, 'non_cancer_AF_eas_oea': [0.0], 'non_cancer_nhomalt_eas_oea': [0], 'non_topmed_AC_nfe_female': [0], 'non_topmed_AN_nfe_female': 358, 'non_topmed_AF_nfe_female': [0.0], 'non_topmed_nhomalt_nfe_female': [0], 'non_cancer_AC_sas_male': [0], 'non_cancer_AN_sas_male': 894, 'non_cancer_AF_sas_male': [0.0], 'non_cancer_nhomalt_sas_male': [0], 'controls_AC_asj_male': [0], 'controls_AN_asj_male': 12, 'controls_AF_asj_male': [0.0], 'controls_nhomalt_asj_male': [0], 'non_cancer_AC_raw': [8], 'non_cancer_AN_raw': 46350, 'non_cancer_AF_raw': [0.0001726], 'non_cancer_nhomalt_raw': [0], 'non_cancer_AC_eas_male': [0], 'non_cancer_AN_eas_male': 278, 'non_cancer_AF_eas_male': [0.0], 'non_cancer_nhomalt_eas_male': [0], 'non_topmed_AC_asj_male': [0], 'non_topmed_AN_asj_male': 18, 'non_topmed_AF_asj_male': [0.0], 'non_topmed_nhomalt_asj_male': [0], 'non_neuro_AC_oth': [0], 'non_neuro_AN_oth': 128, 'non_neuro_AF_oth': [0.0], 'non_neuro_nhomalt_oth': [0], 'AC_male': [0], 'AN_male': 2346, 'AF_male': [0.0], 'nhomalt_male': [0], 'controls_AC_fin_female': [0], 'controls_AN_fin_female': 0, 'controls_nhomalt_fin_female': [0], 'controls_AC_nfe_bgr': [0], 'controls_AN_nfe_bgr': 0, 'controls_nhomalt_nfe_bgr': [0], 'controls_AC_asj_female': [0], 'controls_AN_asj_female': 2, 'controls_AF_asj_female': [0.0], 'controls_nhomalt_asj_female': [0], 'AC_amr_male': [0], 'AN_amr_male': 368, 'AF_amr_male': [0.0], 'nhomalt_amr_male': [0], 'AC_amr_female': [2], 'AN_amr_female': 348, 'AF_amr_female': [0.00574713], 'nhomalt_amr_female': [0], 'non_topmed_AC_sas_male': [0], 'non_topmed_AN_sas_male': 894, 'non_topmed_AF_sas_male': [0.0], 'non_topmed_nhomalt_sas_male': [0], 'AC_oth_male': [0], 'AN_oth_male': 76, 'AF_oth_male': [0.0], 'nhomalt_oth_male': [0], 'non_cancer_AC_sas': [0], 'non_cancer_AN_sas': 1238, 'non_cancer_AF_sas': [0.0], 'non_cancer_nhomalt_sas': [0], 'non_neuro_AC_nfe_seu': [0], 'non_neuro_AN_nfe_seu': 0, 'non_neuro_nhomalt_nfe_seu': [0], 'non_cancer_AC_eas_kor': [0], 'non_cancer_AN_eas_kor': 318, 'non_cancer_AF_eas_kor': [0.0], 'non_cancer_nhomalt_eas_kor': [0], 'non_topmed_AC_afr_female': [0], 'non_topmed_AN_afr_female': 348, 'non_topmed_AF_afr_female': [0.0], 'non_topmed_nhomalt_afr_female': [0], 'controls_AC_sas_male': [0], 'controls_AN_sas_male': 426, 'controls_AF_sas_male': [0.0], 'controls_nhomalt_sas_male': [0], 'non_topmed_AC_sas_female': [0], 'non_topmed_AN_sas_female': 344, 'non_topmed_AF_sas_female': [0.0], 'non_topmed_nhomalt_sas_female': [0], 'non_topmed_AC_afr': [0], 'non_topmed_AN_afr': 604, 'non_topmed_AF_afr': [0.0], 'non_topmed_nhomalt_afr': [0], 'controls_AC': [0], 'controls_AN': 1950, 'controls_AF': [0.0], 'controls_nhomalt': [0], 'non_neuro_AC_oth_female': [0], 'non_neuro_AN_oth_female': 60, 'non_neuro_AF_oth_female': [0.0], 'non_neuro_nhomalt_oth_female': [0], 'non_topmed_faf95_amr': [0.00049863], 'non_topmed_faf99_amr': [0.00049823], 'faf95_afr': [0.0], 'faf99_afr': [0.0], 'faf95_sas': [0.0], 'faf99_sas': [0.0], 'controls_faf95_afr': [0.0], 'controls_faf99_afr': [0.0], 'faf95_amr': [0.00049593], 'faf99_amr': [0.00049537], 'non_neuro_faf95_sas': [0.0], 'non_neuro_faf99_sas': [0.0], 'faf95_eas': [0.0], 'faf99_eas': [0.0], 'faf95': [8.404e-05], 'faf99': [8.405e-05], 'non_neuro_faf95_afr': [0.0], 'non_neuro_faf99_afr': [0.0], 'non_cancer_faf95_eas': [0.0], 'non_cancer_faf99_eas': [0.0], 'non_neuro_faf95_amr': [0.00050106], 'non_neuro_faf99_amr': [0.00050122], 'non_topmed_faf95_sas': [0.0], 'non_topmed_faf99_sas': [0.0], 'controls_faf95_nfe': [0.0], 'controls_faf99_nfe': [0.0], 'non_cancer_faf95_afr': [0.0], 'non_cancer_faf99_afr': [0.0], 'non_cancer_faf95_amr': [0.00049593], 'non_cancer_faf99_amr': [0.00049537], 'non_topmed_faf95': [8.683e-05], 'non_topmed_faf99': [8.708e-05], 'non_neuro_faf95_nfe': [0.0], 'non_neuro_faf99_nfe': [0.0], 'non_neuro_faf95': [8.887e-05], 'non_neuro_faf99': [8.847e-05], 'non_topmed_faf95_nfe': [0.0], 'non_topmed_faf99_nfe': [0.0], 'controls_faf95_eas': [0.0], 'controls_faf99_eas': [0.0], 'controls_faf95_sas': [0.0], 'controls_faf99_sas': [0.0], 'faf95_nfe': [0.0], 'faf99_nfe': [0.0], 'non_topmed_faf95_eas': [0.0], 'non_topmed_faf99_eas': [0.0], 'controls_faf95_amr': [0.0], 'controls_faf99_amr': [0.0], 'non_neuro_faf95_eas': [0.0], 'non_neuro_faf99_eas': [0.0], 'non_cancer_faf95_nfe': [0.0], 'non_cancer_faf99_nfe': [0.0], 'non_cancer_faf95': [8.404e-05], 'non_cancer_faf99': [8.405e-05], 'non_cancer_faf95_sas': [0.0], 'non_cancer_faf99_sas': [0.0], 'non_topmed_faf95_afr': [0.0], 'non_topmed_faf99_afr': [0.0], 'controls_faf95': [0.0], 'controls_faf99': [0.0], 'popmax': ['amr'], 'AC_popmax': [2], 'AN_popmax': [716], 'AF_popmax': [0.0027933], 'nhomalt_popmax': [0], 'age_hist_het_bin_freq': ['0|0|0|0|0|1|1|0|0|0'], 'age_hist_het_n_smaller': [0], 'age_hist_het_n_larger': [0], 'age_hist_hom_bin_freq': ['0|0|0|0|0|0|0|0|0|0'], 'age_hist_hom_n_smaller': [0], 'age_hist_hom_n_larger': [0], 'non_topmed_popmax': ['amr'], 'non_topmed_AC_popmax': [2], 'non_topmed_AN_popmax': [712], 'non_topmed_AF_popmax': [0.00280899], 'non_topmed_nhomalt_popmax': [0], 'non_neuro_popmax': ['amr'], 'non_neuro_AC_popmax': [2], 'non_neuro_AN_popmax': [708], 'non_neuro_AF_popmax': [0.00282486], 'non_neuro_nhomalt_popmax': [0], 'non_cancer_popmax': ['amr'], 'non_cancer_AC_popmax': [2], 'non_cancer_AN_popmax': [716], 'non_cancer_AF_popmax': [0.0027933], 'non_cancer_nhomalt_popmax': [0], 'vep': ['C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000423562|unprocessed_pseudogene|||||||||||1|1790|-1||SNV|1|HGNC|38034||||||||||||||||||||||||||||||||||||||||||', 'C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000438504|unprocessed_pseudogene|||||||||||1|1790|-1||SNV|1|HGNC|38034|YES|||||||||||||||||||||||||||||||||||||||||', 'C|intron_variant&non_coding_transcript_variant|MODIFIER|DDX11L1|ENSG00000223972|Transcript|ENST00000450305|transcribed_unprocessed_pseudogene||2/5|ENST00000450305.2:n.98-40T>C||||||||1||1||SNV|1|HGNC|37102||||||||||||||||||||||||||||||||||||||||||', 'C|intron_variant&non_coding_transcript_variant|MODIFIER|DDX11L1|ENSG00000223972|Transcript|ENST00000456328|processed_transcript||1/2|ENST00000456328.2:n.360-40T>C||||||||1||1||SNV|1|HGNC|37102|YES|||||||||||||||||||||||||||||||||||||||||', 'C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000488147|unprocessed_pseudogene|||||||||||1|1831|-1||SNV|1|HGNC|38034||||||||||||||||||||||||||||||||||||||||||', 'C|intron_variant&non_coding_transcript_variant|MODIFIER|DDX11L1|ENSG00000223972|Transcript|ENST00000515242|transcribed_unprocessed_pseudogene||1/2|ENST00000515242.2:n.357-40T>C||||||||1||1||SNV|1|HGNC|37102||||||||||||||||||||||||||||||||||||||||||', 'C|intron_variant&non_coding_transcript_variant|MODIFIER|DDX11L1|ENSG00000223972|Transcript|ENST00000518655|transcribed_unprocessed_pseudogene||1/3|ENST00000518655.2:n.355-22T>C||||||||1||1||SNV|1|HGNC|37102||||||||||||||||||||||||||||||||||||||||||', 'C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000538476|unprocessed_pseudogene|||||||||||1|1838|-1||SNV|1|HGNC|38034||||||||||||||||||||||||||||||||||||||||||', 'C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000541675|unprocessed_pseudogene|||||||||||1|1790|-1||SNV|1|HGNC|38034||||||||||||||||||||||||||||||||||||||||||', 'C|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00001576075|CTCF_binding_site|||||||||||1||||SNV|1||||||||||||||||||||||||||||||||||||||||||||']}, [], [])"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a = next(reader)\n",
    "a"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Consequence annotations from Ensembl VEP. Format: Allele|Consequence|IMPACT|SYMBOL|Gene|Feature_type|Feature|BIOTYPE|EXON|INTRON|HGVSc|HGVSp|cDNA_position|CDS_position|Protein_position|Amino_acids|Codons|Existing_variation|ALLELE_NUM|DISTANCE|STRAND|FLAGS|VARIANT_CLASS|MINIMISED|SYMBOL_SOURCE|HGNC_ID|CANONICAL|TSL|APPRIS|CCDS|ENSP|SWISSPROT|TREMBL|UNIPARC|GENE_PHENO|SIFT|PolyPhen|DOMAINS|HGVS_OFFSET|GMAF|AFR_MAF|AMR_MAF|EAS_MAF|EUR_MAF|SAS_MAF|AA_MAF|EA_MAF|ExAC_MAF|ExAC_Adj_MAF|ExAC_AFR_MAF|ExAC_AMR_MAF|ExAC_EAS_MAF|ExAC_FIN_MAF|ExAC_NFE_MAF|ExAC_OTH_MAF|ExAC_SAS_MAF|CLIN_SIG|SOMATIC|PHENO|PUBMED|MOTIF_NAME|MOTIF_POS|HIGH_INF_POS|MOTIF_SCORE_CHANGE|LoF|LoF_filter|LoF_flags|LoF_info'"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# reader.header.get_format_field_info(\"vep\")\n",
    "reader.header.get_info_field_info(\"vep\").description"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000423562|unprocessed_pseudogene|||||||||||1|1790|-1||SNV|1|HGNC|38034||||||||||||||||||||||||||||||||||||||||||',\n",
       " 'C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000438504|unprocessed_pseudogene|||||||||||1|1790|-1||SNV|1|HGNC|38034|YES|||||||||||||||||||||||||||||||||||||||||',\n",
       " 'C|intron_variant&non_coding_transcript_variant|MODIFIER|DDX11L1|ENSG00000223972|Transcript|ENST00000450305|transcribed_unprocessed_pseudogene||2/5|ENST00000450305.2:n.98-40T>C||||||||1||1||SNV|1|HGNC|37102||||||||||||||||||||||||||||||||||||||||||',\n",
       " 'C|intron_variant&non_coding_transcript_variant|MODIFIER|DDX11L1|ENSG00000223972|Transcript|ENST00000456328|processed_transcript||1/2|ENST00000456328.2:n.360-40T>C||||||||1||1||SNV|1|HGNC|37102|YES|||||||||||||||||||||||||||||||||||||||||',\n",
       " 'C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000488147|unprocessed_pseudogene|||||||||||1|1831|-1||SNV|1|HGNC|38034||||||||||||||||||||||||||||||||||||||||||',\n",
       " 'C|intron_variant&non_coding_transcript_variant|MODIFIER|DDX11L1|ENSG00000223972|Transcript|ENST00000515242|transcribed_unprocessed_pseudogene||1/2|ENST00000515242.2:n.357-40T>C||||||||1||1||SNV|1|HGNC|37102||||||||||||||||||||||||||||||||||||||||||',\n",
       " 'C|intron_variant&non_coding_transcript_variant|MODIFIER|DDX11L1|ENSG00000223972|Transcript|ENST00000518655|transcribed_unprocessed_pseudogene||1/3|ENST00000518655.2:n.355-22T>C||||||||1||1||SNV|1|HGNC|37102||||||||||||||||||||||||||||||||||||||||||',\n",
       " 'C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000538476|unprocessed_pseudogene|||||||||||1|1838|-1||SNV|1|HGNC|38034||||||||||||||||||||||||||||||||||||||||||',\n",
       " 'C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000541675|unprocessed_pseudogene|||||||||||1|1790|-1||SNV|1|HGNC|38034||||||||||||||||||||||||||||||||||||||||||',\n",
       " 'C|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00001576075|CTCF_binding_site|||||||||||1||||SNV|1||||||||||||||||||||||||||||||||||||||||||||']"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a.INFO[\"vep\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CHROM\tPOS\tREF\tALT\n"
     ]
    }
   ],
   "source": [
    "header = ['CHROM', 'POS', 'REF', 'ALT'] + reader.header.samples.names\n",
    "print('\\t'.join(header))\n",
    "\n",
    "for record in reader:\n",
    "    if length:= len(record.INFO[\"vep\"]) ==1:\n",
    "        print(length)\n",
    "    # if not record.is_snv():\n",
    "    #     continue\n",
    "    line = [record.CHROM, record.POS, record.REF]\n",
    "    line += [alt.value for alt in record.ALT]\n",
    "    line += [call.data.get('GT') or './.' for call in record.calls]\n",
    "    print('\\t'.join(map(str, line)))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "def head(file_dir, n, is_strip:bool = True):\n",
    "    \"\"\"head a file at n rows. like head -n in bash, will return a list of row, if strip, each row will do row.strip(strip)\n",
    "\n",
    "    Args:\n",
    "        file_dir (_type_): _description_\n",
    "        n (_type_): _description_\n",
    "        strip (Union[str, None], optional): _description_. Defaults to None.\n",
    "\n",
    "    Returns:\n",
    "        _type_: _description_\n",
    "    \"\"\"\n",
    "    with open(file_dir, \"r\") as f:\n",
    "        return [f.readline().strip() if is_strip else f.readline() for i in range(n)]\n",
    "\n",
    "            "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['##fileformat=VCFv4.2',\n",
       " '##hailversion=0.2.7-fff3c70bdaf7',\n",
       " '##FILTER=<ID=AC0,Description=\"Allele count is zero after filtering out low-confidence genotypes (GQ < 20; DP < 10; and AB < 0.2 for het calls)\">',\n",
       " '##FILTER=<ID=InbreedingCoeff,Description=\"InbreedingCoeff < -0.3\">',\n",
       " '##FILTER=<ID=PASS,Description=\"Passed all variant filters\">',\n",
       " '##FILTER=<ID=RF,Description=\"Failed random forest filtering thresholds of 0.055272738028512555, 0.20641025579497013 (probabilities of being a true positive variant) for SNPs, indels\">',\n",
       " '##INFO=<ID=AC,Number=A,Type=Integer,Description=\"Alternate allele count for samples\">',\n",
       " '##INFO=<ID=AN,Number=1,Type=Integer,Description=\"Total number of alleles in samples\">',\n",
       " '##INFO=<ID=AF,Number=A,Type=Float,Description=\"Alternate allele frequency in samples\">',\n",
       " '##INFO=<ID=rf_tp_probability,Number=1,Type=Float,Description=\"Random forest prediction probability for a site being a true variant\">',\n",
       " '##INFO=<ID=FS,Number=1,Type=Float,Description=\"Phred-scaled p-value of Fisher\\'s exact test for strand bias\">',\n",
       " '##INFO=<ID=InbreedingCoeff,Number=1,Type=Float,Description=\"Inbreeding coefficient as estimated from the genotype likelihoods per-sample when compared against the Hardy-Weinberg expectation\">',\n",
       " '##INFO=<ID=MQ,Number=1,Type=Float,Description=\"Root mean square of the mapping quality of reads across all samples\">',\n",
       " '##INFO=<ID=MQRankSum,Number=1,Type=Float,Description=\"Z-score from Wilcoxon rank sum test of alternate vs. reference read mapping qualities\">',\n",
       " '##INFO=<ID=QD,Number=1,Type=Float,Description=\"Variant call confidence normalized by depth of sample reads supporting a variant\">',\n",
       " '##INFO=<ID=ReadPosRankSum,Number=1,Type=Float,Description=\"Z-score from Wilcoxon rank sum test of alternate vs. reference read position bias\">',\n",
       " '##INFO=<ID=SOR,Number=1,Type=Float,Description=\"Strand bias estimated by the symmetric odds ratio test\">',\n",
       " '##INFO=<ID=VQSR_POSITIVE_TRAIN_SITE,Number=0,Type=Flag,Description=\"Variant was used to build the positive training set of high-quality variants for VQSR\">',\n",
       " '##INFO=<ID=VQSR_NEGATIVE_TRAIN_SITE,Number=0,Type=Flag,Description=\"Variant was used to build the negative training set of low-quality variants for VQSR\">',\n",
       " '##INFO=<ID=BaseQRankSum,Number=1,Type=Float,Description=\"Z-score from Wilcoxon rank sum test of alternate vs. reference base qualities\">',\n",
       " '##INFO=<ID=ClippingRankSum,Number=1,Type=Float,Description=\"Z-score from Wilcoxon rank sum test of alternate vs. reference number of hard clipped bases\">',\n",
       " '##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Depth of informative coverage for each sample; reads with MQ=255 or with bad mates are filtered\">',\n",
       " '##INFO=<ID=VQSLOD,Number=1,Type=Float,Description=\"Log-odds ratio of being a true variant versus being a false positive under the trained VQSR Gaussian mixture model\">',\n",
       " '##INFO=<ID=VQSR_culprit,Number=1,Type=String,Description=\"Worst-performing annotation in the VQSR Gaussian mixture model\">',\n",
       " '##INFO=<ID=segdup,Number=0,Type=Flag,Description=\"Variant falls within a segmental duplication region\">',\n",
       " '##INFO=<ID=lcr,Number=0,Type=Flag,Description=\"Variant falls within a low complexity region\">',\n",
       " '##INFO=<ID=decoy,Number=0,Type=Flag,Description=\"Variant falls within a reference decoy region\">',\n",
       " '##INFO=<ID=nonpar,Number=0,Type=Flag,Description=\"Variant (on sex chromosome) falls outside a pseudoautosomal region\">',\n",
       " '##INFO=<ID=rf_positive_label,Number=0,Type=Flag,Description=\"Variant was labelled as a positive example for training of random forest model\">',\n",
       " '##INFO=<ID=rf_negative_label,Number=0,Type=Flag,Description=\"Variant was labelled as a negative example for training of random forest model\">',\n",
       " '##INFO=<ID=rf_label,Number=1,Type=String,Description=\"Random forest training label\">',\n",
       " '##INFO=<ID=rf_train,Number=0,Type=Flag,Description=\"Variant was used in training random forest model\">',\n",
       " '##INFO=<ID=transmitted_singleton,Number=0,Type=Flag,Description=\"Variant was a callset-wide doubleton that was transmitted within a family (i.e., a singleton amongst unrelated sampes in cohort)\">',\n",
       " '##INFO=<ID=variant_type,Number=1,Type=String,Description=\"Variant type (snv, indel, multi-snv, multi-indel, or mixed)\">',\n",
       " '##INFO=<ID=allele_type,Number=A,Type=String,Description=\"Allele type (snv, ins, del, or mixed)\">',\n",
       " '##INFO=<ID=n_alt_alleles,Number=A,Type=Integer,Description=\"Total number of alternate alleles observed at variant locus\">',\n",
       " '##INFO=<ID=was_mixed,Number=0,Type=Flag,Description=\"Variant type was mixed\">',\n",
       " '##INFO=<ID=has_star,Number=0,Type=Flag,Description=\"Variant locus coincides with a spanning deletion (represented by a star) observed elsewhere in the callset\">',\n",
       " '##INFO=<ID=pab_max,Number=A,Type=Float,Description=\"Maximum p-value over callset for binomial test of observed allele balance for a heterozygous genotype, given expectation of AB=0.5\">',\n",
       " '##INFO=<ID=gq_hist_alt_bin_freq,Number=A,Type=String,Description=\"Histogram for GQ in heterozygous individuals; bin edges are: 0|5|10|15|20|25|30|35|40|45|50|55|60|65|70|75|80|85|90|95|100\">',\n",
       " '##INFO=<ID=gq_hist_all_bin_freq,Number=A,Type=String,Description=\"Histogram for GQ; bin edges are: 0|5|10|15|20|25|30|35|40|45|50|55|60|65|70|75|80|85|90|95|100\">',\n",
       " '##INFO=<ID=dp_hist_alt_bin_freq,Number=A,Type=String,Description=\"Histogram for DP in heterozygous individuals; bin edges are: 0|5|10|15|20|25|30|35|40|45|50|55|60|65|70|75|80|85|90|95|100\">',\n",
       " '##INFO=<ID=dp_hist_alt_n_larger,Number=A,Type=Integer,Description=\"Count of DP values falling above highest histogram bin edge\">',\n",
       " '##INFO=<ID=dp_hist_all_bin_freq,Number=A,Type=String,Description=\"Histogram for DP; bin edges are: 0|5|10|15|20|25|30|35|40|45|50|55|60|65|70|75|80|85|90|95|100\">',\n",
       " '##INFO=<ID=dp_hist_all_n_larger,Number=A,Type=Integer,Description=\"Count of DP values falling above highest histogram bin edge\">',\n",
       " '##INFO=<ID=ab_hist_alt_bin_freq,Number=A,Type=String,Description=\"Histogram for AB in heterozygous individuals; bin edges are: 0.00|0.05|0.10|0.15|0.20|0.25|0.30|0.35|0.40|0.45|0.50|0.55|0.60|0.65|0.70|0.75|0.80|0.85|0.90|0.95|1.00\">',\n",
       " '##INFO=<ID=AC_nfe_seu,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Southern European ancestry\">',\n",
       " '##INFO=<ID=AN_nfe_seu,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Southern European ancestry\">',\n",
       " '##INFO=<ID=AF_nfe_seu,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Southern European ancestry\">',\n",
       " '##INFO=<ID=nhomalt_nfe_seu,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Southern European ancestry\">',\n",
       " '##INFO=<ID=controls_AC_afr_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of African-American/African ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_afr_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of African-American/African ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_afr_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of African-American/African ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_afr_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of African-American/African ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_eas_kor,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Korean ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_eas_kor,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Korean ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_eas_kor,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Korean ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_eas_kor,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Korean ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_amr,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Latino ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_amr,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Latino ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_amr,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Latino ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_amr,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Latino ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_asj_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Ashkenazi Jewish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_asj_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Ashkenazi Jewish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_asj_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Ashkenazi Jewish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_asj_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Ashkenazi Jewish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=AC_raw,Number=A,Type=Integer,Description=\"Alternate allele count for samples, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=AN_raw,Number=1,Type=Integer,Description=\"Total number of alleles in samples, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=AF_raw,Number=A,Type=Float,Description=\"Alternate allele frequency in samples, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=nhomalt_raw,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=AC_fin_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Finnish ancestry\">',\n",
       " '##INFO=<ID=AN_fin_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Finnish ancestry\">',\n",
       " '##INFO=<ID=AF_fin_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Finnish ancestry\">',\n",
       " '##INFO=<ID=nhomalt_fin_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Finnish ancestry\">',\n",
       " '##INFO=<ID=non_cancer_AC_oth_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Other ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_oth_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Other ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_oth_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Other ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_oth_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Other ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=AC_nfe_bgr,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Bulgarian (Eastern European) ancestry\">',\n",
       " '##INFO=<ID=AN_nfe_bgr,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Bulgarian (Eastern European) ancestry\">',\n",
       " '##INFO=<ID=AF_nfe_bgr,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Bulgarian (Eastern European) ancestry\">',\n",
       " '##INFO=<ID=nhomalt_nfe_bgr,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Bulgarian (Eastern European) ancestry\">',\n",
       " '##INFO=<ID=non_neuro_AC_asj_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Ashkenazi Jewish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_asj_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Ashkenazi Jewish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_asj_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Ashkenazi Jewish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_asj_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Ashkenazi Jewish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=AC_sas_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of South Asian ancestry\">',\n",
       " '##INFO=<ID=AN_sas_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of South Asian ancestry\">',\n",
       " '##INFO=<ID=AF_sas_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of South Asian ancestry\">',\n",
       " '##INFO=<ID=nhomalt_sas_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of South Asian ancestry\">',\n",
       " '##INFO=<ID=non_neuro_AC_afr_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of African-American/African ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_afr_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of African-American/African ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_afr_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of African-American/African ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_afr_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of African-American/African ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=AC_afr_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of African-American/African ancestry\">',\n",
       " '##INFO=<ID=AN_afr_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of African-American/African ancestry\">',\n",
       " '##INFO=<ID=AF_afr_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of African-American/African ancestry\">',\n",
       " '##INFO=<ID=nhomalt_afr_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of African-American/African ancestry\">',\n",
       " '##INFO=<ID=AC_afr,Number=A,Type=Integer,Description=\"Alternate allele count for samples of African-American/African ancestry\">',\n",
       " '##INFO=<ID=AN_afr,Number=1,Type=Integer,Description=\"Total number of alleles in samples of African-American/African ancestry\">',\n",
       " '##INFO=<ID=AF_afr,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of African-American/African ancestry\">',\n",
       " '##INFO=<ID=nhomalt_afr,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of African-American/African ancestry\">',\n",
       " '##INFO=<ID=controls_AC_nfe_swe,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Swedish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_nfe_swe,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Swedish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_nfe_swe,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Swedish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_nfe_swe,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Swedish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_afr_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of African-American/African ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_afr_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of African-American/African ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_afr_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of African-American/African ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_afr_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of African-American/African ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_amr_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Latino ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_amr_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Latino ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_amr_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Latino ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_amr_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Latino ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_nfe_onf,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Other Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_nfe_onf,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Other Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_nfe_onf,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Other Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_nfe_onf,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Other Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_oth_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Other ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_oth_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Other ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_oth_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Other ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_oth_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Other ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=AC_eas_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of East Asian ancestry\">',\n",
       " '##INFO=<ID=AN_eas_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of East Asian ancestry\">',\n",
       " '##INFO=<ID=AF_eas_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of East Asian ancestry\">',\n",
       " '##INFO=<ID=nhomalt_eas_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of East Asian ancestry\">',\n",
       " '##INFO=<ID=non_cancer_AC_sas_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of South Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_sas_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of South Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_sas_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of South Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_sas_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of South Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=AC_afr_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of African-American/African ancestry\">',\n",
       " '##INFO=<ID=AN_afr_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of African-American/African ancestry\">',\n",
       " '##INFO=<ID=AF_afr_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of African-American/African ancestry\">',\n",
       " '##INFO=<ID=nhomalt_afr_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of African-American/African ancestry\">',\n",
       " '##INFO=<ID=AC_sas,Number=A,Type=Integer,Description=\"Alternate allele count for samples of South Asian ancestry\">',\n",
       " '##INFO=<ID=AN_sas,Number=1,Type=Integer,Description=\"Total number of alleles in samples of South Asian ancestry\">',\n",
       " '##INFO=<ID=AF_sas,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of South Asian ancestry\">',\n",
       " '##INFO=<ID=nhomalt_sas,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of South Asian ancestry\">',\n",
       " '##INFO=<ID=non_neuro_AC_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples in the non_neuro subset\">',\n",
       " '##INFO=<ID=controls_AC_afr,Number=A,Type=Integer,Description=\"Alternate allele count for samples of African-American/African ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_afr,Number=1,Type=Integer,Description=\"Total number of alleles in samples of African-American/African ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_afr,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of African-American/African ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_afr,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of African-American/African ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_eas_jpn,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Japanese ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_eas_jpn,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Japanese ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_eas_jpn,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Japanese ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_eas_jpn,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Japanese ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=AC_nfe_onf,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Other Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=AN_nfe_onf,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Other Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=AF_nfe_onf,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Other Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=nhomalt_nfe_onf,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Other Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=non_cancer_AC_amr_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Latino ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_amr_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Latino ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_amr_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Latino ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_amr_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Latino ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=controls_AC_fin_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Finnish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_fin_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Finnish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_fin_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Finnish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_fin_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Finnish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_nfe_nwe,Number=A,Type=Integer,Description=\"Alternate allele count for samples of North-Western European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_nfe_nwe,Number=1,Type=Integer,Description=\"Total number of alleles in samples of North-Western European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_nfe_nwe,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of North-Western European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_nfe_nwe,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of North-Western European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=AC_fin_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Finnish ancestry\">',\n",
       " '##INFO=<ID=AN_fin_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Finnish ancestry\">',\n",
       " '##INFO=<ID=AF_fin_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Finnish ancestry\">',\n",
       " '##INFO=<ID=nhomalt_fin_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Finnish ancestry\">',\n",
       " '##INFO=<ID=AC_nfe_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=AN_nfe_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=AF_nfe_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=nhomalt_nfe_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=AC_amr,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Latino ancestry\">',\n",
       " '##INFO=<ID=AN_amr,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Latino ancestry\">',\n",
       " '##INFO=<ID=AF_amr,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Latino ancestry\">',\n",
       " '##INFO=<ID=nhomalt_amr,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Latino ancestry\">',\n",
       " '##INFO=<ID=non_topmed_AC_nfe_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_nfe_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_nfe_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_nfe_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_sas,Number=A,Type=Integer,Description=\"Alternate allele count for samples of South Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_sas,Number=1,Type=Integer,Description=\"Total number of alleles in samples of South Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_sas,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of South Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_sas,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of South Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_fin_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Finnish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_fin_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Finnish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_fin_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Finnish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_fin_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Finnish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_nfe_seu,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Southern European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_nfe_seu,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Southern European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_nfe_seu,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Southern European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_nfe_seu,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Southern European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=AC_eas,Number=A,Type=Integer,Description=\"Alternate allele count for samples of East Asian ancestry\">',\n",
       " '##INFO=<ID=AN_eas,Number=1,Type=Integer,Description=\"Total number of alleles in samples of East Asian ancestry\">',\n",
       " '##INFO=<ID=AF_eas,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of East Asian ancestry\">',\n",
       " '##INFO=<ID=nhomalt_eas,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of East Asian ancestry\">',\n",
       " '##INFO=<ID=nhomalt,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples\">',\n",
       " '##INFO=<ID=non_neuro_AC_nfe_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_nfe_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_nfe_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_nfe_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_afr,Number=A,Type=Integer,Description=\"Alternate allele count for samples of African-American/African ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_afr,Number=1,Type=Integer,Description=\"Total number of alleles in samples of African-American/African ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_afr,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of African-American/African ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_afr,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of African-American/African ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=controls_AC_raw,Number=A,Type=Integer,Description=\"Alternate allele count for samples in the controls subset, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=controls_AN_raw,Number=1,Type=Integer,Description=\"Total number of alleles in samples in the controls subset, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=controls_AF_raw,Number=A,Type=Float,Description=\"Alternate allele frequency in samples in the controls subset, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=controls_nhomalt_raw,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples in the controls subset, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=non_cancer_AC_eas,Number=A,Type=Integer,Description=\"Alternate allele count for samples of East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_eas,Number=1,Type=Integer,Description=\"Total number of alleles in samples of East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_eas,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_eas,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_amr_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Latino ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_amr_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Latino ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_amr_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Latino ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_amr_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Latino ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_nfe_swe,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Swedish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_nfe_swe,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Swedish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_nfe_swe,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Swedish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_nfe_swe,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Swedish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=controls_AC_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples in the controls subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples in the non_topmed subset\">',\n",
       " '##INFO=<ID=controls_AC_eas_jpn,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Japanese ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_eas_jpn,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Japanese ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_eas_jpn,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Japanese ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_eas_jpn,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Japanese ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AC_nfe_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_nfe_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_nfe_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_nfe_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_amr,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Latino ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_amr,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Latino ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_amr,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Latino ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_amr,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Latino ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_eas_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_eas_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_eas_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_eas_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=AC_asj_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Ashkenazi Jewish ancestry\">',\n",
       " '##INFO=<ID=AN_asj_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Ashkenazi Jewish ancestry\">',\n",
       " '##INFO=<ID=AF_asj_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Ashkenazi Jewish ancestry\">',\n",
       " '##INFO=<ID=nhomalt_asj_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Ashkenazi Jewish ancestry\">',\n",
       " '##INFO=<ID=controls_AC_nfe_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_nfe_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_nfe_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_nfe_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_fin,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Finnish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_fin,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Finnish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_fin,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Finnish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_fin,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Finnish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_sas,Number=A,Type=Integer,Description=\"Alternate allele count for samples of South Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_sas,Number=1,Type=Integer,Description=\"Total number of alleles in samples of South Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_sas,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of South Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_sas,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of South Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_nfe_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_nfe_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_nfe_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_nfe_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=AC_oth_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Other ancestry\">',\n",
       " '##INFO=<ID=AN_oth_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Other ancestry\">',\n",
       " '##INFO=<ID=AF_oth_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Other ancestry\">',\n",
       " '##INFO=<ID=nhomalt_oth_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Other ancestry\">',\n",
       " '##INFO=<ID=non_cancer_AC_asj,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Ashkenazi Jewish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_asj,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Ashkenazi Jewish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_asj,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Ashkenazi Jewish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_asj,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Ashkenazi Jewish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=AC_nfe_swe,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Swedish ancestry\">',\n",
       " '##INFO=<ID=AN_nfe_swe,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Swedish ancestry\">',\n",
       " '##INFO=<ID=AF_nfe_swe,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Swedish ancestry\">',\n",
       " '##INFO=<ID=nhomalt_nfe_swe,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Swedish ancestry\">',\n",
       " '##INFO=<ID=controls_AC_nfe,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_nfe,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_nfe,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_nfe,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AC_oth_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Other ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_oth_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Other ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_oth_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Other ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_oth_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Other ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AC_asj,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Ashkenazi Jewish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_asj,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Ashkenazi Jewish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_asj,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Ashkenazi Jewish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_asj,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Ashkenazi Jewish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_amr_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Latino ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_amr_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Latino ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_amr_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Latino ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_amr_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Latino ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=controls_AC_nfe_nwe,Number=A,Type=Integer,Description=\"Alternate allele count for samples of North-Western European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_nfe_nwe,Number=1,Type=Integer,Description=\"Total number of alleles in samples of North-Western European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_nfe_nwe,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of North-Western European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_nfe_nwe,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of North-Western European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=AC_nfe_nwe,Number=A,Type=Integer,Description=\"Alternate allele count for samples of North-Western European ancestry\">',\n",
       " '##INFO=<ID=AN_nfe_nwe,Number=1,Type=Integer,Description=\"Total number of alleles in samples of North-Western European ancestry\">',\n",
       " '##INFO=<ID=AF_nfe_nwe,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of North-Western European ancestry\">',\n",
       " '##INFO=<ID=nhomalt_nfe_nwe,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of North-Western European ancestry\">',\n",
       " '##INFO=<ID=controls_AC_nfe_seu,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Southern European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_nfe_seu,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Southern European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_nfe_seu,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Southern European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_nfe_seu,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Southern European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AC_sas_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of South Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_sas_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of South Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_sas_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of South Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_sas_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of South Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_amr_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Latino ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_amr_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Latino ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_amr_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Latino ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_amr_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Latino ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_eas_jpn,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Japanese ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_eas_jpn,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Japanese ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_eas_jpn,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Japanese ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_eas_jpn,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Japanese ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_nfe_onf,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Other Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_nfe_onf,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Other Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_nfe_onf,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Other Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_nfe_onf,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Other Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_eas_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_eas_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_eas_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_eas_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=AC_eas_jpn,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Japanese ancestry\">',\n",
       " '##INFO=<ID=AN_eas_jpn,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Japanese ancestry\">',\n",
       " '##INFO=<ID=AF_eas_jpn,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Japanese ancestry\">',\n",
       " '##INFO=<ID=nhomalt_eas_jpn,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Japanese ancestry\">',\n",
       " '##INFO=<ID=non_cancer_AC_afr_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of African-American/African ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_afr_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of African-American/African ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_afr_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of African-American/African ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_afr_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of African-American/African ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_afr,Number=A,Type=Integer,Description=\"Alternate allele count for samples of African-American/African ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_afr,Number=1,Type=Integer,Description=\"Total number of alleles in samples of African-American/African ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_afr,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of African-American/African ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_afr,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of African-American/African ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=controls_AC_amr_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Latino ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_amr_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Latino ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_amr_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Latino ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_amr_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Latino ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_fin_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Finnish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_fin_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Finnish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_fin_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Finnish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_fin_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Finnish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=AC_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples\">',\n",
       " '##INFO=<ID=AN_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples\">',\n",
       " '##INFO=<ID=AF_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples\">',\n",
       " '##INFO=<ID=nhomalt_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples\">',\n",
       " '##INFO=<ID=non_neuro_AC_nfe_bgr,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Bulgarian (Eastern European) ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_nfe_bgr,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Bulgarian (Eastern European) ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_nfe_bgr,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Bulgarian (Eastern European) ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_nfe_bgr,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Bulgarian (Eastern European) ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_oth_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Other ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_oth_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Other ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_oth_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Other ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_oth_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Other ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_nfe_est,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Estonian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_nfe_est,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Estonian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_nfe_est,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Estonian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_nfe_est,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Estonian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_nfe_nwe,Number=A,Type=Integer,Description=\"Alternate allele count for samples of North-Western European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_nfe_nwe,Number=1,Type=Integer,Description=\"Total number of alleles in samples of North-Western European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_nfe_nwe,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of North-Western European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_nfe_nwe,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of North-Western European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_amr_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Latino ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_amr_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Latino ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_amr_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Latino ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_amr_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Latino ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_amr,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Latino ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_amr,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Latino ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_amr,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Latino ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_amr,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Latino ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_nfe_swe,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Swedish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_nfe_swe,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Swedish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_nfe_swe,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Swedish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_nfe_swe,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Swedish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_nfe_onf,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Other Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_nfe_onf,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Other Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_nfe_onf,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Other Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_nfe_onf,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Other Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=controls_AC_eas_kor,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Korean ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_eas_kor,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Korean ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_eas_kor,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Korean ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_eas_kor,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Korean ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_eas_oea,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Other East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_eas_oea,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Other East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_eas_oea,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Other East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_eas_oea,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Other East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=controls_AC_eas_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_eas_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_eas_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_eas_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AC_oth_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Other ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_oth_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Other ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_oth_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Other ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_oth_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Other ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_topmed_AC,Number=A,Type=Integer,Description=\"Alternate allele count for samples in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN,Number=1,Type=Integer,Description=\"Total number of alleles in samples in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF,Number=A,Type=Float,Description=\"Alternate allele frequency in samples in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples in the non_topmed subset\">',\n",
       " '##INFO=<ID=controls_AC_fin,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Finnish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_fin,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Finnish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_fin,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Finnish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_fin,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Finnish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=AC_eas_kor,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Korean ancestry\">',\n",
       " '##INFO=<ID=AN_eas_kor,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Korean ancestry\">',\n",
       " '##INFO=<ID=AF_eas_kor,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Korean ancestry\">',\n",
       " '##INFO=<ID=nhomalt_eas_kor,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Korean ancestry\">',\n",
       " '##INFO=<ID=non_neuro_AC_nfe,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_nfe,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_nfe,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_nfe,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_fin_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Finnish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_fin_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Finnish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_fin_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Finnish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_fin_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Finnish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_nfe_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_nfe_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_nfe_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_nfe_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=controls_AC_eas_oea,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Other East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_eas_oea,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Other East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_eas_oea,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Other East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_eas_oea,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Other East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_nfe_seu,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Southern European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_nfe_seu,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Southern European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_nfe_seu,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Southern European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_nfe_seu,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Southern European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=controls_AC_eas_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_eas_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_eas_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_eas_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_asj,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Ashkenazi Jewish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_asj,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Ashkenazi Jewish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_asj,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Ashkenazi Jewish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_asj,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Ashkenazi Jewish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=controls_AC_nfe_onf,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Other Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_nfe_onf,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Other Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_nfe_onf,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Other Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_nfe_onf,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Other Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_neuro_AC,Number=A,Type=Integer,Description=\"Alternate allele count for samples in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN,Number=1,Type=Integer,Description=\"Total number of alleles in samples in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF,Number=A,Type=Float,Description=\"Alternate allele frequency in samples in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples in the non_neuro subset\">',\n",
       " '##INFO=<ID=AC_eas_oea,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Other East Asian ancestry\">',\n",
       " '##INFO=<ID=AN_eas_oea,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Other East Asian ancestry\">',\n",
       " '##INFO=<ID=AF_eas_oea,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Other East Asian ancestry\">',\n",
       " '##INFO=<ID=nhomalt_eas_oea,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Other East Asian ancestry\">',\n",
       " '##INFO=<ID=non_topmed_AC_nfe,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_nfe,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_nfe,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_nfe,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_oth,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Other ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_oth,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Other ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_oth,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Other ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_oth,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Other ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_raw,Number=A,Type=Integer,Description=\"Alternate allele count for samples in the non_topmed subset, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=non_topmed_AN_raw,Number=1,Type=Integer,Description=\"Total number of alleles in samples in the non_topmed subset, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=non_topmed_AF_raw,Number=A,Type=Float,Description=\"Alternate allele frequency in samples in the non_topmed subset, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_raw,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples in the non_topmed subset, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=non_neuro_AC_nfe_est,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Estonian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_nfe_est,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Estonian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_nfe_est,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Estonian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_nfe_est,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Estonian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_oth_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Other ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_oth_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Other ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_oth_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Other ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_oth_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Other ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_oth_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Other ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_oth_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Other ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_oth_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Other ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_oth_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Other ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=AC_nfe_est,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Estonian ancestry\">',\n",
       " '##INFO=<ID=AN_nfe_est,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Estonian ancestry\">',\n",
       " '##INFO=<ID=AF_nfe_est,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Estonian ancestry\">',\n",
       " '##INFO=<ID=nhomalt_nfe_est,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Estonian ancestry\">',\n",
       " '##INFO=<ID=non_cancer_AC_afr_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of African-American/African ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_afr_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of African-American/African ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_afr_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of African-American/African ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_afr_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of African-American/African ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_afr_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of African-American/African ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_afr_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of African-American/African ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_afr_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of African-American/African ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_afr_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of African-American/African ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=AC_eas_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of East Asian ancestry\">',\n",
       " '##INFO=<ID=AN_eas_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of East Asian ancestry\">',\n",
       " '##INFO=<ID=AF_eas_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of East Asian ancestry\">',\n",
       " '##INFO=<ID=nhomalt_eas_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of East Asian ancestry\">',\n",
       " '##INFO=<ID=controls_AC_eas,Number=A,Type=Integer,Description=\"Alternate allele count for samples of East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_eas,Number=1,Type=Integer,Description=\"Total number of alleles in samples of East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_eas,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_eas,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_eas_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_eas_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_eas_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_eas_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_nfe_nwe,Number=A,Type=Integer,Description=\"Alternate allele count for samples of North-Western European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_nfe_nwe,Number=1,Type=Integer,Description=\"Total number of alleles in samples of North-Western European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_nfe_nwe,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of North-Western European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_nfe_nwe,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of North-Western European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=controls_AC_sas,Number=A,Type=Integer,Description=\"Alternate allele count for samples of South Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_sas,Number=1,Type=Integer,Description=\"Total number of alleles in samples of South Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_sas,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of South Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_sas,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of South Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_sas_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of South Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_sas_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of South Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_sas_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of South Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_sas_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of South Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_asj_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Ashkenazi Jewish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_asj_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Ashkenazi Jewish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_asj_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Ashkenazi Jewish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_asj_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Ashkenazi Jewish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_nfe_bgr,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Bulgarian (Eastern European) ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_nfe_bgr,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Bulgarian (Eastern European) ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_nfe_bgr,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Bulgarian (Eastern European) ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_nfe_bgr,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Bulgarian (Eastern European) ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=controls_AC_oth,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Other ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_oth,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Other ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_oth,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Other ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_oth,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Other ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_eas_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_eas_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_eas_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_eas_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=AC_nfe,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=AN_nfe,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=AF_nfe,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=nhomalt_nfe,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=non_topmed_AC_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_asj,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Ashkenazi Jewish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_asj,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Ashkenazi Jewish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_asj,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Ashkenazi Jewish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_asj,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Ashkenazi Jewish ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_eas_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_eas_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_eas_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_eas_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_raw,Number=A,Type=Integer,Description=\"Alternate allele count for samples in the non_neuro subset, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=non_neuro_AN_raw,Number=1,Type=Integer,Description=\"Total number of alleles in samples in the non_neuro subset, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=non_neuro_AF_raw,Number=A,Type=Float,Description=\"Alternate allele frequency in samples in the non_neuro subset, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_raw,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples in the non_neuro subset, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=non_topmed_AC_eas,Number=A,Type=Integer,Description=\"Alternate allele count for samples of East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_eas,Number=1,Type=Integer,Description=\"Total number of alleles in samples of East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_eas,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_eas,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_fin_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Finnish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_fin_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Finnish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_fin_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Finnish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_fin_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Finnish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_asj_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Ashkenazi Jewish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_asj_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Ashkenazi Jewish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_asj_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Ashkenazi Jewish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_asj_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Ashkenazi Jewish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=AC_fin,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Finnish ancestry\">',\n",
       " '##INFO=<ID=AN_fin,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Finnish ancestry\">',\n",
       " '##INFO=<ID=AF_fin,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Finnish ancestry\">',\n",
       " '##INFO=<ID=nhomalt_fin,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Finnish ancestry\">',\n",
       " '##INFO=<ID=AC_nfe_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=AN_nfe_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=AF_nfe_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=nhomalt_nfe_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=non_topmed_AC_eas_kor,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Korean ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_eas_kor,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Korean ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_eas_kor,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Korean ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_eas_kor,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Korean ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=controls_AC_amr_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Latino ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_amr_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Latino ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_amr_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Latino ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_amr_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Latino ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_eas_oea,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Other East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_eas_oea,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Other East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_eas_oea,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Other East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_eas_oea,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Other East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=AC_sas_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of South Asian ancestry\">',\n",
       " '##INFO=<ID=AN_sas_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of South Asian ancestry\">',\n",
       " '##INFO=<ID=AF_sas_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of South Asian ancestry\">',\n",
       " '##INFO=<ID=nhomalt_sas_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of South Asian ancestry\">',\n",
       " '##INFO=<ID=controls_AC_afr_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of African-American/African ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_afr_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of African-American/African ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_afr_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of African-American/African ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_afr_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of African-American/African ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AC_amr,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Latino ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_amr,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Latino ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_amr,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Latino ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_amr,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Latino ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_eas_jpn,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Japanese ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_eas_jpn,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Japanese ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_eas_jpn,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Japanese ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_eas_jpn,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Japanese ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=AC_asj_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Ashkenazi Jewish ancestry\">',\n",
       " '##INFO=<ID=AN_asj_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Ashkenazi Jewish ancestry\">',\n",
       " '##INFO=<ID=AF_asj_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Ashkenazi Jewish ancestry\">',\n",
       " '##INFO=<ID=nhomalt_asj_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Ashkenazi Jewish ancestry\">',\n",
       " '##INFO=<ID=non_topmed_AC_nfe_bgr,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Bulgarian (Eastern European) ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_nfe_bgr,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Bulgarian (Eastern European) ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_nfe_bgr,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Bulgarian (Eastern European) ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_nfe_bgr,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Bulgarian (Eastern European) ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_nfe_est,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Estonian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_nfe_est,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Estonian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_nfe_est,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Estonian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_nfe_est,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Estonian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_eas,Number=A,Type=Integer,Description=\"Alternate allele count for samples of East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_eas,Number=1,Type=Integer,Description=\"Total number of alleles in samples of East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_eas,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_eas,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_nfe,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_nfe,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_nfe,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_nfe,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_sas_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of South Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_sas_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of South Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_sas_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of South Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_sas_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of South Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=AC_asj,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Ashkenazi Jewish ancestry\">',\n",
       " '##INFO=<ID=AN_asj,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Ashkenazi Jewish ancestry\">',\n",
       " '##INFO=<ID=AF_asj,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Ashkenazi Jewish ancestry\">',\n",
       " '##INFO=<ID=nhomalt_asj,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Ashkenazi Jewish ancestry\">',\n",
       " '##INFO=<ID=controls_AC_nfe_est,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Estonian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_nfe_est,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Estonian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_nfe_est,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Estonian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_nfe_est,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Estonian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_asj_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Ashkenazi Jewish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_asj_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Ashkenazi Jewish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_asj_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Ashkenazi Jewish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_asj_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Ashkenazi Jewish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_nfe_swe,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Swedish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_nfe_swe,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Swedish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_nfe_swe,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Swedish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_nfe_swe,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Swedish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AC,Number=A,Type=Integer,Description=\"Alternate allele count for samples in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN,Number=1,Type=Integer,Description=\"Total number of alleles in samples in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF,Number=A,Type=Float,Description=\"Alternate allele frequency in samples in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_oth,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Other ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_oth,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Other ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_oth,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Other ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_oth,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Other ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_fin_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Finnish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_fin_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Finnish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_fin_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Finnish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_fin_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Finnish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_fin_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Finnish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_fin_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Finnish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_fin_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Finnish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_fin_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Finnish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=AC_oth,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Other ancestry\">',\n",
       " '##INFO=<ID=AN_oth,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Other ancestry\">',\n",
       " '##INFO=<ID=AF_oth,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Other ancestry\">',\n",
       " '##INFO=<ID=nhomalt_oth,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Other ancestry\">',\n",
       " '##INFO=<ID=non_neuro_AC_nfe_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_nfe_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_nfe_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_nfe_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=controls_AC_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples in the controls subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_fin,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Finnish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_fin,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Finnish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_fin,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Finnish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_fin,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Finnish ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_fin,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Finnish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_fin,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Finnish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_fin,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Finnish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_fin,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Finnish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_eas_oea,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Other East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_eas_oea,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Other East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_eas_oea,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Other East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_eas_oea,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Other East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_nfe_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_nfe_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_nfe_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_nfe_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_sas_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of South Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_sas_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of South Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_sas_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of South Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_sas_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of South Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=controls_AC_asj_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Ashkenazi Jewish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_asj_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Ashkenazi Jewish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_asj_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Ashkenazi Jewish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_asj_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Ashkenazi Jewish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_raw,Number=A,Type=Integer,Description=\"Alternate allele count for samples in the non_cancer subset, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=non_cancer_AN_raw,Number=1,Type=Integer,Description=\"Total number of alleles in samples in the non_cancer subset, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=non_cancer_AF_raw,Number=A,Type=Float,Description=\"Alternate allele frequency in samples in the non_cancer subset, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_raw,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples in the non_cancer subset, before removing low-confidence genotypes\">',\n",
       " '##INFO=<ID=non_cancer_AC_eas_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_eas_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_eas_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_eas_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_asj_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Ashkenazi Jewish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_asj_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Ashkenazi Jewish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_asj_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Ashkenazi Jewish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_asj_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Ashkenazi Jewish ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_oth,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Other ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_oth,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Other ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_oth,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Other ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_oth,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Other ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=AC_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples\">',\n",
       " '##INFO=<ID=AN_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples\">',\n",
       " '##INFO=<ID=AF_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples\">',\n",
       " '##INFO=<ID=nhomalt_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples\">',\n",
       " '##INFO=<ID=controls_AC_fin_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Finnish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_fin_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Finnish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_fin_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Finnish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_fin_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Finnish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AC_nfe_bgr,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Bulgarian (Eastern European) ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_nfe_bgr,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Bulgarian (Eastern European) ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_nfe_bgr,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Bulgarian (Eastern European) ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_nfe_bgr,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Bulgarian (Eastern European) ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AC_asj_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Ashkenazi Jewish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_asj_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Ashkenazi Jewish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_asj_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Ashkenazi Jewish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_asj_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Ashkenazi Jewish ancestry in the controls subset\">',\n",
       " '##INFO=<ID=AC_amr_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Latino ancestry\">',\n",
       " '##INFO=<ID=AN_amr_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Latino ancestry\">',\n",
       " '##INFO=<ID=AF_amr_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Latino ancestry\">',\n",
       " '##INFO=<ID=nhomalt_amr_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Latino ancestry\">',\n",
       " '##INFO=<ID=AC_amr_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Latino ancestry\">',\n",
       " '##INFO=<ID=AN_amr_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Latino ancestry\">',\n",
       " '##INFO=<ID=AF_amr_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Latino ancestry\">',\n",
       " '##INFO=<ID=nhomalt_amr_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Latino ancestry\">',\n",
       " '##INFO=<ID=non_topmed_AC_sas_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of South Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_sas_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of South Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_sas_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of South Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_sas_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of South Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=AC_oth_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of Other ancestry\">',\n",
       " '##INFO=<ID=AN_oth_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of Other ancestry\">',\n",
       " '##INFO=<ID=AF_oth_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of Other ancestry\">',\n",
       " '##INFO=<ID=nhomalt_oth_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of Other ancestry\">',\n",
       " '##INFO=<ID=non_cancer_AC_sas,Number=A,Type=Integer,Description=\"Alternate allele count for samples of South Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_sas,Number=1,Type=Integer,Description=\"Total number of alleles in samples of South Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_sas,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of South Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_sas,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of South Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_nfe_seu,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Southern European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_nfe_seu,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Southern European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_nfe_seu,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Southern European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_nfe_seu,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Southern European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_eas_kor,Number=A,Type=Integer,Description=\"Alternate allele count for samples of Korean ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_eas_kor,Number=1,Type=Integer,Description=\"Total number of alleles in samples of Korean ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_eas_kor,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of Korean ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_eas_kor,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of Korean ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_afr_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of African-American/African ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_afr_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of African-American/African ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_afr_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of African-American/African ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_afr_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of African-American/African ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=controls_AC_sas_male,Number=A,Type=Integer,Description=\"Alternate allele count for male samples of South Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_sas_male,Number=1,Type=Integer,Description=\"Total number of alleles in male samples of South Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_sas_male,Number=A,Type=Float,Description=\"Alternate allele frequency in male samples of South Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_sas_male,Number=A,Type=Integer,Description=\"Count of homozygous individuals in male samples of South Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_sas_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of South Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_sas_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of South Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_sas_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of South Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_sas_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of South Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_afr,Number=A,Type=Integer,Description=\"Alternate allele count for samples of African-American/African ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_afr,Number=1,Type=Integer,Description=\"Total number of alleles in samples of African-American/African ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_afr,Number=A,Type=Float,Description=\"Alternate allele frequency in samples of African-American/African ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_afr,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples of African-American/African ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=controls_AC,Number=A,Type=Integer,Description=\"Alternate allele count for samples in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN,Number=1,Type=Integer,Description=\"Total number of alleles in samples in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF,Number=A,Type=Float,Description=\"Alternate allele frequency in samples in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt,Number=A,Type=Integer,Description=\"Count of homozygous individuals in samples in the controls subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_oth_female,Number=A,Type=Integer,Description=\"Alternate allele count for female samples of Other ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_oth_female,Number=1,Type=Integer,Description=\"Total number of alleles in female samples of Other ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_oth_female,Number=A,Type=Float,Description=\"Alternate allele frequency in female samples of Other ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_oth_female,Number=A,Type=Integer,Description=\"Count of homozygous individuals in female samples of Other ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_topmed_faf95_amr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of Latino ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_faf99_amr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of Latino ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=faf95_afr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of African-American/African ancestry\">',\n",
       " '##INFO=<ID=faf99_afr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of African-American/African ancestry\">',\n",
       " '##INFO=<ID=faf95_sas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of South Asian ancestry\">',\n",
       " '##INFO=<ID=faf99_sas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of South Asian ancestry\">',\n",
       " '##INFO=<ID=controls_faf95_afr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of African-American/African ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_faf99_afr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of African-American/African ancestry in the controls subset\">',\n",
       " '##INFO=<ID=faf95_amr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of Latino ancestry\">',\n",
       " '##INFO=<ID=faf99_amr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of Latino ancestry\">',\n",
       " '##INFO=<ID=non_neuro_faf95_sas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of South Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_faf99_sas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of South Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=faf95_eas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of East Asian ancestry\">',\n",
       " '##INFO=<ID=faf99_eas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of East Asian ancestry\">',\n",
       " '##INFO=<ID=faf95,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples\">',\n",
       " '##INFO=<ID=faf99,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples\">',\n",
       " '##INFO=<ID=non_neuro_faf95_afr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of African-American/African ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_faf99_afr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of African-American/African ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_cancer_faf95_eas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_faf99_eas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of East Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_neuro_faf95_amr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of Latino ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_faf99_amr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of Latino ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_topmed_faf95_sas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of South Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_faf99_sas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of South Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=controls_faf95_nfe,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_faf99_nfe,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of Non-Finnish European ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_cancer_faf95_afr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of African-American/African ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_faf99_afr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of African-American/African ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_faf95_amr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of Latino ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_faf99_amr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of Latino ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_topmed_faf95,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_faf99,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_neuro_faf95_nfe,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_faf99_nfe,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of Non-Finnish European ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_faf95,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_faf99,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_topmed_faf95_nfe,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_faf99_nfe,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of Non-Finnish European ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=controls_faf95_eas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_faf99_eas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of East Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_faf95_sas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of South Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_faf99_sas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of South Asian ancestry in the controls subset\">',\n",
       " '##INFO=<ID=faf95_nfe,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=faf99_nfe,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of Non-Finnish European ancestry\">',\n",
       " '##INFO=<ID=non_topmed_faf95_eas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_faf99_eas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of East Asian ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=controls_faf95_amr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of Latino ancestry in the controls subset\">',\n",
       " '##INFO=<ID=controls_faf99_amr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of Latino ancestry in the controls subset\">',\n",
       " '##INFO=<ID=non_neuro_faf95_eas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_faf99_eas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of East Asian ancestry in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_cancer_faf95_nfe,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_faf99_nfe,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of Non-Finnish European ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_faf95,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_faf99,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_faf95_sas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of South Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_faf99_sas,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of South Asian ancestry in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_topmed_faf95_afr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples of African-American/African ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_faf99_afr,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples of African-American/African ancestry in the non_topmed subset\">',\n",
       " '##INFO=<ID=controls_faf95,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 95% CI) for samples in the controls subset\">',\n",
       " '##INFO=<ID=controls_faf99,Number=A,Type=Float,Description=\"Filtering allele frequency (using Poisson 99% CI) for samples in the controls subset\">',\n",
       " '##INFO=<ID=popmax,Number=A,Type=String,Description=\"Population with maximum AF\">',\n",
       " '##INFO=<ID=AC_popmax,Number=A,Type=Integer,Description=\"Allele count in the population with the maximum AF\">',\n",
       " '##INFO=<ID=AN_popmax,Number=A,Type=Integer,Description=\"Total number of alleles in the population with the maximum AF\">',\n",
       " '##INFO=<ID=AF_popmax,Number=A,Type=Float,Description=\"Maximum allele frequency across populations (excluding samples of Ashkenazi, Finnish, and indeterminate ancestry)\">',\n",
       " '##INFO=<ID=nhomalt_popmax,Number=A,Type=Integer,Description=\"Count of homozygous individuals in the population with the maximum allele frequency\">',\n",
       " '##INFO=<ID=age_hist_het_bin_freq,Number=A,Type=String,Description=\"Histogram of ages of heterozygous individuals; bin edges are: 30.0|35.0|40.0|45.0|50.0|55.0|60.0|65.0|70.0|75.0|80.0; total number of individuals of any genotype bin: 2547|3423|4546|8487|10355|12693|11933|10534|8882|5991|4136|1935\">',\n",
       " '##INFO=<ID=age_hist_het_n_smaller,Number=A,Type=Integer,Description=\"Count of age values falling below lowest histogram bin edge for heterozygous individuals\">',\n",
       " '##INFO=<ID=age_hist_het_n_larger,Number=A,Type=Integer,Description=\"Count of age values falling above highest histogram bin edge for heterozygous individuals\">',\n",
       " '##INFO=<ID=age_hist_hom_bin_freq,Number=A,Type=String,Description=\"Histogram of ages of homozygous alternate individuals; bin edges are: 30.0|35.0|40.0|45.0|50.0|55.0|60.0|65.0|70.0|75.0|80.0; total number of individuals of any genotype bin: 2547|3423|4546|8487|10355|12693|11933|10534|8882|5991|4136|1935\">',\n",
       " '##INFO=<ID=age_hist_hom_n_smaller,Number=A,Type=Integer,Description=\"Count of age values falling below lowest histogram bin edge for homozygous alternate individuals\">',\n",
       " '##INFO=<ID=age_hist_hom_n_larger,Number=A,Type=Integer,Description=\"Count of age values falling above highest histogram bin edge for homozygous alternate individuals\">',\n",
       " '##INFO=<ID=non_topmed_popmax,Number=A,Type=String,Description=\"Population with maximum AF in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AC_popmax,Number=A,Type=Integer,Description=\"Allele count in the population with the maximum AF in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AN_popmax,Number=A,Type=Integer,Description=\"Total number of alleles in the population with the maximum AF in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_AF_popmax,Number=A,Type=Float,Description=\"Maximum allele frequency across populations (excluding samples of Ashkenazi, Finnish, and indeterminate ancestry) in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_topmed_nhomalt_popmax,Number=A,Type=Integer,Description=\"Count of homozygous individuals in the population with the maximum allele frequency in the non_topmed subset\">',\n",
       " '##INFO=<ID=non_neuro_popmax,Number=A,Type=String,Description=\"Population with maximum AF in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AC_popmax,Number=A,Type=Integer,Description=\"Allele count in the population with the maximum AF in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AN_popmax,Number=A,Type=Integer,Description=\"Total number of alleles in the population with the maximum AF in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_AF_popmax,Number=A,Type=Float,Description=\"Maximum allele frequency across populations (excluding samples of Ashkenazi, Finnish, and indeterminate ancestry) in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_neuro_nhomalt_popmax,Number=A,Type=Integer,Description=\"Count of homozygous individuals in the population with the maximum allele frequency in the non_neuro subset\">',\n",
       " '##INFO=<ID=non_cancer_popmax,Number=A,Type=String,Description=\"Population with maximum AF in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AC_popmax,Number=A,Type=Integer,Description=\"Allele count in the population with the maximum AF in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AN_popmax,Number=A,Type=Integer,Description=\"Total number of alleles in the population with the maximum AF in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_AF_popmax,Number=A,Type=Float,Description=\"Maximum allele frequency across populations (excluding samples of Ashkenazi, Finnish, and indeterminate ancestry) in the non_cancer subset\">',\n",
       " '##INFO=<ID=non_cancer_nhomalt_popmax,Number=A,Type=Integer,Description=\"Count of homozygous individuals in the population with the maximum allele frequency in the non_cancer subset\">',\n",
       " '##INFO=<ID=controls_popmax,Number=A,Type=String,Description=\"Population with maximum AF in the controls subset\">',\n",
       " '##INFO=<ID=controls_AC_popmax,Number=A,Type=Integer,Description=\"Allele count in the population with the maximum AF in the controls subset\">',\n",
       " '##INFO=<ID=controls_AN_popmax,Number=A,Type=Integer,Description=\"Total number of alleles in the population with the maximum AF in the controls subset\">',\n",
       " '##INFO=<ID=controls_AF_popmax,Number=A,Type=Float,Description=\"Maximum allele frequency across populations (excluding samples of Ashkenazi, Finnish, and indeterminate ancestry) in the controls subset\">',\n",
       " '##INFO=<ID=controls_nhomalt_popmax,Number=A,Type=Integer,Description=\"Count of homozygous individuals in the population with the maximum allele frequency in the controls subset\">',\n",
       " '##INFO=<ID=vep,Number=.,Type=String,Description=\"Consequence annotations from Ensembl VEP. Format: Allele|Consequence|IMPACT|SYMBOL|Gene|Feature_type|Feature|BIOTYPE|EXON|INTRON|HGVSc|HGVSp|cDNA_position|CDS_position|Protein_position|Amino_acids|Codons|Existing_variation|ALLELE_NUM|DISTANCE|STRAND|FLAGS|VARIANT_CLASS|MINIMISED|SYMBOL_SOURCE|HGNC_ID|CANONICAL|TSL|APPRIS|CCDS|ENSP|SWISSPROT|TREMBL|UNIPARC|GENE_PHENO|SIFT|PolyPhen|DOMAINS|HGVS_OFFSET|GMAF|AFR_MAF|AMR_MAF|EAS_MAF|EUR_MAF|SAS_MAF|AA_MAF|EA_MAF|ExAC_MAF|ExAC_Adj_MAF|ExAC_AFR_MAF|ExAC_AMR_MAF|ExAC_EAS_MAF|ExAC_FIN_MAF|ExAC_NFE_MAF|ExAC_OTH_MAF|ExAC_SAS_MAF|CLIN_SIG|SOMATIC|PHENO|PUBMED|MOTIF_NAME|MOTIF_POS|HIGH_INF_POS|MOTIF_SCORE_CHANGE|LoF|LoF_filter|LoF_flags|LoF_info\">',\n",
       " '##contig=<ID=1,length=249250621,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=2,length=243199373,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=3,length=198022430,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=4,length=191154276,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=5,length=180915260,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=6,length=171115067,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=7,length=159138663,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=8,length=146364022,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=9,length=141213431,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=10,length=135534747,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=11,length=135006516,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=12,length=133851895,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=13,length=115169878,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=14,length=107349540,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=15,length=102531392,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=16,length=90354753,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=17,length=81195210,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=18,length=78077248,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=19,length=59128983,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=20,length=63025520,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=21,length=48129895,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=22,length=51304566,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=X,length=155270560,assembly=gnomAD_GRCh37>',\n",
       " '##contig=<ID=Y,length=59373566,assembly=gnomAD_GRCh37>']"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "info_line = find_comment(gnomAD_path, r\"##\")\n",
    "\n",
    "head(gnomAD_path, info_line+1)\n",
    "# info_line\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def read_vcf(vcf_path:str):\n",
    "    info_line = find_comment(vcf_path, r\"##\")\n",
    "    info = head(file_dir = vcf_path, n=info_line + 1, is_strip=True) if info_line != -1 else \"\"\n",
    "    vcf = strip_comment_head(pd.read_csv(vcf_path, sep= \"\\t\", skiprows=find_comment(gnomAD_path, r\"#\")), inplace=False)\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>CHROM</th>\n",
       "      <th>POS</th>\n",
       "      <th>ID</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "      <th>QUAL</th>\n",
       "      <th>FILTER</th>\n",
       "      <th>INFO</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>12573</td>\n",
       "      <td>rs1273605438</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>366.59</td>\n",
       "      <td>PASS</td>\n",
       "      <td>AC=2;AN=4196;AF=4.76644e-04;rf_tp_probability=...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>12586</td>\n",
       "      <td>rs1336625132</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>223.87</td>\n",
       "      <td>PASS</td>\n",
       "      <td>AC=2;AN=3578;AF=5.58971e-04;rf_tp_probability=...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1</td>\n",
       "      <td>12625</td>\n",
       "      <td>rs1235144565</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>55.63</td>\n",
       "      <td>PASS</td>\n",
       "      <td>AC=1;AN=5720;AF=1.74825e-04;rf_tp_probability=...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1</td>\n",
       "      <td>12680</td>\n",
       "      <td>rs1163072234</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>796.70</td>\n",
       "      <td>PASS</td>\n",
       "      <td>AC=6;AN=7148;AF=8.39396e-04;rf_tp_probability=...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1</td>\n",
       "      <td>12754</td>\n",
       "      <td>rs1231210093</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>272.84</td>\n",
       "      <td>PASS</td>\n",
       "      <td>AC=1;AN=14718;AF=6.79440e-05;rf_tp_probability...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>595</th>\n",
       "      <td>1</td>\n",
       "      <td>325042</td>\n",
       "      <td>rs748079918</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>1155.64</td>\n",
       "      <td>PASS</td>\n",
       "      <td>AC=1;AN=46642;AF=2.14399e-05;rf_tp_probability...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>596</th>\n",
       "      <td>1</td>\n",
       "      <td>325052</td>\n",
       "      <td>rs1204939883</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>1009.07</td>\n",
       "      <td>PASS</td>\n",
       "      <td>AC=2;AN=41838;AF=4.78034e-05;rf_tp_probability...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>597</th>\n",
       "      <td>1</td>\n",
       "      <td>325059</td>\n",
       "      <td>rs771872895</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>5680.16</td>\n",
       "      <td>PASS</td>\n",
       "      <td>AC=8;AN=39476;AF=2.02655e-04;rf_tp_probability...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>598</th>\n",
       "      <td>1</td>\n",
       "      <td>325062</td>\n",
       "      <td>rs1262815702</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>2799.99</td>\n",
       "      <td>PASS</td>\n",
       "      <td>AC=6;AN=38646;AF=1.55255e-04;rf_tp_probability...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>599</th>\n",
       "      <td>1</td>\n",
       "      <td>325069</td>\n",
       "      <td>rs1191745868</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>127.10</td>\n",
       "      <td>PASS</td>\n",
       "      <td>AC=1;AN=37378;AF=2.67537e-05;rf_tp_probability...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>600 rows × 8 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     CHROM     POS            ID REF ALT     QUAL FILTER  \\\n",
       "0        1   12573  rs1273605438   T   C   366.59   PASS   \n",
       "1        1   12586  rs1336625132   C   T   223.87   PASS   \n",
       "2        1   12625  rs1235144565   G   A    55.63   PASS   \n",
       "3        1   12680  rs1163072234   G   A   796.70   PASS   \n",
       "4        1   12754  rs1231210093   C   G   272.84   PASS   \n",
       "..     ...     ...           ...  ..  ..      ...    ...   \n",
       "595      1  325042   rs748079918   A   C  1155.64   PASS   \n",
       "596      1  325052  rs1204939883   T   C  1009.07   PASS   \n",
       "597      1  325059   rs771872895   C   T  5680.16   PASS   \n",
       "598      1  325062  rs1262815702   G   A  2799.99   PASS   \n",
       "599      1  325069  rs1191745868   T   C   127.10   PASS   \n",
       "\n",
       "                                                  INFO  \n",
       "0    AC=2;AN=4196;AF=4.76644e-04;rf_tp_probability=...  \n",
       "1    AC=2;AN=3578;AF=5.58971e-04;rf_tp_probability=...  \n",
       "2    AC=1;AN=5720;AF=1.74825e-04;rf_tp_probability=...  \n",
       "3    AC=6;AN=7148;AF=8.39396e-04;rf_tp_probability=...  \n",
       "4    AC=1;AN=14718;AF=6.79440e-05;rf_tp_probability...  \n",
       "..                                                 ...  \n",
       "595  AC=1;AN=46642;AF=2.14399e-05;rf_tp_probability...  \n",
       "596  AC=2;AN=41838;AF=4.78034e-05;rf_tp_probability...  \n",
       "597  AC=8;AN=39476;AF=2.02655e-04;rf_tp_probability...  \n",
       "598  AC=6;AN=38646;AF=1.55255e-04;rf_tp_probability...  \n",
       "599  AC=1;AN=37378;AF=2.67537e-05;rf_tp_probability...  \n",
       "\n",
       "[600 rows x 8 columns]"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "vcf = strip_comment_head(pd.read_csv(gnomAD_path, sep= \"\\t\", skiprows=find_comment(gnomAD_path, r\"#\")), inplace=False)\n",
    "vcf "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['AC=2',\n",
       " 'AN=4196',\n",
       " 'AF=4.76644e-04',\n",
       " 'rf_tp_probability=3.22028e-01',\n",
       " 'FS=0.00000e+00',\n",
       " 'InbreedingCoeff=-7.56000e-02',\n",
       " 'MQ=2.20000e+01',\n",
       " 'MQRankSum=-7.20000e-01',\n",
       " 'QD=4.03000e+00',\n",
       " 'ReadPosRankSum=4.06000e-01',\n",
       " 'SOR=5.91000e-01',\n",
       " 'BaseQRankSum=1.56000e-01',\n",
       " 'ClippingRankSum=-3.87000e-01',\n",
       " 'DP=159660',\n",
       " 'VQSLOD=3.25000e+00',\n",
       " 'VQSR_culprit=QD',\n",
       " 'segdup',\n",
       " 'rf_negative_label',\n",
       " 'rf_label=FP',\n",
       " 'rf_train',\n",
       " 'variant_type=snv',\n",
       " 'allele_type=snv',\n",
       " 'n_alt_alleles=1',\n",
       " 'pab_max=1.00000e+00',\n",
       " 'gq_hist_alt_bin_freq=0|0|0|2|0|0|1|1|1|0|0|1|0|0|0|0|0|0|0|2',\n",
       " 'gq_hist_all_bin_freq=5328|7127|2608|3532|2074|679|849|436|132|204|108|31|43|1|14|0|7|1|1|2',\n",
       " 'dp_hist_alt_bin_freq=2|4|1|1|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0',\n",
       " 'dp_hist_alt_n_larger=0',\n",
       " 'dp_hist_all_bin_freq=116953|6572|1691|421|85|22|4|0|0|0|0|0|0|0|0|0|0|0|0|0',\n",
       " 'dp_hist_all_n_larger=0',\n",
       " 'ab_hist_alt_bin_freq=0|0|0|0|0|1|1|0|0|1|1|0|2|1|0|1|0|0|0|0',\n",
       " 'AC_nfe_seu=0',\n",
       " 'AN_nfe_seu=0',\n",
       " 'nhomalt_nfe_seu=0',\n",
       " 'controls_AC_afr_male=0',\n",
       " 'controls_AN_afr_male=120',\n",
       " 'controls_AF_afr_male=0.00000e+00',\n",
       " 'controls_nhomalt_afr_male=0',\n",
       " 'non_neuro_AC_eas_kor=0',\n",
       " 'non_neuro_AN_eas_kor=318',\n",
       " 'non_neuro_AF_eas_kor=0.00000e+00',\n",
       " 'non_neuro_nhomalt_eas_kor=0',\n",
       " 'non_topmed_AC_amr=2',\n",
       " 'non_topmed_AN_amr=712',\n",
       " 'non_topmed_AF_amr=2.80899e-03',\n",
       " 'non_topmed_nhomalt_amr=0',\n",
       " 'non_cancer_AC_asj_female=0',\n",
       " 'non_cancer_AN_asj_female=24',\n",
       " 'non_cancer_AF_asj_female=0.00000e+00',\n",
       " 'non_cancer_nhomalt_asj_female=0',\n",
       " 'AC_raw=8',\n",
       " 'AN_raw=46354',\n",
       " 'AF_raw=1.72585e-04',\n",
       " 'nhomalt_raw=0',\n",
       " 'AC_fin_female=0',\n",
       " 'AN_fin_female=14',\n",
       " 'AF_fin_female=0.00000e+00',\n",
       " 'nhomalt_fin_female=0',\n",
       " 'non_cancer_AC_oth_female=0',\n",
       " 'non_cancer_AN_oth_female=64',\n",
       " 'non_cancer_AF_oth_female=0.00000e+00',\n",
       " 'non_cancer_nhomalt_oth_female=0',\n",
       " 'AC_nfe_bgr=0',\n",
       " 'AN_nfe_bgr=0',\n",
       " 'nhomalt_nfe_bgr=0',\n",
       " 'non_neuro_AC_asj_female=0',\n",
       " 'non_neuro_AN_asj_female=4',\n",
       " 'non_neuro_AF_asj_female=0.00000e+00',\n",
       " 'non_neuro_nhomalt_asj_female=0',\n",
       " 'AC_sas_male=0',\n",
       " 'AN_sas_male=894',\n",
       " 'AF_sas_male=0.00000e+00',\n",
       " 'nhomalt_sas_male=0',\n",
       " 'non_neuro_AC_afr_male=0',\n",
       " 'non_neuro_AN_afr_male=260',\n",
       " 'non_neuro_AF_afr_male=0.00000e+00',\n",
       " 'non_neuro_nhomalt_afr_male=0',\n",
       " 'AC_afr_male=0',\n",
       " 'AN_afr_male=262',\n",
       " 'AF_afr_male=0.00000e+00',\n",
       " 'nhomalt_afr_male=0',\n",
       " 'AC_afr=0',\n",
       " 'AN_afr=624',\n",
       " 'AF_afr=0.00000e+00',\n",
       " 'nhomalt_afr=0',\n",
       " 'controls_AC_nfe_swe=0',\n",
       " 'controls_AN_nfe_swe=0',\n",
       " 'controls_nhomalt_nfe_swe=0',\n",
       " 'non_neuro_AC_afr_female=0',\n",
       " 'non_neuro_AN_afr_female=360',\n",
       " 'non_neuro_AF_afr_female=0.00000e+00',\n",
       " 'non_neuro_nhomalt_afr_female=0',\n",
       " 'non_topmed_AC_amr_female=2',\n",
       " 'non_topmed_AN_amr_female=348',\n",
       " 'non_topmed_AF_amr_female=5.74713e-03',\n",
       " 'non_topmed_nhomalt_amr_female=0',\n",
       " 'non_cancer_AC_female=2',\n",
       " 'non_cancer_AN_female=1850',\n",
       " 'non_cancer_AF_female=1.08108e-03',\n",
       " 'non_cancer_nhomalt_female=0',\n",
       " 'non_cancer_AC_nfe_onf=0',\n",
       " 'non_cancer_AN_nfe_onf=166',\n",
       " 'non_cancer_AF_nfe_onf=0.00000e+00',\n",
       " 'non_cancer_nhomalt_nfe_onf=0',\n",
       " 'non_cancer_AC_male=0',\n",
       " 'non_cancer_AN_male=2346',\n",
       " 'non_cancer_AF_male=0.00000e+00',\n",
       " 'non_cancer_nhomalt_male=0',\n",
       " 'non_topmed_AC_oth_female=0',\n",
       " 'non_topmed_AN_oth_female=62',\n",
       " 'non_topmed_AF_oth_female=0.00000e+00',\n",
       " 'non_topmed_nhomalt_oth_female=0',\n",
       " 'AC_eas_female=0',\n",
       " 'AN_eas_female=278',\n",
       " 'AF_eas_female=0.00000e+00',\n",
       " 'nhomalt_eas_female=0',\n",
       " 'non_cancer_AC_sas_female=0',\n",
       " 'non_cancer_AN_sas_female=344',\n",
       " 'non_cancer_AF_sas_female=0.00000e+00',\n",
       " 'non_cancer_nhomalt_sas_female=0',\n",
       " 'AC_afr_female=0',\n",
       " 'AN_afr_female=362',\n",
       " 'AF_afr_female=0.00000e+00',\n",
       " 'nhomalt_afr_female=0',\n",
       " 'AC_sas=0',\n",
       " 'AN_sas=1238',\n",
       " 'AF_sas=0.00000e+00',\n",
       " 'nhomalt_sas=0',\n",
       " 'non_neuro_AC_female=2',\n",
       " 'non_neuro_AN_female=1768',\n",
       " 'non_neuro_AF_female=1.13122e-03',\n",
       " 'non_neuro_nhomalt_female=0',\n",
       " 'controls_AC_afr=0',\n",
       " 'controls_AN_afr=304',\n",
       " 'controls_AF_afr=0.00000e+00',\n",
       " 'controls_nhomalt_afr=0',\n",
       " 'non_neuro_AC_eas_jpn=0',\n",
       " 'non_neuro_AN_eas_jpn=0',\n",
       " 'non_neuro_nhomalt_eas_jpn=0',\n",
       " 'AC_nfe_onf=0',\n",
       " 'AN_nfe_onf=166',\n",
       " 'AF_nfe_onf=0.00000e+00',\n",
       " 'nhomalt_nfe_onf=0',\n",
       " 'non_cancer_AC_amr_male=0',\n",
       " 'non_cancer_AN_amr_male=368',\n",
       " 'non_cancer_AF_amr_male=0.00000e+00',\n",
       " 'non_cancer_nhomalt_amr_male=0',\n",
       " 'controls_AC_fin_male=0',\n",
       " 'controls_AN_fin_male=0',\n",
       " 'controls_nhomalt_fin_male=0',\n",
       " 'non_neuro_AC_nfe_nwe=0',\n",
       " 'non_neuro_AN_nfe_nwe=540',\n",
       " 'non_neuro_AF_nfe_nwe=0.00000e+00',\n",
       " 'non_neuro_nhomalt_nfe_nwe=0',\n",
       " 'AC_fin_male=0',\n",
       " 'AN_fin_male=22',\n",
       " 'AF_fin_male=0.00000e+00',\n",
       " 'nhomalt_fin_male=0',\n",
       " 'AC_nfe_female=0',\n",
       " 'AN_nfe_female=416',\n",
       " 'AF_nfe_female=0.00000e+00',\n",
       " 'nhomalt_nfe_female=0',\n",
       " 'AC_amr=2',\n",
       " 'AN_amr=716',\n",
       " 'AF_amr=2.79330e-03',\n",
       " 'nhomalt_amr=0',\n",
       " 'non_topmed_AC_nfe_male=0',\n",
       " 'non_topmed_AN_nfe_male=372',\n",
       " 'non_topmed_AF_nfe_male=0.00000e+00',\n",
       " 'non_topmed_nhomalt_nfe_male=0',\n",
       " 'non_neuro_AC_sas=0',\n",
       " 'non_neuro_AN_sas=1238',\n",
       " 'non_neuro_AF_sas=0.00000e+00',\n",
       " 'non_neuro_nhomalt_sas=0',\n",
       " 'non_cancer_AC_fin_male=0',\n",
       " 'non_cancer_AN_fin_male=22',\n",
       " 'non_cancer_AF_fin_male=0.00000e+00',\n",
       " 'non_cancer_nhomalt_fin_male=0',\n",
       " 'non_cancer_AC_nfe_seu=0',\n",
       " 'non_cancer_AN_nfe_seu=0',\n",
       " 'non_cancer_nhomalt_nfe_seu=0',\n",
       " 'AC_eas=0',\n",
       " 'AN_eas=556',\n",
       " 'AF_eas=0.00000e+00',\n",
       " 'nhomalt_eas=0',\n",
       " 'nhomalt=0',\n",
       " 'non_neuro_AC_nfe_female=0',\n",
       " 'non_neuro_AN_nfe_female=364',\n",
       " 'non_neuro_AF_nfe_female=0.00000e+00',\n",
       " 'non_neuro_nhomalt_nfe_female=0',\n",
       " 'non_neuro_AC_afr=0',\n",
       " 'non_neuro_AN_afr=620',\n",
       " 'non_neuro_AF_afr=0.00000e+00',\n",
       " 'non_neuro_nhomalt_afr=0',\n",
       " 'controls_AC_raw=4',\n",
       " 'controls_AN_raw=19152',\n",
       " 'controls_AF_raw=2.08855e-04',\n",
       " 'controls_nhomalt_raw=0',\n",
       " 'non_cancer_AC_eas=0',\n",
       " 'non_cancer_AN_eas=556',\n",
       " 'non_cancer_AF_eas=0.00000e+00',\n",
       " 'non_cancer_nhomalt_eas=0',\n",
       " 'non_cancer_AC_amr_female=2',\n",
       " 'non_cancer_AN_amr_female=348',\n",
       " 'non_cancer_AF_amr_female=5.74713e-03',\n",
       " 'non_cancer_nhomalt_amr_female=0',\n",
       " 'non_neuro_AC_nfe_swe=0',\n",
       " 'non_neuro_AN_nfe_swe=0',\n",
       " 'non_neuro_nhomalt_nfe_swe=0',\n",
       " 'controls_AC_male=0',\n",
       " 'controls_AN_male=1074',\n",
       " 'controls_AF_male=0.00000e+00',\n",
       " 'controls_nhomalt_male=0',\n",
       " 'non_topmed_AC_male=0',\n",
       " 'non_topmed_AN_male=2276',\n",
       " 'non_topmed_AF_male=0.00000e+00',\n",
       " 'non_topmed_nhomalt_male=0',\n",
       " 'controls_AC_eas_jpn=0',\n",
       " 'controls_AN_eas_jpn=0',\n",
       " 'controls_nhomalt_eas_jpn=0',\n",
       " 'controls_AC_nfe_female=0',\n",
       " 'controls_AN_nfe_female=198',\n",
       " 'controls_AF_nfe_female=0.00000e+00',\n",
       " 'controls_nhomalt_nfe_female=0',\n",
       " 'non_neuro_AC_amr=2',\n",
       " 'non_neuro_AN_amr=708',\n",
       " 'non_neuro_AF_amr=2.82486e-03',\n",
       " 'non_neuro_nhomalt_amr=0',\n",
       " 'non_neuro_AC_eas_female=0',\n",
       " 'non_neuro_AN_eas_female=278',\n",
       " 'non_neuro_AF_eas_female=0.00000e+00',\n",
       " 'non_neuro_nhomalt_eas_female=0',\n",
       " 'AC_asj_male=0',\n",
       " 'AN_asj_male=18',\n",
       " 'AF_asj_male=0.00000e+00',\n",
       " 'nhomalt_asj_male=0',\n",
       " 'controls_AC_nfe_male=0',\n",
       " 'controls_AN_nfe_male=182',\n",
       " 'controls_AF_nfe_male=0.00000e+00',\n",
       " 'controls_nhomalt_nfe_male=0',\n",
       " 'non_neuro_AC_fin=0',\n",
       " 'non_neuro_AN_fin=36',\n",
       " 'non_neuro_AF_fin=0.00000e+00',\n",
       " 'non_neuro_nhomalt_fin=0',\n",
       " 'non_topmed_AC_sas=0',\n",
       " 'non_topmed_AN_sas=1238',\n",
       " 'non_topmed_AF_sas=0.00000e+00',\n",
       " 'non_topmed_nhomalt_sas=0',\n",
       " 'non_cancer_AC_nfe_female=0',\n",
       " 'non_cancer_AN_nfe_female=416',\n",
       " 'non_cancer_AF_nfe_female=0.00000e+00',\n",
       " 'non_cancer_nhomalt_nfe_female=0',\n",
       " 'AC_oth_female=0',\n",
       " 'AN_oth_female=64',\n",
       " 'AF_oth_female=0.00000e+00',\n",
       " 'nhomalt_oth_female=0',\n",
       " 'non_cancer_AC_asj=0',\n",
       " 'non_cancer_AN_asj=42',\n",
       " 'non_cancer_AF_asj=0.00000e+00',\n",
       " 'non_cancer_nhomalt_asj=0',\n",
       " 'AC_nfe_swe=0',\n",
       " 'AN_nfe_swe=0',\n",
       " 'nhomalt_nfe_swe=0',\n",
       " 'controls_AC_nfe=0',\n",
       " 'controls_AN_nfe=380',\n",
       " 'controls_AF_nfe=0.00000e+00',\n",
       " 'controls_nhomalt_nfe=0',\n",
       " 'controls_AC_oth_female=0',\n",
       " 'controls_AN_oth_female=34',\n",
       " 'controls_AF_oth_female=0.00000e+00',\n",
       " 'controls_nhomalt_oth_female=0',\n",
       " 'controls_AC_asj=0',\n",
       " 'controls_AN_asj=14',\n",
       " 'controls_AF_asj=0.00000e+00',\n",
       " 'controls_nhomalt_asj=0',\n",
       " 'non_neuro_AC_amr_male=0',\n",
       " 'non_neuro_AN_amr_male=364',\n",
       " 'non_neuro_AF_amr_male=0.00000e+00',\n",
       " 'non_neuro_nhomalt_amr_male=0',\n",
       " 'controls_AC_nfe_nwe=0',\n",
       " 'controls_AN_nfe_nwe=294',\n",
       " 'controls_AF_nfe_nwe=0.00000e+00',\n",
       " 'controls_nhomalt_nfe_nwe=0',\n",
       " 'AC_nfe_nwe=0',\n",
       " 'AN_nfe_nwe=672',\n",
       " 'AF_nfe_nwe=0.00000e+00',\n",
       " 'nhomalt_nfe_nwe=0',\n",
       " 'controls_AC_nfe_seu=0',\n",
       " 'controls_AN_nfe_seu=0',\n",
       " 'controls_nhomalt_nfe_seu=0',\n",
       " 'controls_AC_sas_female=0',\n",
       " 'controls_AN_sas_female=188',\n",
       " 'controls_AF_sas_female=0.00000e+00',\n",
       " 'controls_nhomalt_sas_female=0',\n",
       " 'non_neuro_AC_amr_female=2',\n",
       " 'non_neuro_AN_amr_female=344',\n",
       " 'non_neuro_AF_amr_female=5.81395e-03',\n",
       " 'non_neuro_nhomalt_amr_female=0',\n",
       " 'non_cancer_AC_eas_jpn=0',\n",
       " 'non_cancer_AN_eas_jpn=0',\n",
       " 'non_cancer_nhomalt_eas_jpn=0',\n",
       " 'non_neuro_AC_nfe_onf=0',\n",
       " 'non_neuro_AN_nfe_onf=146',\n",
       " 'non_neuro_AF_nfe_onf=0.00000e+00',\n",
       " 'non_neuro_nhomalt_nfe_onf=0',\n",
       " 'non_topmed_AC_eas_male=0',\n",
       " 'non_topmed_AN_eas_male=278',\n",
       " 'non_topmed_AF_eas_male=0.00000e+00',\n",
       " 'non_topmed_nhomalt_eas_male=0',\n",
       " 'AC_eas_jpn=0',\n",
       " 'AN_eas_jpn=0',\n",
       " 'nhomalt_eas_jpn=0',\n",
       " 'non_cancer_AC_afr_male=0',\n",
       " 'non_cancer_AN_afr_male=262',\n",
       " 'non_cancer_AF_afr_male=0.00000e+00',\n",
       " 'non_cancer_nhomalt_afr_male=0',\n",
       " 'non_cancer_AC_afr=0',\n",
       " 'non_cancer_AN_afr=624',\n",
       " 'non_cancer_AF_afr=0.00000e+00',\n",
       " 'non_cancer_nhomalt_afr=0',\n",
       " 'controls_AC_amr_female=0',\n",
       " 'controls_AN_amr_female=154',\n",
       " 'controls_AF_amr_female=0.00000e+00',\n",
       " 'controls_nhomalt_amr_female=0',\n",
       " 'non_neuro_AC_fin_male=0',\n",
       " 'non_neuro_AN_fin_male=22',\n",
       " 'non_neuro_AF_fin_male=0.00000e+00',\n",
       " 'non_neuro_nhomalt_fin_male=0',\n",
       " 'AC_female=2',\n",
       " 'AN_female=1850',\n",
       " 'AF_female=1.08108e-03',\n",
       " 'nhomalt_female=0',\n",
       " 'non_neuro_AC_nfe_bgr=0',\n",
       " 'non_neuro_AN_nfe_bgr=0',\n",
       " 'non_neuro_nhomalt_nfe_bgr=0',\n",
       " 'non_neuro_AC_oth_male=0',\n",
       " 'non_neuro_AN_oth_male=68',\n",
       " 'non_neuro_AF_oth_male=0.00000e+00',\n",
       " 'non_neuro_nhomalt_oth_male=0',\n",
       " 'non_topmed_AC_nfe_est=0',\n",
       " 'non_topmed_AN_nfe_est=4',\n",
       " 'non_topmed_AF_nfe_est=0.00000e+00',\n",
       " 'non_topmed_nhomalt_nfe_est=0',\n",
       " 'non_topmed_AC_nfe_nwe=0',\n",
       " 'non_topmed_AN_nfe_nwe=592',\n",
       " 'non_topmed_AF_nfe_nwe=0.00000e+00',\n",
       " 'non_topmed_nhomalt_nfe_nwe=0',\n",
       " 'non_topmed_AC_amr_male=0',\n",
       " 'non_topmed_AN_amr_male=364',\n",
       " 'non_topmed_AF_amr_male=0.00000e+00',\n",
       " 'non_topmed_nhomalt_amr_male=0',\n",
       " 'non_cancer_AC_amr=2',\n",
       " 'non_cancer_AN_amr=716',\n",
       " 'non_cancer_AF_amr=2.79330e-03',\n",
       " 'non_cancer_nhomalt_amr=0',\n",
       " 'non_topmed_AC_nfe_swe=0',\n",
       " 'non_topmed_AN_nfe_swe=0',\n",
       " 'non_topmed_nhomalt_nfe_swe=0',\n",
       " 'non_topmed_AC_nfe_onf=0',\n",
       " 'non_topmed_AN_nfe_onf=134',\n",
       " 'non_topmed_AF_nfe_onf=0.00000e+00',\n",
       " 'non_topmed_nhomalt_nfe_onf=0',\n",
       " 'controls_AC_eas_kor=0',\n",
       " 'controls_AN_eas_kor=158',\n",
       " 'controls_AF_eas_kor=0.00000e+00',\n",
       " 'controls_nhomalt_eas_kor=0',\n",
       " 'non_topmed_AC_eas_oea=0',\n",
       " 'non_topmed_AN_eas_oea=238',\n",
       " 'non_topmed_AF_eas_oea=0.00000e+00',\n",
       " 'non_topmed_nhomalt_eas_oea=0',\n",
       " 'controls_AC_eas_male=0',\n",
       " 'controls_AN_eas_male=130',\n",
       " 'controls_AF_eas_male=0.00000e+00',\n",
       " 'controls_nhomalt_eas_male=0',\n",
       " 'controls_AC_oth_male=0',\n",
       " 'controls_AN_oth_male=32',\n",
       " 'controls_AF_oth_male=0.00000e+00',\n",
       " 'controls_nhomalt_oth_male=0',\n",
       " 'non_topmed_AC=2',\n",
       " 'non_topmed_AN=4050',\n",
       " 'non_topmed_AF=4.93827e-04',\n",
       " 'non_topmed_nhomalt=0',\n",
       " 'controls_AC_fin=0',\n",
       " 'controls_AN_fin=0',\n",
       " 'controls_nhomalt_fin=0',\n",
       " 'AC_eas_kor=0',\n",
       " 'AN_eas_kor=318',\n",
       " 'AF_eas_kor=0.00000e+00',\n",
       " 'nhomalt_eas_kor=0',\n",
       " 'non_neuro_AC_nfe=0',\n",
       " 'non_neuro_AN_nfe=692',\n",
       " 'non_neuro_AF_nfe=0.00000e+00',\n",
       " 'non_neuro_nhomalt_nfe=0',\n",
       " 'non_neuro_AC_fin_female=0',\n",
       " 'non_neuro_AN_fin_female=14',\n",
       " 'non_neuro_AF_fin_female=0.00000e+00',\n",
       " 'non_neuro_nhomalt_fin_female=0',\n",
       " 'non_cancer_AC_nfe_male=0',\n",
       " 'non_cancer_AN_nfe_male=428',\n",
       " 'non_cancer_AF_nfe_male=0.00000e+00',\n",
       " 'non_cancer_nhomalt_nfe_male=0',\n",
       " 'controls_AC_eas_oea=0',\n",
       " 'controls_AN_eas_oea=88',\n",
       " 'controls_AF_eas_oea=0.00000e+00',\n",
       " 'controls_nhomalt_eas_oea=0',\n",
       " 'non_topmed_AC_nfe_seu=0',\n",
       " 'non_topmed_AN_nfe_seu=0',\n",
       " 'non_topmed_nhomalt_nfe_seu=0',\n",
       " 'controls_AC_eas_female=0',\n",
       " 'controls_AN_eas_female=116',\n",
       " 'controls_AF_eas_female=0.00000e+00',\n",
       " 'controls_nhomalt_eas_female=0',\n",
       " 'non_topmed_AC_asj=0',\n",
       " 'non_topmed_AN_asj=40',\n",
       " 'non_topmed_AF_asj=0.00000e+00',\n",
       " 'non_topmed_nhomalt_asj=0',\n",
       " 'controls_AC_nfe_onf=0',\n",
       " 'controls_AN_nfe_onf=84',\n",
       " 'controls_AF_nfe_onf=0.00000e+00',\n",
       " 'controls_nhomalt_nfe_onf=0',\n",
       " 'non_neuro_AC=2',\n",
       " 'non_neuro_AN=3998',\n",
       " 'non_neuro_AF=5.00250e-04',\n",
       " 'non_neuro_nhomalt=0',\n",
       " 'AC_eas_oea=0',\n",
       " 'AN_eas_oea=238',\n",
       " 'AF_eas_oea=0.00000e+00',\n",
       " 'nhomalt_eas_oea=0',\n",
       " 'non_topmed_AC_nfe=0',\n",
       " 'non_topmed_AN_nfe=730',\n",
       " 'non_topmed_AF_nfe=0.00000e+00',\n",
       " 'non_topmed_nhomalt_nfe=0',\n",
       " 'non_cancer_AC_oth=0',\n",
       " 'non_cancer_AN_oth=140',\n",
       " 'non_cancer_AF_oth=0.00000e+00',\n",
       " 'non_cancer_nhomalt_oth=0',\n",
       " 'non_topmed_AC_raw=8',\n",
       " 'non_topmed_AN_raw=45148',\n",
       " 'non_topmed_AF_raw=1.77195e-04',\n",
       " 'non_topmed_nhomalt_raw=0',\n",
       " 'non_neuro_AC_nfe_est=0',\n",
       " 'non_neuro_AN_nfe_est=6',\n",
       " 'non_neuro_AF_nfe_est=0.00000e+00',\n",
       " 'non_neuro_nhomalt_nfe_est=0',\n",
       " 'non_topmed_AC_oth_male=0',\n",
       " 'non_topmed_AN_oth_male=72',\n",
       " 'non_topmed_AF_oth_male=0.00000e+00',\n",
       " 'non_topmed_nhomalt_oth_male=0',\n",
       " 'non_cancer_AC_oth_male=0',\n",
       " 'non_cancer_AN_oth_male=76',\n",
       " 'non_cancer_AF_oth_male=0.00000e+00',\n",
       " 'non_cancer_nhomalt_oth_male=0',\n",
       " 'AC_nfe_est=0',\n",
       " 'AN_nfe_est=6',\n",
       " 'AF_nfe_est=0.00000e+00',\n",
       " 'nhomalt_nfe_est=0',\n",
       " 'non_cancer_AC_afr_female=0',\n",
       " 'non_cancer_AN_afr_female=362',\n",
       " 'non_cancer_AF_afr_female=0.00000e+00',\n",
       " 'non_cancer_nhomalt_afr_female=0',\n",
       " 'non_topmed_AC_afr_male=0',\n",
       " 'non_topmed_AN_afr_male=256',\n",
       " 'non_topmed_AF_afr_male=0.00000e+00',\n",
       " 'non_topmed_nhomalt_afr_male=0',\n",
       " 'AC_eas_male=0',\n",
       " 'AN_eas_male=278',\n",
       " 'AF_eas_male=0.00000e+00',\n",
       " 'nhomalt_eas_male=0',\n",
       " 'controls_AC_eas=0',\n",
       " 'controls_AN_eas=246',\n",
       " 'controls_AF_eas=0.00000e+00',\n",
       " 'controls_nhomalt_eas=0',\n",
       " 'non_neuro_AC_eas_male=0',\n",
       " 'non_neuro_AN_eas_male=278',\n",
       " 'non_neuro_AF_eas_male=0.00000e+00',\n",
       " 'non_neuro_nhomalt_eas_male=0',\n",
       " 'non_cancer_AC_nfe_nwe=0',\n",
       " 'non_cancer_AN_nfe_nwe=672',\n",
       " 'non_cancer_AF_nfe_nwe=0.00000e+00',\n",
       " 'non_cancer_nhomalt_nfe_nwe=0',\n",
       " 'controls_AC_sas=0',\n",
       " 'controls_AN_sas=614',\n",
       " 'controls_AF_sas=0.00000e+00',\n",
       " 'controls_nhomalt_sas=0',\n",
       " 'non_neuro_AC_sas_male=0',\n",
       " 'non_neuro_AN_sas_male=894',\n",
       " 'non_neuro_AF_sas_male=0.00000e+00',\n",
       " 'non_neuro_nhomalt_sas_male=0',\n",
       " 'non_neuro_AC_asj_male=0',\n",
       " 'non_neuro_AN_asj_male=16',\n",
       " 'non_neuro_AF_asj_male=0.00000e+00',\n",
       " 'non_neuro_nhomalt_asj_male=0',\n",
       " 'non_cancer_AC_nfe_bgr=0',\n",
       " 'non_cancer_AN_nfe_bgr=0',\n",
       " 'non_cancer_nhomalt_nfe_bgr=0',\n",
       " 'controls_AC_oth=0',\n",
       " 'controls_AN_oth=66',\n",
       " 'controls_AF_oth=0.00000e+00',\n",
       " 'controls_nhomalt_oth=0',\n",
       " 'non_cancer_AC_eas_female=0',\n",
       " 'non_cancer_AN_eas_female=278',\n",
       " 'non_cancer_AF_eas_female=0.00000e+00',\n",
       " 'non_cancer_nhomalt_eas_female=0',\n",
       " 'AC_nfe=0',\n",
       " 'AN_nfe=844',\n",
       " 'AF_nfe=0.00000e+00',\n",
       " 'nhomalt_nfe=0',\n",
       " 'non_topmed_AC_female=2',\n",
       " 'non_topmed_AN_female=1774',\n",
       " 'non_topmed_AF_female=1.12740e-03',\n",
       " 'non_topmed_nhomalt_female=0',\n",
       " 'non_neuro_AC_asj=0',\n",
       " 'non_neuro_AN_asj=20',\n",
       " 'non_neuro_AF_asj=0.00000e+00',\n",
       " 'non_neuro_nhomalt_asj=0',\n",
       " 'non_topmed_AC_eas_female=0',\n",
       " 'non_topmed_AN_eas_female=278',\n",
       " 'non_topmed_AF_eas_female=0.00000e+00',\n",
       " 'non_topmed_nhomalt_eas_female=0',\n",
       " 'non_neuro_AC_raw=7',\n",
       " 'non_neuro_AN_raw=39484',\n",
       " 'non_neuro_AF_raw=1.77287e-04',\n",
       " 'non_neuro_nhomalt_raw=0',\n",
       " 'non_topmed_AC_eas=0',\n",
       " 'non_topmed_AN_eas=556',\n",
       " 'non_topmed_AF_eas=0.00000e+00',\n",
       " 'non_topmed_nhomalt_eas=0',\n",
       " 'non_topmed_AC_fin_male=0',\n",
       " 'non_topmed_AN_fin_male=22',\n",
       " 'non_topmed_AF_fin_male=0.00000e+00',\n",
       " 'non_topmed_nhomalt_fin_male=0',\n",
       " 'non_cancer_AC_asj_male=0',\n",
       " 'non_cancer_AN_asj_male=18',\n",
       " 'non_cancer_AF_asj_male=0.00000e+00',\n",
       " 'non_cancer_nhomalt_asj_male=0',\n",
       " 'AC_fin=0',\n",
       " 'AN_fin=36',\n",
       " 'AF_fin=0.00000e+00',\n",
       " 'nhomalt_fin=0',\n",
       " 'AC_nfe_male=0',\n",
       " 'AN_nfe_male=428',\n",
       " 'AF_nfe_male=0.00000e+00',\n",
       " 'nhomalt_nfe_male=0',\n",
       " 'non_topmed_AC_eas_kor=0',\n",
       " 'non_topmed_AN_eas_kor=318',\n",
       " 'non_topmed_AF_eas_kor=0.00000e+00',\n",
       " 'non_topmed_nhomalt_eas_kor=0',\n",
       " 'controls_AC_amr_male=0',\n",
       " 'controls_AN_amr_male=172',\n",
       " 'controls_AF_amr_male=0.00000e+00',\n",
       " 'controls_nhomalt_amr_male=0',\n",
       " 'non_neuro_AC_eas_oea=0',\n",
       " 'non_neuro_AN_eas_oea=238',\n",
       " 'non_neuro_AF_eas_oea=0.00000e+00',\n",
       " 'non_neuro_nhomalt_eas_oea=0',\n",
       " 'AC_sas_female=0',\n",
       " 'AN_sas_female=344',\n",
       " 'AF_sas_female=0.00000e+00',\n",
       " 'nhomalt_sas_female=0',\n",
       " 'controls_AC_afr_female=0',\n",
       " 'controls_AN_afr_female=184',\n",
       " 'controls_AF_afr_female=0.00000e+00',\n",
       " 'controls_nhomalt_afr_female=0',\n",
       " 'controls_AC_amr=0',\n",
       " 'controls_AN_amr=326',\n",
       " 'controls_AF_amr=0.00000e+00',\n",
       " 'controls_nhomalt_amr=0',\n",
       " 'non_topmed_AC_eas_jpn=0',\n",
       " 'non_topmed_AN_eas_jpn=0',\n",
       " 'non_topmed_nhomalt_eas_jpn=0',\n",
       " 'AC_asj_female=0',\n",
       " 'AN_asj_female=24',\n",
       " 'AF_asj_female=0.00000e+00',\n",
       " 'nhomalt_asj_female=0',\n",
       " 'non_topmed_AC_nfe_bgr=0',\n",
       " 'non_topmed_AN_nfe_bgr=0',\n",
       " 'non_topmed_nhomalt_nfe_bgr=0',\n",
       " 'non_cancer_AC_nfe_est=0',\n",
       " 'non_cancer_AN_nfe_est=6',\n",
       " 'non_cancer_AF_nfe_est=0.00000e+00',\n",
       " 'non_cancer_nhomalt_nfe_est=0',\n",
       " 'non_neuro_AC_eas=0',\n",
       " 'non_neuro_AN_eas=556',\n",
       " 'non_neuro_AF_eas=0.00000e+00',\n",
       " 'non_neuro_nhomalt_eas=0',\n",
       " 'non_cancer_AC_nfe=0',\n",
       " 'non_cancer_AN_nfe=844',\n",
       " 'non_cancer_AF_nfe=0.00000e+00',\n",
       " 'non_cancer_nhomalt_nfe=0',\n",
       " 'non_neuro_AC_male=0',\n",
       " 'non_neuro_AN_male=2230',\n",
       " 'non_neuro_AF_male=0.00000e+00',\n",
       " 'non_neuro_nhomalt_male=0',\n",
       " 'non_neuro_AC_sas_female=0',\n",
       " 'non_neuro_AN_sas_female=344',\n",
       " 'non_neuro_AF_sas_female=0.00000e+00',\n",
       " 'non_neuro_nhomalt_sas_female=0',\n",
       " 'AC_asj=0',\n",
       " 'AN_asj=42',\n",
       " 'AF_asj=0.00000e+00',\n",
       " 'nhomalt_asj=0',\n",
       " 'controls_AC_nfe_est=0',\n",
       " 'controls_AN_nfe_est=2',\n",
       " 'controls_AF_nfe_est=0.00000e+00',\n",
       " 'controls_nhomalt_nfe_est=0',\n",
       " 'non_topmed_AC_asj_female=0',\n",
       " 'non_topmed_AN_asj_female=22',\n",
       " 'non_topmed_AF_asj_female=0.00000e+00',\n",
       " 'non_topmed_nhomalt_asj_female=0',\n",
       " 'non_cancer_AC_nfe_swe=0',\n",
       " 'non_cancer_AN_nfe_swe=0',\n",
       " 'non_cancer_nhomalt_nfe_swe=0',\n",
       " 'non_cancer_AC=2',\n",
       " 'non_cancer_AN=4196',\n",
       " 'non_cancer_AF=4.76644e-04',\n",
       " 'non_cancer_nhomalt=0',\n",
       " 'non_topmed_AC_oth=0',\n",
       " 'non_topmed_AN_oth=134',\n",
       " 'non_topmed_AF_oth=0.00000e+00',\n",
       " 'non_topmed_nhomalt_oth=0',\n",
       " 'non_topmed_AC_fin_female=0',\n",
       " 'non_topmed_AN_fin_female=14',\n",
       " 'non_topmed_AF_fin_female=0.00000e+00',\n",
       " 'non_topmed_nhomalt_fin_female=0',\n",
       " 'non_cancer_AC_fin_female=0',\n",
       " 'non_cancer_AN_fin_female=14',\n",
       " 'non_cancer_AF_fin_female=0.00000e+00',\n",
       " 'non_cancer_nhomalt_fin_female=0',\n",
       " 'AC_oth=0',\n",
       " 'AN_oth=140',\n",
       " 'AF_oth=0.00000e+00',\n",
       " 'nhomalt_oth=0',\n",
       " 'non_neuro_AC_nfe_male=0',\n",
       " 'non_neuro_AN_nfe_male=328',\n",
       " 'non_neuro_AF_nfe_male=0.00000e+00',\n",
       " 'non_neuro_nhomalt_nfe_male=0',\n",
       " 'controls_AC_female=0',\n",
       " 'controls_AN_female=876',\n",
       " 'controls_AF_female=0.00000e+00',\n",
       " 'controls_nhomalt_female=0',\n",
       " 'non_cancer_AC_fin=0',\n",
       " 'non_cancer_AN_fin=36',\n",
       " 'non_cancer_AF_fin=0.00000e+00',\n",
       " 'non_cancer_nhomalt_fin=0',\n",
       " 'non_topmed_AC_fin=0',\n",
       " 'non_topmed_AN_fin=36',\n",
       " 'non_topmed_AF_fin=0.00000e+00',\n",
       " 'non_topmed_nhomalt_fin=0',\n",
       " 'non_cancer_AC_eas_oea=0',\n",
       " 'non_cancer_AN_eas_oea=238',\n",
       " 'non_cancer_AF_eas_oea=0.00000e+00',\n",
       " 'non_cancer_nhomalt_eas_oea=0',\n",
       " 'non_topmed_AC_nfe_female=0',\n",
       " 'non_topmed_AN_nfe_female=358',\n",
       " 'non_topmed_AF_nfe_female=0.00000e+00',\n",
       " 'non_topmed_nhomalt_nfe_female=0',\n",
       " 'non_cancer_AC_sas_male=0',\n",
       " 'non_cancer_AN_sas_male=894',\n",
       " 'non_cancer_AF_sas_male=0.00000e+00',\n",
       " 'non_cancer_nhomalt_sas_male=0',\n",
       " 'controls_AC_asj_male=0',\n",
       " 'controls_AN_asj_male=12',\n",
       " 'controls_AF_asj_male=0.00000e+00',\n",
       " 'controls_nhomalt_asj_male=0',\n",
       " 'non_cancer_AC_raw=8',\n",
       " 'non_cancer_AN_raw=46350',\n",
       " 'non_cancer_AF_raw=1.72600e-04',\n",
       " 'non_cancer_nhomalt_raw=0',\n",
       " 'non_cancer_AC_eas_male=0',\n",
       " 'non_cancer_AN_eas_male=278',\n",
       " 'non_cancer_AF_eas_male=0.00000e+00',\n",
       " 'non_cancer_nhomalt_eas_male=0',\n",
       " 'non_topmed_AC_asj_male=0',\n",
       " 'non_topmed_AN_asj_male=18',\n",
       " 'non_topmed_AF_asj_male=0.00000e+00',\n",
       " 'non_topmed_nhomalt_asj_male=0',\n",
       " 'non_neuro_AC_oth=0',\n",
       " 'non_neuro_AN_oth=128',\n",
       " 'non_neuro_AF_oth=0.00000e+00',\n",
       " 'non_neuro_nhomalt_oth=0',\n",
       " 'AC_male=0',\n",
       " 'AN_male=2346',\n",
       " 'AF_male=0.00000e+00',\n",
       " 'nhomalt_male=0',\n",
       " 'controls_AC_fin_female=0',\n",
       " 'controls_AN_fin_female=0',\n",
       " 'controls_nhomalt_fin_female=0',\n",
       " 'controls_AC_nfe_bgr=0',\n",
       " 'controls_AN_nfe_bgr=0',\n",
       " 'controls_nhomalt_nfe_bgr=0',\n",
       " 'controls_AC_asj_female=0',\n",
       " 'controls_AN_asj_female=2',\n",
       " 'controls_AF_asj_female=0.00000e+00',\n",
       " 'controls_nhomalt_asj_female=0',\n",
       " 'AC_amr_male=0',\n",
       " 'AN_amr_male=368',\n",
       " 'AF_amr_male=0.00000e+00',\n",
       " 'nhomalt_amr_male=0',\n",
       " 'AC_amr_female=2',\n",
       " 'AN_amr_female=348',\n",
       " 'AF_amr_female=5.74713e-03',\n",
       " 'nhomalt_amr_female=0',\n",
       " 'non_topmed_AC_sas_male=0',\n",
       " 'non_topmed_AN_sas_male=894',\n",
       " 'non_topmed_AF_sas_male=0.00000e+00',\n",
       " 'non_topmed_nhomalt_sas_male=0',\n",
       " 'AC_oth_male=0',\n",
       " 'AN_oth_male=76',\n",
       " 'AF_oth_male=0.00000e+00',\n",
       " 'nhomalt_oth_male=0',\n",
       " 'non_cancer_AC_sas=0',\n",
       " 'non_cancer_AN_sas=1238',\n",
       " 'non_cancer_AF_sas=0.00000e+00',\n",
       " 'non_cancer_nhomalt_sas=0',\n",
       " 'non_neuro_AC_nfe_seu=0',\n",
       " 'non_neuro_AN_nfe_seu=0',\n",
       " 'non_neuro_nhomalt_nfe_seu=0',\n",
       " 'non_cancer_AC_eas_kor=0',\n",
       " 'non_cancer_AN_eas_kor=318',\n",
       " 'non_cancer_AF_eas_kor=0.00000e+00',\n",
       " 'non_cancer_nhomalt_eas_kor=0',\n",
       " 'non_topmed_AC_afr_female=0',\n",
       " 'non_topmed_AN_afr_female=348',\n",
       " 'non_topmed_AF_afr_female=0.00000e+00',\n",
       " 'non_topmed_nhomalt_afr_female=0',\n",
       " 'controls_AC_sas_male=0',\n",
       " 'controls_AN_sas_male=426',\n",
       " 'controls_AF_sas_male=0.00000e+00',\n",
       " 'controls_nhomalt_sas_male=0',\n",
       " 'non_topmed_AC_sas_female=0',\n",
       " 'non_topmed_AN_sas_female=344',\n",
       " 'non_topmed_AF_sas_female=0.00000e+00',\n",
       " 'non_topmed_nhomalt_sas_female=0',\n",
       " 'non_topmed_AC_afr=0',\n",
       " 'non_topmed_AN_afr=604',\n",
       " 'non_topmed_AF_afr=0.00000e+00',\n",
       " 'non_topmed_nhomalt_afr=0',\n",
       " 'controls_AC=0',\n",
       " 'controls_AN=1950',\n",
       " 'controls_AF=0.00000e+00',\n",
       " 'controls_nhomalt=0',\n",
       " 'non_neuro_AC_oth_female=0',\n",
       " 'non_neuro_AN_oth_female=60',\n",
       " 'non_neuro_AF_oth_female=0.00000e+00',\n",
       " 'non_neuro_nhomalt_oth_female=0',\n",
       " 'non_topmed_faf95_amr=4.98630e-04',\n",
       " 'non_topmed_faf99_amr=4.98230e-04',\n",
       " 'faf95_afr=0.00000e+00',\n",
       " 'faf99_afr=0.00000e+00',\n",
       " 'faf95_sas=0.00000e+00',\n",
       " 'faf99_sas=0.00000e+00',\n",
       " 'controls_faf95_afr=0.00000e+00',\n",
       " 'controls_faf99_afr=0.00000e+00',\n",
       " 'faf95_amr=4.95930e-04',\n",
       " 'faf99_amr=4.95370e-04',\n",
       " 'non_neuro_faf95_sas=0.00000e+00',\n",
       " 'non_neuro_faf99_sas=0.00000e+00',\n",
       " 'faf95_eas=0.00000e+00',\n",
       " 'faf99_eas=0.00000e+00',\n",
       " 'faf95=8.40400e-05',\n",
       " 'faf99=8.40500e-05',\n",
       " 'non_neuro_faf95_afr=0.00000e+00',\n",
       " 'non_neuro_faf99_afr=0.00000e+00',\n",
       " 'non_cancer_faf95_eas=0.00000e+00',\n",
       " 'non_cancer_faf99_eas=0.00000e+00',\n",
       " 'non_neuro_faf95_amr=5.01060e-04',\n",
       " 'non_neuro_faf99_amr=5.01220e-04',\n",
       " 'non_topmed_faf95_sas=0.00000e+00',\n",
       " 'non_topmed_faf99_sas=0.00000e+00',\n",
       " 'controls_faf95_nfe=0.00000e+00',\n",
       " 'controls_faf99_nfe=0.00000e+00',\n",
       " 'non_cancer_faf95_afr=0.00000e+00',\n",
       " 'non_cancer_faf99_afr=0.00000e+00',\n",
       " 'non_cancer_faf95_amr=4.95930e-04',\n",
       " 'non_cancer_faf99_amr=4.95370e-04',\n",
       " 'non_topmed_faf95=8.68300e-05',\n",
       " 'non_topmed_faf99=8.70800e-05',\n",
       " 'non_neuro_faf95_nfe=0.00000e+00',\n",
       " 'non_neuro_faf99_nfe=0.00000e+00',\n",
       " 'non_neuro_faf95=8.88700e-05',\n",
       " 'non_neuro_faf99=8.84700e-05',\n",
       " 'non_topmed_faf95_nfe=0.00000e+00',\n",
       " 'non_topmed_faf99_nfe=0.00000e+00',\n",
       " 'controls_faf95_eas=0.00000e+00',\n",
       " 'controls_faf99_eas=0.00000e+00',\n",
       " 'controls_faf95_sas=0.00000e+00',\n",
       " 'controls_faf99_sas=0.00000e+00',\n",
       " 'faf95_nfe=0.00000e+00',\n",
       " 'faf99_nfe=0.00000e+00',\n",
       " 'non_topmed_faf95_eas=0.00000e+00',\n",
       " 'non_topmed_faf99_eas=0.00000e+00',\n",
       " 'controls_faf95_amr=0.00000e+00',\n",
       " 'controls_faf99_amr=0.00000e+00',\n",
       " 'non_neuro_faf95_eas=0.00000e+00',\n",
       " 'non_neuro_faf99_eas=0.00000e+00',\n",
       " 'non_cancer_faf95_nfe=0.00000e+00',\n",
       " 'non_cancer_faf99_nfe=0.00000e+00',\n",
       " 'non_cancer_faf95=8.40400e-05',\n",
       " 'non_cancer_faf99=8.40500e-05',\n",
       " 'non_cancer_faf95_sas=0.00000e+00',\n",
       " 'non_cancer_faf99_sas=0.00000e+00',\n",
       " 'non_topmed_faf95_afr=0.00000e+00',\n",
       " 'non_topmed_faf99_afr=0.00000e+00',\n",
       " 'controls_faf95=0.00000e+00',\n",
       " 'controls_faf99=0.00000e+00',\n",
       " 'popmax=amr',\n",
       " 'AC_popmax=2',\n",
       " 'AN_popmax=716',\n",
       " 'AF_popmax=2.79330e-03',\n",
       " 'nhomalt_popmax=0',\n",
       " 'age_hist_het_bin_freq=0|0|0|0|0|1|1|0|0|0',\n",
       " 'age_hist_het_n_smaller=0',\n",
       " 'age_hist_het_n_larger=0',\n",
       " 'age_hist_hom_bin_freq=0|0|0|0|0|0|0|0|0|0',\n",
       " 'age_hist_hom_n_smaller=0',\n",
       " 'age_hist_hom_n_larger=0',\n",
       " 'non_topmed_popmax=amr',\n",
       " 'non_topmed_AC_popmax=2',\n",
       " 'non_topmed_AN_popmax=712',\n",
       " 'non_topmed_AF_popmax=2.80899e-03',\n",
       " 'non_topmed_nhomalt_popmax=0',\n",
       " 'non_neuro_popmax=amr',\n",
       " 'non_neuro_AC_popmax=2',\n",
       " 'non_neuro_AN_popmax=708',\n",
       " 'non_neuro_AF_popmax=2.82486e-03',\n",
       " 'non_neuro_nhomalt_popmax=0',\n",
       " 'non_cancer_popmax=amr',\n",
       " 'non_cancer_AC_popmax=2',\n",
       " 'non_cancer_AN_popmax=716',\n",
       " 'non_cancer_AF_popmax=2.79330e-03',\n",
       " 'non_cancer_nhomalt_popmax=0',\n",
       " 'vep=C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000423562|unprocessed_pseudogene|||||||||||1|1790|-1||SNV|1|HGNC|38034||||||||||||||||||||||||||||||||||||||||||,C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000438504|unprocessed_pseudogene|||||||||||1|1790|-1||SNV|1|HGNC|38034|YES|||||||||||||||||||||||||||||||||||||||||,C|intron_variant&non_coding_transcript_variant|MODIFIER|DDX11L1|ENSG00000223972|Transcript|ENST00000450305|transcribed_unprocessed_pseudogene||2/5|ENST00000450305.2:n.98-40T>C||||||||1||1||SNV|1|HGNC|37102||||||||||||||||||||||||||||||||||||||||||,C|intron_variant&non_coding_transcript_variant|MODIFIER|DDX11L1|ENSG00000223972|Transcript|ENST00000456328|processed_transcript||1/2|ENST00000456328.2:n.360-40T>C||||||||1||1||SNV|1|HGNC|37102|YES|||||||||||||||||||||||||||||||||||||||||,C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000488147|unprocessed_pseudogene|||||||||||1|1831|-1||SNV|1|HGNC|38034||||||||||||||||||||||||||||||||||||||||||,C|intron_variant&non_coding_transcript_variant|MODIFIER|DDX11L1|ENSG00000223972|Transcript|ENST00000515242|transcribed_unprocessed_pseudogene||1/2|ENST00000515242.2:n.357-40T>C||||||||1||1||SNV|1|HGNC|37102||||||||||||||||||||||||||||||||||||||||||,C|intron_variant&non_coding_transcript_variant|MODIFIER|DDX11L1|ENSG00000223972|Transcript|ENST00000518655|transcribed_unprocessed_pseudogene||1/3|ENST00000518655.2:n.355-22T>C||||||||1||1||SNV|1|HGNC|37102||||||||||||||||||||||||||||||||||||||||||,C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000538476|unprocessed_pseudogene|||||||||||1|1838|-1||SNV|1|HGNC|38034||||||||||||||||||||||||||||||||||||||||||,C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000541675|unprocessed_pseudogene|||||||||||1|1790|-1||SNV|1|HGNC|38034||||||||||||||||||||||||||||||||||||||||||,C|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00001576075|CTCF_binding_site|||||||||||1||||SNV|1||||||||||||||||||||||||||||||||||||||||||||']"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test_str = vcf[\"INFO\"].loc[0].split(\";\")\n",
    "test_str"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[['AC', '2'],\n",
       " ['AN', '4196'],\n",
       " ['AF', '4.76644e-04'],\n",
       " ['rf_tp_probability', '3.22028e-01'],\n",
       " ['FS', '0.00000e+00'],\n",
       " ['InbreedingCoeff', '-7.56000e-02'],\n",
       " ['MQ', '2.20000e+01'],\n",
       " ['MQRankSum', '-7.20000e-01'],\n",
       " ['QD', '4.03000e+00'],\n",
       " ['ReadPosRankSum', '4.06000e-01'],\n",
       " ['SOR', '5.91000e-01'],\n",
       " ['BaseQRankSum', '1.56000e-01'],\n",
       " ['ClippingRankSum', '-3.87000e-01'],\n",
       " ['DP', '159660'],\n",
       " ['VQSLOD', '3.25000e+00'],\n",
       " ['VQSR_culprit', 'QD'],\n",
       " None,\n",
       " None,\n",
       " ['rf_label', 'FP'],\n",
       " None,\n",
       " ['variant_type', 'snv'],\n",
       " ['allele_type', 'snv'],\n",
       " ['n_alt_alleles', '1'],\n",
       " ['pab_max', '1.00000e+00'],\n",
       " ['gq_hist_alt_bin_freq', '0|0|0|2|0|0|1|1|1|0|0|1|0|0|0|0|0|0|0|2'],\n",
       " ['gq_hist_all_bin_freq',\n",
       "  '5328|7127|2608|3532|2074|679|849|436|132|204|108|31|43|1|14|0|7|1|1|2'],\n",
       " ['dp_hist_alt_bin_freq', '2|4|1|1|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0'],\n",
       " ['dp_hist_alt_n_larger', '0'],\n",
       " ['dp_hist_all_bin_freq',\n",
       "  '116953|6572|1691|421|85|22|4|0|0|0|0|0|0|0|0|0|0|0|0|0'],\n",
       " ['dp_hist_all_n_larger', '0'],\n",
       " ['ab_hist_alt_bin_freq', '0|0|0|0|0|1|1|0|0|1|1|0|2|1|0|1|0|0|0|0'],\n",
       " ['AC_nfe_seu', '0'],\n",
       " ['AN_nfe_seu', '0'],\n",
       " ['nhomalt_nfe_seu', '0'],\n",
       " ['controls_AC_afr_male', '0'],\n",
       " ['controls_AN_afr_male', '120'],\n",
       " ['controls_AF_afr_male', '0.00000e+00'],\n",
       " ['controls_nhomalt_afr_male', '0'],\n",
       " ['non_neuro_AC_eas_kor', '0'],\n",
       " ['non_neuro_AN_eas_kor', '318'],\n",
       " ['non_neuro_AF_eas_kor', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_eas_kor', '0'],\n",
       " ['non_topmed_AC_amr', '2'],\n",
       " ['non_topmed_AN_amr', '712'],\n",
       " ['non_topmed_AF_amr', '2.80899e-03'],\n",
       " ['non_topmed_nhomalt_amr', '0'],\n",
       " ['non_cancer_AC_asj_female', '0'],\n",
       " ['non_cancer_AN_asj_female', '24'],\n",
       " ['non_cancer_AF_asj_female', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_asj_female', '0'],\n",
       " ['AC_raw', '8'],\n",
       " ['AN_raw', '46354'],\n",
       " ['AF_raw', '1.72585e-04'],\n",
       " ['nhomalt_raw', '0'],\n",
       " ['AC_fin_female', '0'],\n",
       " ['AN_fin_female', '14'],\n",
       " ['AF_fin_female', '0.00000e+00'],\n",
       " ['nhomalt_fin_female', '0'],\n",
       " ['non_cancer_AC_oth_female', '0'],\n",
       " ['non_cancer_AN_oth_female', '64'],\n",
       " ['non_cancer_AF_oth_female', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_oth_female', '0'],\n",
       " ['AC_nfe_bgr', '0'],\n",
       " ['AN_nfe_bgr', '0'],\n",
       " ['nhomalt_nfe_bgr', '0'],\n",
       " ['non_neuro_AC_asj_female', '0'],\n",
       " ['non_neuro_AN_asj_female', '4'],\n",
       " ['non_neuro_AF_asj_female', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_asj_female', '0'],\n",
       " ['AC_sas_male', '0'],\n",
       " ['AN_sas_male', '894'],\n",
       " ['AF_sas_male', '0.00000e+00'],\n",
       " ['nhomalt_sas_male', '0'],\n",
       " ['non_neuro_AC_afr_male', '0'],\n",
       " ['non_neuro_AN_afr_male', '260'],\n",
       " ['non_neuro_AF_afr_male', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_afr_male', '0'],\n",
       " ['AC_afr_male', '0'],\n",
       " ['AN_afr_male', '262'],\n",
       " ['AF_afr_male', '0.00000e+00'],\n",
       " ['nhomalt_afr_male', '0'],\n",
       " ['AC_afr', '0'],\n",
       " ['AN_afr', '624'],\n",
       " ['AF_afr', '0.00000e+00'],\n",
       " ['nhomalt_afr', '0'],\n",
       " ['controls_AC_nfe_swe', '0'],\n",
       " ['controls_AN_nfe_swe', '0'],\n",
       " ['controls_nhomalt_nfe_swe', '0'],\n",
       " ['non_neuro_AC_afr_female', '0'],\n",
       " ['non_neuro_AN_afr_female', '360'],\n",
       " ['non_neuro_AF_afr_female', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_afr_female', '0'],\n",
       " ['non_topmed_AC_amr_female', '2'],\n",
       " ['non_topmed_AN_amr_female', '348'],\n",
       " ['non_topmed_AF_amr_female', '5.74713e-03'],\n",
       " ['non_topmed_nhomalt_amr_female', '0'],\n",
       " ['non_cancer_AC_female', '2'],\n",
       " ['non_cancer_AN_female', '1850'],\n",
       " ['non_cancer_AF_female', '1.08108e-03'],\n",
       " ['non_cancer_nhomalt_female', '0'],\n",
       " ['non_cancer_AC_nfe_onf', '0'],\n",
       " ['non_cancer_AN_nfe_onf', '166'],\n",
       " ['non_cancer_AF_nfe_onf', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_nfe_onf', '0'],\n",
       " ['non_cancer_AC_male', '0'],\n",
       " ['non_cancer_AN_male', '2346'],\n",
       " ['non_cancer_AF_male', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_male', '0'],\n",
       " ['non_topmed_AC_oth_female', '0'],\n",
       " ['non_topmed_AN_oth_female', '62'],\n",
       " ['non_topmed_AF_oth_female', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_oth_female', '0'],\n",
       " ['AC_eas_female', '0'],\n",
       " ['AN_eas_female', '278'],\n",
       " ['AF_eas_female', '0.00000e+00'],\n",
       " ['nhomalt_eas_female', '0'],\n",
       " ['non_cancer_AC_sas_female', '0'],\n",
       " ['non_cancer_AN_sas_female', '344'],\n",
       " ['non_cancer_AF_sas_female', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_sas_female', '0'],\n",
       " ['AC_afr_female', '0'],\n",
       " ['AN_afr_female', '362'],\n",
       " ['AF_afr_female', '0.00000e+00'],\n",
       " ['nhomalt_afr_female', '0'],\n",
       " ['AC_sas', '0'],\n",
       " ['AN_sas', '1238'],\n",
       " ['AF_sas', '0.00000e+00'],\n",
       " ['nhomalt_sas', '0'],\n",
       " ['non_neuro_AC_female', '2'],\n",
       " ['non_neuro_AN_female', '1768'],\n",
       " ['non_neuro_AF_female', '1.13122e-03'],\n",
       " ['non_neuro_nhomalt_female', '0'],\n",
       " ['controls_AC_afr', '0'],\n",
       " ['controls_AN_afr', '304'],\n",
       " ['controls_AF_afr', '0.00000e+00'],\n",
       " ['controls_nhomalt_afr', '0'],\n",
       " ['non_neuro_AC_eas_jpn', '0'],\n",
       " ['non_neuro_AN_eas_jpn', '0'],\n",
       " ['non_neuro_nhomalt_eas_jpn', '0'],\n",
       " ['AC_nfe_onf', '0'],\n",
       " ['AN_nfe_onf', '166'],\n",
       " ['AF_nfe_onf', '0.00000e+00'],\n",
       " ['nhomalt_nfe_onf', '0'],\n",
       " ['non_cancer_AC_amr_male', '0'],\n",
       " ['non_cancer_AN_amr_male', '368'],\n",
       " ['non_cancer_AF_amr_male', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_amr_male', '0'],\n",
       " ['controls_AC_fin_male', '0'],\n",
       " ['controls_AN_fin_male', '0'],\n",
       " ['controls_nhomalt_fin_male', '0'],\n",
       " ['non_neuro_AC_nfe_nwe', '0'],\n",
       " ['non_neuro_AN_nfe_nwe', '540'],\n",
       " ['non_neuro_AF_nfe_nwe', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_nfe_nwe', '0'],\n",
       " ['AC_fin_male', '0'],\n",
       " ['AN_fin_male', '22'],\n",
       " ['AF_fin_male', '0.00000e+00'],\n",
       " ['nhomalt_fin_male', '0'],\n",
       " ['AC_nfe_female', '0'],\n",
       " ['AN_nfe_female', '416'],\n",
       " ['AF_nfe_female', '0.00000e+00'],\n",
       " ['nhomalt_nfe_female', '0'],\n",
       " ['AC_amr', '2'],\n",
       " ['AN_amr', '716'],\n",
       " ['AF_amr', '2.79330e-03'],\n",
       " ['nhomalt_amr', '0'],\n",
       " ['non_topmed_AC_nfe_male', '0'],\n",
       " ['non_topmed_AN_nfe_male', '372'],\n",
       " ['non_topmed_AF_nfe_male', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_nfe_male', '0'],\n",
       " ['non_neuro_AC_sas', '0'],\n",
       " ['non_neuro_AN_sas', '1238'],\n",
       " ['non_neuro_AF_sas', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_sas', '0'],\n",
       " ['non_cancer_AC_fin_male', '0'],\n",
       " ['non_cancer_AN_fin_male', '22'],\n",
       " ['non_cancer_AF_fin_male', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_fin_male', '0'],\n",
       " ['non_cancer_AC_nfe_seu', '0'],\n",
       " ['non_cancer_AN_nfe_seu', '0'],\n",
       " ['non_cancer_nhomalt_nfe_seu', '0'],\n",
       " ['AC_eas', '0'],\n",
       " ['AN_eas', '556'],\n",
       " ['AF_eas', '0.00000e+00'],\n",
       " ['nhomalt_eas', '0'],\n",
       " ['nhomalt', '0'],\n",
       " ['non_neuro_AC_nfe_female', '0'],\n",
       " ['non_neuro_AN_nfe_female', '364'],\n",
       " ['non_neuro_AF_nfe_female', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_nfe_female', '0'],\n",
       " ['non_neuro_AC_afr', '0'],\n",
       " ['non_neuro_AN_afr', '620'],\n",
       " ['non_neuro_AF_afr', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_afr', '0'],\n",
       " ['controls_AC_raw', '4'],\n",
       " ['controls_AN_raw', '19152'],\n",
       " ['controls_AF_raw', '2.08855e-04'],\n",
       " ['controls_nhomalt_raw', '0'],\n",
       " ['non_cancer_AC_eas', '0'],\n",
       " ['non_cancer_AN_eas', '556'],\n",
       " ['non_cancer_AF_eas', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_eas', '0'],\n",
       " ['non_cancer_AC_amr_female', '2'],\n",
       " ['non_cancer_AN_amr_female', '348'],\n",
       " ['non_cancer_AF_amr_female', '5.74713e-03'],\n",
       " ['non_cancer_nhomalt_amr_female', '0'],\n",
       " ['non_neuro_AC_nfe_swe', '0'],\n",
       " ['non_neuro_AN_nfe_swe', '0'],\n",
       " ['non_neuro_nhomalt_nfe_swe', '0'],\n",
       " ['controls_AC_male', '0'],\n",
       " ['controls_AN_male', '1074'],\n",
       " ['controls_AF_male', '0.00000e+00'],\n",
       " ['controls_nhomalt_male', '0'],\n",
       " ['non_topmed_AC_male', '0'],\n",
       " ['non_topmed_AN_male', '2276'],\n",
       " ['non_topmed_AF_male', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_male', '0'],\n",
       " ['controls_AC_eas_jpn', '0'],\n",
       " ['controls_AN_eas_jpn', '0'],\n",
       " ['controls_nhomalt_eas_jpn', '0'],\n",
       " ['controls_AC_nfe_female', '0'],\n",
       " ['controls_AN_nfe_female', '198'],\n",
       " ['controls_AF_nfe_female', '0.00000e+00'],\n",
       " ['controls_nhomalt_nfe_female', '0'],\n",
       " ['non_neuro_AC_amr', '2'],\n",
       " ['non_neuro_AN_amr', '708'],\n",
       " ['non_neuro_AF_amr', '2.82486e-03'],\n",
       " ['non_neuro_nhomalt_amr', '0'],\n",
       " ['non_neuro_AC_eas_female', '0'],\n",
       " ['non_neuro_AN_eas_female', '278'],\n",
       " ['non_neuro_AF_eas_female', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_eas_female', '0'],\n",
       " ['AC_asj_male', '0'],\n",
       " ['AN_asj_male', '18'],\n",
       " ['AF_asj_male', '0.00000e+00'],\n",
       " ['nhomalt_asj_male', '0'],\n",
       " ['controls_AC_nfe_male', '0'],\n",
       " ['controls_AN_nfe_male', '182'],\n",
       " ['controls_AF_nfe_male', '0.00000e+00'],\n",
       " ['controls_nhomalt_nfe_male', '0'],\n",
       " ['non_neuro_AC_fin', '0'],\n",
       " ['non_neuro_AN_fin', '36'],\n",
       " ['non_neuro_AF_fin', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_fin', '0'],\n",
       " ['non_topmed_AC_sas', '0'],\n",
       " ['non_topmed_AN_sas', '1238'],\n",
       " ['non_topmed_AF_sas', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_sas', '0'],\n",
       " ['non_cancer_AC_nfe_female', '0'],\n",
       " ['non_cancer_AN_nfe_female', '416'],\n",
       " ['non_cancer_AF_nfe_female', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_nfe_female', '0'],\n",
       " ['AC_oth_female', '0'],\n",
       " ['AN_oth_female', '64'],\n",
       " ['AF_oth_female', '0.00000e+00'],\n",
       " ['nhomalt_oth_female', '0'],\n",
       " ['non_cancer_AC_asj', '0'],\n",
       " ['non_cancer_AN_asj', '42'],\n",
       " ['non_cancer_AF_asj', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_asj', '0'],\n",
       " ['AC_nfe_swe', '0'],\n",
       " ['AN_nfe_swe', '0'],\n",
       " ['nhomalt_nfe_swe', '0'],\n",
       " ['controls_AC_nfe', '0'],\n",
       " ['controls_AN_nfe', '380'],\n",
       " ['controls_AF_nfe', '0.00000e+00'],\n",
       " ['controls_nhomalt_nfe', '0'],\n",
       " ['controls_AC_oth_female', '0'],\n",
       " ['controls_AN_oth_female', '34'],\n",
       " ['controls_AF_oth_female', '0.00000e+00'],\n",
       " ['controls_nhomalt_oth_female', '0'],\n",
       " ['controls_AC_asj', '0'],\n",
       " ['controls_AN_asj', '14'],\n",
       " ['controls_AF_asj', '0.00000e+00'],\n",
       " ['controls_nhomalt_asj', '0'],\n",
       " ['non_neuro_AC_amr_male', '0'],\n",
       " ['non_neuro_AN_amr_male', '364'],\n",
       " ['non_neuro_AF_amr_male', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_amr_male', '0'],\n",
       " ['controls_AC_nfe_nwe', '0'],\n",
       " ['controls_AN_nfe_nwe', '294'],\n",
       " ['controls_AF_nfe_nwe', '0.00000e+00'],\n",
       " ['controls_nhomalt_nfe_nwe', '0'],\n",
       " ['AC_nfe_nwe', '0'],\n",
       " ['AN_nfe_nwe', '672'],\n",
       " ['AF_nfe_nwe', '0.00000e+00'],\n",
       " ['nhomalt_nfe_nwe', '0'],\n",
       " ['controls_AC_nfe_seu', '0'],\n",
       " ['controls_AN_nfe_seu', '0'],\n",
       " ['controls_nhomalt_nfe_seu', '0'],\n",
       " ['controls_AC_sas_female', '0'],\n",
       " ['controls_AN_sas_female', '188'],\n",
       " ['controls_AF_sas_female', '0.00000e+00'],\n",
       " ['controls_nhomalt_sas_female', '0'],\n",
       " ['non_neuro_AC_amr_female', '2'],\n",
       " ['non_neuro_AN_amr_female', '344'],\n",
       " ['non_neuro_AF_amr_female', '5.81395e-03'],\n",
       " ['non_neuro_nhomalt_amr_female', '0'],\n",
       " ['non_cancer_AC_eas_jpn', '0'],\n",
       " ['non_cancer_AN_eas_jpn', '0'],\n",
       " ['non_cancer_nhomalt_eas_jpn', '0'],\n",
       " ['non_neuro_AC_nfe_onf', '0'],\n",
       " ['non_neuro_AN_nfe_onf', '146'],\n",
       " ['non_neuro_AF_nfe_onf', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_nfe_onf', '0'],\n",
       " ['non_topmed_AC_eas_male', '0'],\n",
       " ['non_topmed_AN_eas_male', '278'],\n",
       " ['non_topmed_AF_eas_male', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_eas_male', '0'],\n",
       " ['AC_eas_jpn', '0'],\n",
       " ['AN_eas_jpn', '0'],\n",
       " ['nhomalt_eas_jpn', '0'],\n",
       " ['non_cancer_AC_afr_male', '0'],\n",
       " ['non_cancer_AN_afr_male', '262'],\n",
       " ['non_cancer_AF_afr_male', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_afr_male', '0'],\n",
       " ['non_cancer_AC_afr', '0'],\n",
       " ['non_cancer_AN_afr', '624'],\n",
       " ['non_cancer_AF_afr', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_afr', '0'],\n",
       " ['controls_AC_amr_female', '0'],\n",
       " ['controls_AN_amr_female', '154'],\n",
       " ['controls_AF_amr_female', '0.00000e+00'],\n",
       " ['controls_nhomalt_amr_female', '0'],\n",
       " ['non_neuro_AC_fin_male', '0'],\n",
       " ['non_neuro_AN_fin_male', '22'],\n",
       " ['non_neuro_AF_fin_male', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_fin_male', '0'],\n",
       " ['AC_female', '2'],\n",
       " ['AN_female', '1850'],\n",
       " ['AF_female', '1.08108e-03'],\n",
       " ['nhomalt_female', '0'],\n",
       " ['non_neuro_AC_nfe_bgr', '0'],\n",
       " ['non_neuro_AN_nfe_bgr', '0'],\n",
       " ['non_neuro_nhomalt_nfe_bgr', '0'],\n",
       " ['non_neuro_AC_oth_male', '0'],\n",
       " ['non_neuro_AN_oth_male', '68'],\n",
       " ['non_neuro_AF_oth_male', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_oth_male', '0'],\n",
       " ['non_topmed_AC_nfe_est', '0'],\n",
       " ['non_topmed_AN_nfe_est', '4'],\n",
       " ['non_topmed_AF_nfe_est', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_nfe_est', '0'],\n",
       " ['non_topmed_AC_nfe_nwe', '0'],\n",
       " ['non_topmed_AN_nfe_nwe', '592'],\n",
       " ['non_topmed_AF_nfe_nwe', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_nfe_nwe', '0'],\n",
       " ['non_topmed_AC_amr_male', '0'],\n",
       " ['non_topmed_AN_amr_male', '364'],\n",
       " ['non_topmed_AF_amr_male', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_amr_male', '0'],\n",
       " ['non_cancer_AC_amr', '2'],\n",
       " ['non_cancer_AN_amr', '716'],\n",
       " ['non_cancer_AF_amr', '2.79330e-03'],\n",
       " ['non_cancer_nhomalt_amr', '0'],\n",
       " ['non_topmed_AC_nfe_swe', '0'],\n",
       " ['non_topmed_AN_nfe_swe', '0'],\n",
       " ['non_topmed_nhomalt_nfe_swe', '0'],\n",
       " ['non_topmed_AC_nfe_onf', '0'],\n",
       " ['non_topmed_AN_nfe_onf', '134'],\n",
       " ['non_topmed_AF_nfe_onf', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_nfe_onf', '0'],\n",
       " ['controls_AC_eas_kor', '0'],\n",
       " ['controls_AN_eas_kor', '158'],\n",
       " ['controls_AF_eas_kor', '0.00000e+00'],\n",
       " ['controls_nhomalt_eas_kor', '0'],\n",
       " ['non_topmed_AC_eas_oea', '0'],\n",
       " ['non_topmed_AN_eas_oea', '238'],\n",
       " ['non_topmed_AF_eas_oea', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_eas_oea', '0'],\n",
       " ['controls_AC_eas_male', '0'],\n",
       " ['controls_AN_eas_male', '130'],\n",
       " ['controls_AF_eas_male', '0.00000e+00'],\n",
       " ['controls_nhomalt_eas_male', '0'],\n",
       " ['controls_AC_oth_male', '0'],\n",
       " ['controls_AN_oth_male', '32'],\n",
       " ['controls_AF_oth_male', '0.00000e+00'],\n",
       " ['controls_nhomalt_oth_male', '0'],\n",
       " ['non_topmed_AC', '2'],\n",
       " ['non_topmed_AN', '4050'],\n",
       " ['non_topmed_AF', '4.93827e-04'],\n",
       " ['non_topmed_nhomalt', '0'],\n",
       " ['controls_AC_fin', '0'],\n",
       " ['controls_AN_fin', '0'],\n",
       " ['controls_nhomalt_fin', '0'],\n",
       " ['AC_eas_kor', '0'],\n",
       " ['AN_eas_kor', '318'],\n",
       " ['AF_eas_kor', '0.00000e+00'],\n",
       " ['nhomalt_eas_kor', '0'],\n",
       " ['non_neuro_AC_nfe', '0'],\n",
       " ['non_neuro_AN_nfe', '692'],\n",
       " ['non_neuro_AF_nfe', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_nfe', '0'],\n",
       " ['non_neuro_AC_fin_female', '0'],\n",
       " ['non_neuro_AN_fin_female', '14'],\n",
       " ['non_neuro_AF_fin_female', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_fin_female', '0'],\n",
       " ['non_cancer_AC_nfe_male', '0'],\n",
       " ['non_cancer_AN_nfe_male', '428'],\n",
       " ['non_cancer_AF_nfe_male', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_nfe_male', '0'],\n",
       " ['controls_AC_eas_oea', '0'],\n",
       " ['controls_AN_eas_oea', '88'],\n",
       " ['controls_AF_eas_oea', '0.00000e+00'],\n",
       " ['controls_nhomalt_eas_oea', '0'],\n",
       " ['non_topmed_AC_nfe_seu', '0'],\n",
       " ['non_topmed_AN_nfe_seu', '0'],\n",
       " ['non_topmed_nhomalt_nfe_seu', '0'],\n",
       " ['controls_AC_eas_female', '0'],\n",
       " ['controls_AN_eas_female', '116'],\n",
       " ['controls_AF_eas_female', '0.00000e+00'],\n",
       " ['controls_nhomalt_eas_female', '0'],\n",
       " ['non_topmed_AC_asj', '0'],\n",
       " ['non_topmed_AN_asj', '40'],\n",
       " ['non_topmed_AF_asj', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_asj', '0'],\n",
       " ['controls_AC_nfe_onf', '0'],\n",
       " ['controls_AN_nfe_onf', '84'],\n",
       " ['controls_AF_nfe_onf', '0.00000e+00'],\n",
       " ['controls_nhomalt_nfe_onf', '0'],\n",
       " ['non_neuro_AC', '2'],\n",
       " ['non_neuro_AN', '3998'],\n",
       " ['non_neuro_AF', '5.00250e-04'],\n",
       " ['non_neuro_nhomalt', '0'],\n",
       " ['AC_eas_oea', '0'],\n",
       " ['AN_eas_oea', '238'],\n",
       " ['AF_eas_oea', '0.00000e+00'],\n",
       " ['nhomalt_eas_oea', '0'],\n",
       " ['non_topmed_AC_nfe', '0'],\n",
       " ['non_topmed_AN_nfe', '730'],\n",
       " ['non_topmed_AF_nfe', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_nfe', '0'],\n",
       " ['non_cancer_AC_oth', '0'],\n",
       " ['non_cancer_AN_oth', '140'],\n",
       " ['non_cancer_AF_oth', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_oth', '0'],\n",
       " ['non_topmed_AC_raw', '8'],\n",
       " ['non_topmed_AN_raw', '45148'],\n",
       " ['non_topmed_AF_raw', '1.77195e-04'],\n",
       " ['non_topmed_nhomalt_raw', '0'],\n",
       " ['non_neuro_AC_nfe_est', '0'],\n",
       " ['non_neuro_AN_nfe_est', '6'],\n",
       " ['non_neuro_AF_nfe_est', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_nfe_est', '0'],\n",
       " ['non_topmed_AC_oth_male', '0'],\n",
       " ['non_topmed_AN_oth_male', '72'],\n",
       " ['non_topmed_AF_oth_male', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_oth_male', '0'],\n",
       " ['non_cancer_AC_oth_male', '0'],\n",
       " ['non_cancer_AN_oth_male', '76'],\n",
       " ['non_cancer_AF_oth_male', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_oth_male', '0'],\n",
       " ['AC_nfe_est', '0'],\n",
       " ['AN_nfe_est', '6'],\n",
       " ['AF_nfe_est', '0.00000e+00'],\n",
       " ['nhomalt_nfe_est', '0'],\n",
       " ['non_cancer_AC_afr_female', '0'],\n",
       " ['non_cancer_AN_afr_female', '362'],\n",
       " ['non_cancer_AF_afr_female', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_afr_female', '0'],\n",
       " ['non_topmed_AC_afr_male', '0'],\n",
       " ['non_topmed_AN_afr_male', '256'],\n",
       " ['non_topmed_AF_afr_male', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_afr_male', '0'],\n",
       " ['AC_eas_male', '0'],\n",
       " ['AN_eas_male', '278'],\n",
       " ['AF_eas_male', '0.00000e+00'],\n",
       " ['nhomalt_eas_male', '0'],\n",
       " ['controls_AC_eas', '0'],\n",
       " ['controls_AN_eas', '246'],\n",
       " ['controls_AF_eas', '0.00000e+00'],\n",
       " ['controls_nhomalt_eas', '0'],\n",
       " ['non_neuro_AC_eas_male', '0'],\n",
       " ['non_neuro_AN_eas_male', '278'],\n",
       " ['non_neuro_AF_eas_male', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_eas_male', '0'],\n",
       " ['non_cancer_AC_nfe_nwe', '0'],\n",
       " ['non_cancer_AN_nfe_nwe', '672'],\n",
       " ['non_cancer_AF_nfe_nwe', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_nfe_nwe', '0'],\n",
       " ['controls_AC_sas', '0'],\n",
       " ['controls_AN_sas', '614'],\n",
       " ['controls_AF_sas', '0.00000e+00'],\n",
       " ['controls_nhomalt_sas', '0'],\n",
       " ['non_neuro_AC_sas_male', '0'],\n",
       " ['non_neuro_AN_sas_male', '894'],\n",
       " ['non_neuro_AF_sas_male', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_sas_male', '0'],\n",
       " ['non_neuro_AC_asj_male', '0'],\n",
       " ['non_neuro_AN_asj_male', '16'],\n",
       " ['non_neuro_AF_asj_male', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_asj_male', '0'],\n",
       " ['non_cancer_AC_nfe_bgr', '0'],\n",
       " ['non_cancer_AN_nfe_bgr', '0'],\n",
       " ['non_cancer_nhomalt_nfe_bgr', '0'],\n",
       " ['controls_AC_oth', '0'],\n",
       " ['controls_AN_oth', '66'],\n",
       " ['controls_AF_oth', '0.00000e+00'],\n",
       " ['controls_nhomalt_oth', '0'],\n",
       " ['non_cancer_AC_eas_female', '0'],\n",
       " ['non_cancer_AN_eas_female', '278'],\n",
       " ['non_cancer_AF_eas_female', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_eas_female', '0'],\n",
       " ['AC_nfe', '0'],\n",
       " ['AN_nfe', '844'],\n",
       " ['AF_nfe', '0.00000e+00'],\n",
       " ['nhomalt_nfe', '0'],\n",
       " ['non_topmed_AC_female', '2'],\n",
       " ['non_topmed_AN_female', '1774'],\n",
       " ['non_topmed_AF_female', '1.12740e-03'],\n",
       " ['non_topmed_nhomalt_female', '0'],\n",
       " ['non_neuro_AC_asj', '0'],\n",
       " ['non_neuro_AN_asj', '20'],\n",
       " ['non_neuro_AF_asj', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_asj', '0'],\n",
       " ['non_topmed_AC_eas_female', '0'],\n",
       " ['non_topmed_AN_eas_female', '278'],\n",
       " ['non_topmed_AF_eas_female', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_eas_female', '0'],\n",
       " ['non_neuro_AC_raw', '7'],\n",
       " ['non_neuro_AN_raw', '39484'],\n",
       " ['non_neuro_AF_raw', '1.77287e-04'],\n",
       " ['non_neuro_nhomalt_raw', '0'],\n",
       " ['non_topmed_AC_eas', '0'],\n",
       " ['non_topmed_AN_eas', '556'],\n",
       " ['non_topmed_AF_eas', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_eas', '0'],\n",
       " ['non_topmed_AC_fin_male', '0'],\n",
       " ['non_topmed_AN_fin_male', '22'],\n",
       " ['non_topmed_AF_fin_male', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_fin_male', '0'],\n",
       " ['non_cancer_AC_asj_male', '0'],\n",
       " ['non_cancer_AN_asj_male', '18'],\n",
       " ['non_cancer_AF_asj_male', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_asj_male', '0'],\n",
       " ['AC_fin', '0'],\n",
       " ['AN_fin', '36'],\n",
       " ['AF_fin', '0.00000e+00'],\n",
       " ['nhomalt_fin', '0'],\n",
       " ['AC_nfe_male', '0'],\n",
       " ['AN_nfe_male', '428'],\n",
       " ['AF_nfe_male', '0.00000e+00'],\n",
       " ['nhomalt_nfe_male', '0'],\n",
       " ['non_topmed_AC_eas_kor', '0'],\n",
       " ['non_topmed_AN_eas_kor', '318'],\n",
       " ['non_topmed_AF_eas_kor', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_eas_kor', '0'],\n",
       " ['controls_AC_amr_male', '0'],\n",
       " ['controls_AN_amr_male', '172'],\n",
       " ['controls_AF_amr_male', '0.00000e+00'],\n",
       " ['controls_nhomalt_amr_male', '0'],\n",
       " ['non_neuro_AC_eas_oea', '0'],\n",
       " ['non_neuro_AN_eas_oea', '238'],\n",
       " ['non_neuro_AF_eas_oea', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_eas_oea', '0'],\n",
       " ['AC_sas_female', '0'],\n",
       " ['AN_sas_female', '344'],\n",
       " ['AF_sas_female', '0.00000e+00'],\n",
       " ['nhomalt_sas_female', '0'],\n",
       " ['controls_AC_afr_female', '0'],\n",
       " ['controls_AN_afr_female', '184'],\n",
       " ['controls_AF_afr_female', '0.00000e+00'],\n",
       " ['controls_nhomalt_afr_female', '0'],\n",
       " ['controls_AC_amr', '0'],\n",
       " ['controls_AN_amr', '326'],\n",
       " ['controls_AF_amr', '0.00000e+00'],\n",
       " ['controls_nhomalt_amr', '0'],\n",
       " ['non_topmed_AC_eas_jpn', '0'],\n",
       " ['non_topmed_AN_eas_jpn', '0'],\n",
       " ['non_topmed_nhomalt_eas_jpn', '0'],\n",
       " ['AC_asj_female', '0'],\n",
       " ['AN_asj_female', '24'],\n",
       " ['AF_asj_female', '0.00000e+00'],\n",
       " ['nhomalt_asj_female', '0'],\n",
       " ['non_topmed_AC_nfe_bgr', '0'],\n",
       " ['non_topmed_AN_nfe_bgr', '0'],\n",
       " ['non_topmed_nhomalt_nfe_bgr', '0'],\n",
       " ['non_cancer_AC_nfe_est', '0'],\n",
       " ['non_cancer_AN_nfe_est', '6'],\n",
       " ['non_cancer_AF_nfe_est', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_nfe_est', '0'],\n",
       " ['non_neuro_AC_eas', '0'],\n",
       " ['non_neuro_AN_eas', '556'],\n",
       " ['non_neuro_AF_eas', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_eas', '0'],\n",
       " ['non_cancer_AC_nfe', '0'],\n",
       " ['non_cancer_AN_nfe', '844'],\n",
       " ['non_cancer_AF_nfe', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_nfe', '0'],\n",
       " ['non_neuro_AC_male', '0'],\n",
       " ['non_neuro_AN_male', '2230'],\n",
       " ['non_neuro_AF_male', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_male', '0'],\n",
       " ['non_neuro_AC_sas_female', '0'],\n",
       " ['non_neuro_AN_sas_female', '344'],\n",
       " ['non_neuro_AF_sas_female', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_sas_female', '0'],\n",
       " ['AC_asj', '0'],\n",
       " ['AN_asj', '42'],\n",
       " ['AF_asj', '0.00000e+00'],\n",
       " ['nhomalt_asj', '0'],\n",
       " ['controls_AC_nfe_est', '0'],\n",
       " ['controls_AN_nfe_est', '2'],\n",
       " ['controls_AF_nfe_est', '0.00000e+00'],\n",
       " ['controls_nhomalt_nfe_est', '0'],\n",
       " ['non_topmed_AC_asj_female', '0'],\n",
       " ['non_topmed_AN_asj_female', '22'],\n",
       " ['non_topmed_AF_asj_female', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_asj_female', '0'],\n",
       " ['non_cancer_AC_nfe_swe', '0'],\n",
       " ['non_cancer_AN_nfe_swe', '0'],\n",
       " ['non_cancer_nhomalt_nfe_swe', '0'],\n",
       " ['non_cancer_AC', '2'],\n",
       " ['non_cancer_AN', '4196'],\n",
       " ['non_cancer_AF', '4.76644e-04'],\n",
       " ['non_cancer_nhomalt', '0'],\n",
       " ['non_topmed_AC_oth', '0'],\n",
       " ['non_topmed_AN_oth', '134'],\n",
       " ['non_topmed_AF_oth', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_oth', '0'],\n",
       " ['non_topmed_AC_fin_female', '0'],\n",
       " ['non_topmed_AN_fin_female', '14'],\n",
       " ['non_topmed_AF_fin_female', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_fin_female', '0'],\n",
       " ['non_cancer_AC_fin_female', '0'],\n",
       " ['non_cancer_AN_fin_female', '14'],\n",
       " ['non_cancer_AF_fin_female', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_fin_female', '0'],\n",
       " ['AC_oth', '0'],\n",
       " ['AN_oth', '140'],\n",
       " ['AF_oth', '0.00000e+00'],\n",
       " ['nhomalt_oth', '0'],\n",
       " ['non_neuro_AC_nfe_male', '0'],\n",
       " ['non_neuro_AN_nfe_male', '328'],\n",
       " ['non_neuro_AF_nfe_male', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_nfe_male', '0'],\n",
       " ['controls_AC_female', '0'],\n",
       " ['controls_AN_female', '876'],\n",
       " ['controls_AF_female', '0.00000e+00'],\n",
       " ['controls_nhomalt_female', '0'],\n",
       " ['non_cancer_AC_fin', '0'],\n",
       " ['non_cancer_AN_fin', '36'],\n",
       " ['non_cancer_AF_fin', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_fin', '0'],\n",
       " ['non_topmed_AC_fin', '0'],\n",
       " ['non_topmed_AN_fin', '36'],\n",
       " ['non_topmed_AF_fin', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_fin', '0'],\n",
       " ['non_cancer_AC_eas_oea', '0'],\n",
       " ['non_cancer_AN_eas_oea', '238'],\n",
       " ['non_cancer_AF_eas_oea', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_eas_oea', '0'],\n",
       " ['non_topmed_AC_nfe_female', '0'],\n",
       " ['non_topmed_AN_nfe_female', '358'],\n",
       " ['non_topmed_AF_nfe_female', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_nfe_female', '0'],\n",
       " ['non_cancer_AC_sas_male', '0'],\n",
       " ['non_cancer_AN_sas_male', '894'],\n",
       " ['non_cancer_AF_sas_male', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_sas_male', '0'],\n",
       " ['controls_AC_asj_male', '0'],\n",
       " ['controls_AN_asj_male', '12'],\n",
       " ['controls_AF_asj_male', '0.00000e+00'],\n",
       " ['controls_nhomalt_asj_male', '0'],\n",
       " ['non_cancer_AC_raw', '8'],\n",
       " ['non_cancer_AN_raw', '46350'],\n",
       " ['non_cancer_AF_raw', '1.72600e-04'],\n",
       " ['non_cancer_nhomalt_raw', '0'],\n",
       " ['non_cancer_AC_eas_male', '0'],\n",
       " ['non_cancer_AN_eas_male', '278'],\n",
       " ['non_cancer_AF_eas_male', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_eas_male', '0'],\n",
       " ['non_topmed_AC_asj_male', '0'],\n",
       " ['non_topmed_AN_asj_male', '18'],\n",
       " ['non_topmed_AF_asj_male', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_asj_male', '0'],\n",
       " ['non_neuro_AC_oth', '0'],\n",
       " ['non_neuro_AN_oth', '128'],\n",
       " ['non_neuro_AF_oth', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_oth', '0'],\n",
       " ['AC_male', '0'],\n",
       " ['AN_male', '2346'],\n",
       " ['AF_male', '0.00000e+00'],\n",
       " ['nhomalt_male', '0'],\n",
       " ['controls_AC_fin_female', '0'],\n",
       " ['controls_AN_fin_female', '0'],\n",
       " ['controls_nhomalt_fin_female', '0'],\n",
       " ['controls_AC_nfe_bgr', '0'],\n",
       " ['controls_AN_nfe_bgr', '0'],\n",
       " ['controls_nhomalt_nfe_bgr', '0'],\n",
       " ['controls_AC_asj_female', '0'],\n",
       " ['controls_AN_asj_female', '2'],\n",
       " ['controls_AF_asj_female', '0.00000e+00'],\n",
       " ['controls_nhomalt_asj_female', '0'],\n",
       " ['AC_amr_male', '0'],\n",
       " ['AN_amr_male', '368'],\n",
       " ['AF_amr_male', '0.00000e+00'],\n",
       " ['nhomalt_amr_male', '0'],\n",
       " ['AC_amr_female', '2'],\n",
       " ['AN_amr_female', '348'],\n",
       " ['AF_amr_female', '5.74713e-03'],\n",
       " ['nhomalt_amr_female', '0'],\n",
       " ['non_topmed_AC_sas_male', '0'],\n",
       " ['non_topmed_AN_sas_male', '894'],\n",
       " ['non_topmed_AF_sas_male', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_sas_male', '0'],\n",
       " ['AC_oth_male', '0'],\n",
       " ['AN_oth_male', '76'],\n",
       " ['AF_oth_male', '0.00000e+00'],\n",
       " ['nhomalt_oth_male', '0'],\n",
       " ['non_cancer_AC_sas', '0'],\n",
       " ['non_cancer_AN_sas', '1238'],\n",
       " ['non_cancer_AF_sas', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_sas', '0'],\n",
       " ['non_neuro_AC_nfe_seu', '0'],\n",
       " ['non_neuro_AN_nfe_seu', '0'],\n",
       " ['non_neuro_nhomalt_nfe_seu', '0'],\n",
       " ['non_cancer_AC_eas_kor', '0'],\n",
       " ['non_cancer_AN_eas_kor', '318'],\n",
       " ['non_cancer_AF_eas_kor', '0.00000e+00'],\n",
       " ['non_cancer_nhomalt_eas_kor', '0'],\n",
       " ['non_topmed_AC_afr_female', '0'],\n",
       " ['non_topmed_AN_afr_female', '348'],\n",
       " ['non_topmed_AF_afr_female', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_afr_female', '0'],\n",
       " ['controls_AC_sas_male', '0'],\n",
       " ['controls_AN_sas_male', '426'],\n",
       " ['controls_AF_sas_male', '0.00000e+00'],\n",
       " ['controls_nhomalt_sas_male', '0'],\n",
       " ['non_topmed_AC_sas_female', '0'],\n",
       " ['non_topmed_AN_sas_female', '344'],\n",
       " ['non_topmed_AF_sas_female', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_sas_female', '0'],\n",
       " ['non_topmed_AC_afr', '0'],\n",
       " ['non_topmed_AN_afr', '604'],\n",
       " ['non_topmed_AF_afr', '0.00000e+00'],\n",
       " ['non_topmed_nhomalt_afr', '0'],\n",
       " ['controls_AC', '0'],\n",
       " ['controls_AN', '1950'],\n",
       " ['controls_AF', '0.00000e+00'],\n",
       " ['controls_nhomalt', '0'],\n",
       " ['non_neuro_AC_oth_female', '0'],\n",
       " ['non_neuro_AN_oth_female', '60'],\n",
       " ['non_neuro_AF_oth_female', '0.00000e+00'],\n",
       " ['non_neuro_nhomalt_oth_female', '0'],\n",
       " ['non_topmed_faf95_amr', '4.98630e-04'],\n",
       " ['non_topmed_faf99_amr', '4.98230e-04'],\n",
       " ['faf95_afr', '0.00000e+00'],\n",
       " ['faf99_afr', '0.00000e+00'],\n",
       " ['faf95_sas', '0.00000e+00'],\n",
       " ['faf99_sas', '0.00000e+00'],\n",
       " ['controls_faf95_afr', '0.00000e+00'],\n",
       " ['controls_faf99_afr', '0.00000e+00'],\n",
       " ['faf95_amr', '4.95930e-04'],\n",
       " ['faf99_amr', '4.95370e-04'],\n",
       " ['non_neuro_faf95_sas', '0.00000e+00'],\n",
       " ['non_neuro_faf99_sas', '0.00000e+00'],\n",
       " ['faf95_eas', '0.00000e+00'],\n",
       " ['faf99_eas', '0.00000e+00'],\n",
       " ['faf95', '8.40400e-05'],\n",
       " ['faf99', '8.40500e-05'],\n",
       " ['non_neuro_faf95_afr', '0.00000e+00'],\n",
       " ['non_neuro_faf99_afr', '0.00000e+00'],\n",
       " ['non_cancer_faf95_eas', '0.00000e+00'],\n",
       " ['non_cancer_faf99_eas', '0.00000e+00'],\n",
       " ['non_neuro_faf95_amr', '5.01060e-04'],\n",
       " ['non_neuro_faf99_amr', '5.01220e-04'],\n",
       " ['non_topmed_faf95_sas', '0.00000e+00'],\n",
       " ['non_topmed_faf99_sas', '0.00000e+00'],\n",
       " ['controls_faf95_nfe', '0.00000e+00'],\n",
       " ['controls_faf99_nfe', '0.00000e+00'],\n",
       " ['non_cancer_faf95_afr', '0.00000e+00'],\n",
       " ['non_cancer_faf99_afr', '0.00000e+00'],\n",
       " ['non_cancer_faf95_amr', '4.95930e-04'],\n",
       " ['non_cancer_faf99_amr', '4.95370e-04'],\n",
       " ['non_topmed_faf95', '8.68300e-05'],\n",
       " ['non_topmed_faf99', '8.70800e-05'],\n",
       " ['non_neuro_faf95_nfe', '0.00000e+00'],\n",
       " ['non_neuro_faf99_nfe', '0.00000e+00'],\n",
       " ['non_neuro_faf95', '8.88700e-05'],\n",
       " ['non_neuro_faf99', '8.84700e-05'],\n",
       " ['non_topmed_faf95_nfe', '0.00000e+00'],\n",
       " ['non_topmed_faf99_nfe', '0.00000e+00'],\n",
       " ['controls_faf95_eas', '0.00000e+00'],\n",
       " ['controls_faf99_eas', '0.00000e+00'],\n",
       " ['controls_faf95_sas', '0.00000e+00'],\n",
       " ['controls_faf99_sas', '0.00000e+00'],\n",
       " ['faf95_nfe', '0.00000e+00'],\n",
       " ['faf99_nfe', '0.00000e+00'],\n",
       " ['non_topmed_faf95_eas', '0.00000e+00'],\n",
       " ['non_topmed_faf99_eas', '0.00000e+00'],\n",
       " ['controls_faf95_amr', '0.00000e+00'],\n",
       " ['controls_faf99_amr', '0.00000e+00'],\n",
       " ['non_neuro_faf95_eas', '0.00000e+00'],\n",
       " ['non_neuro_faf99_eas', '0.00000e+00'],\n",
       " ['non_cancer_faf95_nfe', '0.00000e+00'],\n",
       " ['non_cancer_faf99_nfe', '0.00000e+00'],\n",
       " ['non_cancer_faf95', '8.40400e-05'],\n",
       " ['non_cancer_faf99', '8.40500e-05'],\n",
       " ['non_cancer_faf95_sas', '0.00000e+00'],\n",
       " ['non_cancer_faf99_sas', '0.00000e+00'],\n",
       " ['non_topmed_faf95_afr', '0.00000e+00'],\n",
       " ['non_topmed_faf99_afr', '0.00000e+00'],\n",
       " ['controls_faf95', '0.00000e+00'],\n",
       " ['controls_faf99', '0.00000e+00'],\n",
       " ['popmax', 'amr'],\n",
       " ['AC_popmax', '2'],\n",
       " ['AN_popmax', '716'],\n",
       " ['AF_popmax', '2.79330e-03'],\n",
       " ['nhomalt_popmax', '0'],\n",
       " ['age_hist_het_bin_freq', '0|0|0|0|0|1|1|0|0|0'],\n",
       " ['age_hist_het_n_smaller', '0'],\n",
       " ['age_hist_het_n_larger', '0'],\n",
       " ['age_hist_hom_bin_freq', '0|0|0|0|0|0|0|0|0|0'],\n",
       " ['age_hist_hom_n_smaller', '0'],\n",
       " ['age_hist_hom_n_larger', '0'],\n",
       " ['non_topmed_popmax', 'amr'],\n",
       " ['non_topmed_AC_popmax', '2'],\n",
       " ['non_topmed_AN_popmax', '712'],\n",
       " ['non_topmed_AF_popmax', '2.80899e-03'],\n",
       " ['non_topmed_nhomalt_popmax', '0'],\n",
       " ['non_neuro_popmax', 'amr'],\n",
       " ['non_neuro_AC_popmax', '2'],\n",
       " ['non_neuro_AN_popmax', '708'],\n",
       " ['non_neuro_AF_popmax', '2.82486e-03'],\n",
       " ['non_neuro_nhomalt_popmax', '0'],\n",
       " ['non_cancer_popmax', 'amr'],\n",
       " ['non_cancer_AC_popmax', '2'],\n",
       " ['non_cancer_AN_popmax', '716'],\n",
       " ['non_cancer_AF_popmax', '2.79330e-03'],\n",
       " ['non_cancer_nhomalt_popmax', '0'],\n",
       " ['vep',\n",
       "  'C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000423562|unprocessed_pseudogene|||||||||||1|1790|-1||SNV|1|HGNC|38034||||||||||||||||||||||||||||||||||||||||||,C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000438504|unprocessed_pseudogene|||||||||||1|1790|-1||SNV|1|HGNC|38034|YES|||||||||||||||||||||||||||||||||||||||||,C|intron_variant&non_coding_transcript_variant|MODIFIER|DDX11L1|ENSG00000223972|Transcript|ENST00000450305|transcribed_unprocessed_pseudogene||2/5|ENST00000450305.2:n.98-40T>C||||||||1||1||SNV|1|HGNC|37102||||||||||||||||||||||||||||||||||||||||||,C|intron_variant&non_coding_transcript_variant|MODIFIER|DDX11L1|ENSG00000223972|Transcript|ENST00000456328|processed_transcript||1/2|ENST00000456328.2:n.360-40T>C||||||||1||1||SNV|1|HGNC|37102|YES|||||||||||||||||||||||||||||||||||||||||,C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000488147|unprocessed_pseudogene|||||||||||1|1831|-1||SNV|1|HGNC|38034||||||||||||||||||||||||||||||||||||||||||,C|intron_variant&non_coding_transcript_variant|MODIFIER|DDX11L1|ENSG00000223972|Transcript|ENST00000515242|transcribed_unprocessed_pseudogene||1/2|ENST00000515242.2:n.357-40T>C||||||||1||1||SNV|1|HGNC|37102||||||||||||||||||||||||||||||||||||||||||,C|intron_variant&non_coding_transcript_variant|MODIFIER|DDX11L1|ENSG00000223972|Transcript|ENST00000518655|transcribed_unprocessed_pseudogene||1/3|ENST00000518655.2:n.355-22T>C||||||||1||1||SNV|1|HGNC|37102||||||||||||||||||||||||||||||||||||||||||,C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000538476|unprocessed_pseudogene|||||||||||1|1838|-1||SNV|1|HGNC|38034||||||||||||||||||||||||||||||||||||||||||,C|downstream_gene_variant|MODIFIER|WASH7P|ENSG00000227232|Transcript|ENST00000541675|unprocessed_pseudogene|||||||||||1|1790|-1||SNV|1|HGNC|38034||||||||||||||||||||||||||||||||||||||||||,C|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00001576075|CTCF_binding_site|||||||||||1||||SNV|1||||||||||||||||||||||||||||||||||||||||||||']]"
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "list(map(lambda x: out if len(out:=x.split(\"=\"))>=2 elif out out.append(None), vcf[\"INFO\"].loc[0].split(\";\")))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1, 3, 5, 6]\n"
     ]
    }
   ],
   "source": [
    "a = [1, 3 ,5 ,6]\n",
    "\n",
    "if (count_size := a)[-1] >=3:\n",
    "    print(count_size)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def vcf_info_parse(info_text, INFO)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3.9.7 ('pytorch')",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  },
  "orig_nbformat": 4,
  "vscode": {
   "interpreter": {
    "hash": "3ab96668433c026957251cf6f38badc2738d85614bdebdf1faa3ece8c883f8f7"
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
